{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Finding metadata files\n",
    "\n",
    "In this analysis we'll explore our samples, looking for self-describing datasets and metadata files.\n",
    "\n",
    "From the file listing exercise we found that among the archive content there are several files of extensions like `*.json` `*.xml` `*.ttl` `*.rdf` indicating structured data, some of which potentially contain metadata. Here we will look further into their content and find to what extent these can be called self-describing or considered prototypical research objects.  The hypothesis is that such files, if present, would be neighbouring the files they describe within the same archive or record.\n",
    "\n",
    "We will also look for files which are directly named \"metadata\" and similar variants."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir -p data/metadata-files/rdf data/metadata-files/json data/metadata-files/xml data/metadata-files/metadata"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## RDF files\n",
    "\n",
    "Although RDF was conceived as a mechanism for structuredr metadata on a web, it has for a long time been used for holding any semantically structured data.  Recently a resurgence in metadata has occurred thanks to [schema.org](http://schema.org/) and [JSON-LD](http://json-ld.org/). An hypothesis is that most of the RDF data in Zenodo are domain-specific data dumps, as metadata would mainly be useful on the web (to be indexed by search engines) rather than in static archives. \n",
    "\n",
    "However there is no clear distinction between data and metadata, so we should also inspect the RDF files we have sampled to see if they use metadata-like properties such as <http://purl.org/dc/terms/creator> or <https://www.w3.org/ns/prov/wasDerivedFrom>. Metadata files describing neighbouring files should in theory use relative paths to identify them, so we should also check for URIs that are non-HTTP-based. \n",
    "\n",
    "First we'll explore what kind of filenames are used for RDF files, then we'll (re)download and extract them to look at their properties and URIs used. While we look for the common RDF extensions we are here excluding RDFa as it would be embedded in HTML/XHTML/SVG. These would be typically self-describing the document they are within, rather than the containing archive, but are worh a later look."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "! egrep -ri '(ttl|rdf|jsonld|n3|nq|nt|trig)$' data/*/listing > data/metadata-files/rdf/listing.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "By looking up the sample id we can find which corresponding ZIP files to download and extract.\n",
    "\n",
    "**TODO**: Change below to a workflow"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cat data/metadata-files/rdf/listing.txt | cut -d \":\" -f 1 | sort | uniq \\\n",
    "  | sed s/listing/sample/ | sed s/txt$/tsv/ | \\\n",
    "  xargs awk '{print $5}' > data/metadata-files/rdf/urls.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Lang_1.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC403.zip\r\n",
      "https://zenodo.org/api/files/602ad012-6cad-4e1c-9e13-ffaf65b282b6/raptr-manuscript.zip\r\n",
      "https://zenodo.org/api/files/5f4072d6-d4b9-4f95-afb4-4ca2da9366fd/moving-block-req-models.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_29.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC204.zip\r\n",
      "https://zenodo.org/api/files/d642ee67-643b-4687-a5f8-27a56a465eb8/AE-AT-Opt-PACT18.zip\r\n",
      "https://zenodo.org/api/files/978c426e-1be6-41c9-bf3f-017df44ce850/GC_QTOF_PP_samples.zip\r\n",
      "https://zenodo.org/api/files/5b9631aa-2aaa-4d1f-9bc7-64b68a184ee7/BostonFingerprints2014_RawData_tc10.zip\r\n",
      "https://zenodo.org/api/files/c7cd6c50-ca6a-4974-9073-ae4cdd77b515/Lang_57.zip\r\n",
      "https://zenodo.org/api/files/755f6ed5-133f-4d9d-b43b-bff56d814821/Bohdan-Khomtchouk/ENCODE_histone_geneXtendeR_analysis-v1.0.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC599.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC240.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC385.zip\r\n",
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Math_15.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC280.zip\r\n",
      "https://zenodo.org/api/files/a2fec3ee-69f7-467e-aeba-f9d8bab7853d/persons2.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC226.zip\r\n",
      "https://zenodo.org/api/files/d1656228-3f4c-416c-8dd0-c6a5699ca5bc/Math_81.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC102.zip\r\n",
      "https://zenodo.org/api/files/c7cd6c50-ca6a-4974-9073-ae4cdd77b515/Math_53.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_98.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC397.zip\r\n",
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Lang_48.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC529.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_3.zip\r\n",
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Math_34.zip\r\n",
      "https://zenodo.org/api/files/b69b3cff-1bdb-49be-911f-98160b72a315/sodra.zip\r\n",
      "https://zenodo.org/api/files/c7cd6c50-ca6a-4974-9073-ae4cdd77b515/Math_29.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC325.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC335.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Time_20.zip\r\n",
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Lang_6.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC394.zip\r\n",
      "https://zenodo.org/api/files/d1656228-3f4c-416c-8dd0-c6a5699ca5bc/Lang_16.zip\r\n",
      "https://zenodo.org/api/files/c7cd6c50-ca6a-4974-9073-ae4cdd77b515/Math_38.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_67.zip\r\n",
      "https://zenodo.org/api/files/f9911cdb-7923-4d1a-9693-9465ddd8dea3/Math_35.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_89.zip\r\n",
      "https://zenodo.org/api/files/abab110d-78c0-4425-883a-f7ec576d7eb9/Math_6.zip\r\n",
      "https://zenodo.org/api/files/6720773f-d88c-40ae-adee-70ba9e2cdb1d/jboss-5.1.0.zip\r\n",
      "https://zenodo.org/api/files/978c426e-1be6-41c9-bf3f-017df44ce850/GC_QTOF_PVC_samples.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC838.zip\r\n",
      "https://zenodo.org/api/files/021841c1-717b-456c-8f31-f7968f4122b5/CORALCorpus.zip\r\n",
      "https://zenodo.org/api/files/01dc9114-1695-4051-8076-b2028f32852b/apex2-0625-00315-625.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC434.zip\r\n",
      "https://zenodo.org/api/files/c7cd6c50-ca6a-4974-9073-ae4cdd77b515/Lang_18.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC129.zip\r\n",
      "https://zenodo.org/api/files/54def73c-2a6b-4741-82c6-9d6bd1618184/PMC199.zip\r\n",
      "https://zenodo.org/api/files/4f2abd85-48aa-4318-b022-0dd8d5b0ba97/all2bass.zip\r\n",
      "https://zenodo.org/api/files/4f8e78e2-e47b-4dcd-9556-508c5478a4b0/scene_metamers.zip\r\n",
      "https://zenodo.org/api/files/6f487bcb-e25f-47a0-8167-b963a0c9143d/ssd_resnet34_21.2_ckpt_0720.zip\r\n",
      "https://zenodo.org/api/files/f3b6428c-df73-4a38-af76-61ae429fb877/cib.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/28_1PQ6_AF2_prod_rep1.zip\r\n",
      "https://zenodo.org/api/files/a57dbac4-107b-47f1-b7a9-389880ff9555/kelu124/echomods-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/52b7ade3-6fb4-4b84-9646-b29ee72161e7/Archive.zip\r\n",
      "https://zenodo.org/api/files/ded35234-409b-4dac-be09-979db1e92ab8/ro-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/20_1PQ6_LBP_prod_rep3.zip\r\n",
      "https://zenodo.org/api/files/bebb3e23-1ec7-45a1-8442-7e7fa52c362f/earlyWearablesKit-v1.1.zip\r\n",
      "https://zenodo.org/api/files/ffc91401-c7bc-4d86-b704-196d99e338de/Research_information_management__Tools_for_the_humanities.zip\r\n",
      "https://zenodo.org/api/files/06562a9c-22e1-4c16-af79-efc21cec2b89/ro-crate-0.2.1.zip\r\n",
      "https://zenodo.org/api/files/f9496d11-7054-4fb4-9113-8aab197cca8d/BitBucket_Software_20180314.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/27_3IPS_AF2_prod_rep2.zip\r\n",
      "https://zenodo.org/api/files/a4aa53a2-f16e-4b35-b9f5-de1c88a77023/saved_results.zip\r\n",
      "https://zenodo.org/api/files/eadc289e-3fbf-4597-8672-68b363149bfd/python-docker-cme.zip\r\n",
      "https://zenodo.org/api/files/f5b0cfe1-96c4-4cec-9aa8-89294e0517f7/diggr-video-game-vocabulary-1.1.zip\r\n",
      "https://zenodo.org/api/files/6f00f3f6-d08a-4e15-9df4-a2b3d0abd102/hpc-uk/archer-benchmarks-report.03.2019.zip\r\n",
      "https://zenodo.org/api/files/f2bbed2c-ac77-4418-8aec-9b5acf650d7b/Esercitazioni_Birtele_Bogoni_Borsati.zip\r\n",
      "https://zenodo.org/api/files/93f3b2bc-3c89-4c53-8885-5bf3da7c4857/seenet-medieval/caedmonshymn-v1.1-beta.05.zip\r\n",
      "https://zenodo.org/api/files/4f2abd85-48aa-4318-b022-0dd8d5b0ba97/bass2bass_seq2seq.zip\r\n",
      "https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/RunTimeResearchObject-f0b553d37e4255a3291393948f3e308bd88ed301.zip\r\n",
      "https://zenodo.org/api/files/6800a11f-ec08-467d-b47f-773764c52997/seenet-medieval/caedmonshymn-v1.1.zip\r\n",
      "https://zenodo.org/api/files/321cb93e-a7b3-46fd-88af-9a5dc6998fcc/Research_information_management_guides.zip\r\n",
      "https://zenodo.org/api/files/54656f60-54c6-441c-bfca-1a9f34c5e0ac/UEL-Data_management_for_geoinformatics.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/29_3L0E_AF2_prod_rep3.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/20_1PQ6_LBP_prod_rep1.zip\r\n",
      "https://zenodo.org/api/files/c2826116-edcb-41be-bfb4-605c503de80e/sleep_transfer_learning.zip\r\n",
      "https://zenodo.org/api/files/4f2abd85-48aa-4318-b022-0dd8d5b0ba97/piano2piano.zip\r\n",
      "https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/cwltool-61797e04db7db33486d6074ec4ecc7dadd20d5e6.zip\r\n",
      "https://zenodo.org/api/files/e00adda9-1351-4a6a-b1e3-f0c4851afd38/tf_ssd_resnet34_22.1.zip\r\n",
      "https://zenodo.org/api/files/976daaeb-8755-431d-8c19-04d1a4cd2a35/UEL-supportDM_module_1__introducing_research_data_management.zip\r\n",
      "https://zenodo.org/api/files/ae20c12b-5380-496d-9569-d57c5f5bfcca/ensembleTopic-1.0.zip\r\n",
      "https://zenodo.org/api/files/e321cc47-d097-491a-89f6-eccc4c337def/UEL-supportDM_module_4__what_data_to_keep_and_why.zip\r\n",
      "https://zenodo.org/api/files/3c254ff2-836f-42d7-b159-2f94907fc0da/Relazione%20di%20idrologia%202.zip\r\n",
      "https://zenodo.org/api/files/bf7be498-8007-4800-a193-478e32ba9035/Marialaura-2017.zip\r\n",
      "https://zenodo.org/api/files/0c23d0b8-d61c-4dd1-8438-7bb71bfa3a57/artifact78.zip\r\n",
      "https://zenodo.org/api/files/c037e866-1fcd-4c7a-ab00-fc18061527bc/ore-0.9.0.zip\r\n",
      "https://zenodo.org/api/files/978810ac-4a1c-4906-b89d-de1875b3ad43/UEL-Sharing_and_Archiving_Your_Research_Data.zip\r\n",
      "https://zenodo.org/api/files/ff5099f4-a8ef-4f0c-9c51-343e4240e2f4/UEL-supportDM_module_5__cataloguing_data.zip\r\n",
      "https://zenodo.org/api/files/3a1ed3fe-25e7-4ea2-9b35-8ec89ff5caa5/Research_information_management__Organising_humanities_material.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/19_3IPS_LBP_prod_rep2.zip\r\n",
      "https://zenodo.org/api/files/e00adda9-1351-4a6a-b1e3-f0c4851afd38/ssd_resnet34_21.2_ckpt_0720.zip\r\n",
      "https://zenodo.org/api/files/28dacf54-8c8d-45b3-9d21-9d479e6c01b6/CloudSocket-Ontology-master.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/19_3IPS_LBP_prod_rep3.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/19_3IPS_LBP_prod_rep1.zip\r\n",
      "https://zenodo.org/api/files/d57a6567-7afb-4ccf-8ae3-bc815c2c33a3/commands_md64_tgrads_fgrads_8k_checkpoints.zip\r\n",
      "https://zenodo.org/api/files/483c858c-9308-43da-bb1e-078633b8e6a4/scene_metamers.zip\r\n",
      "https://zenodo.org/api/files/b0200664-832d-412b-98e7-18108b9fb37b/RDMoutreachvideos%202.zip\r\n",
      "https://zenodo.org/api/files/a51f933a-aa1a-4167-833d-65a00b6b4e2e/UEL-Writing_a_Data_Management_Plan-workshop.zip\r\n",
      "https://zenodo.org/api/files/1426557c-0ad9-44d6-a788-2740a528f9fa/ALog-BottleLogger-2.2.0.zip\r\n",
      "https://zenodo.org/api/files/11571323-8cc3-47fe-a183-ea79c2661469/letter_distortion_materials.zip\r\n",
      "https://zenodo.org/api/files/d57a6567-7afb-4ccf-8ae3-bc815c2c33a3/commands_md64_8k_checkpoints.zip\r\n",
      "https://zenodo.org/api/files/0459adf5-fe5f-44dd-891c-6f303fda45c1/UEL-supportDM_module_3__data_management_planning.zip\r\n",
      "https://zenodo.org/api/files/423c6fa2-8888-4f02-877e-292849c5ab74/Research_data_management_-_introductory_slideshows.zip\r\n",
      "https://zenodo.org/api/files/b77595c4-0fbf-4f77-8338-df03bc6fd23a/29_3L0E_AF2_prod_rep1.zip\r\n",
      "https://zenodo.org/api/files/f7d49068-7fd9-4d36-b331-7318f39d38e2/UEL-supportDM_module_6__sharing_data.zip\r\n",
      "https://zenodo.org/api/files/93af605e-e301-4bea-9bba-9490e2f06d6e/SAOImageDS9/SAOImageDS9-v8.0rc2.zip\r\n",
      "https://zenodo.org/api/files/00ef40ed-9d26-4742-a13f-c4385f161b03/ncsa/clowder-v1.6.2.zip\r\n",
      "https://zenodo.org/api/files/f79d369a-d343-408d-acb7-1579f515e3f5/PolMine/polmineR-v0.7.11.zip\r\n",
      "https://zenodo.org/api/files/824b3d3c-c591-4741-a863-193c793aa67f/common-workflow-language/cwlprov-0.6.0.zip\r\n",
      "https://zenodo.org/api/files/61df3fa6-eed3-4017-b965-38d44bc6d2ee/giuseppelettieri/linux-pspat-v4.13_pspat-0.9.zip\r\n",
      "https://zenodo.org/api/files/3f7aaa9a-45f9-4971-95f0-c960152761dc/cpmpercussion/PhaseRings-v1.2.1.zip\r\n",
      "https://zenodo.org/api/files/abbde7ea-7b20-48d5-8e0a-4550177ad66d/raamana/neuropredict-v0.3-DOI.zip\r\n",
      "https://zenodo.org/api/files/cfcebe14-7da9-4a07-a7c8-455b7ee0608c/geodynamics/aspect-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/3c7673f2-bf1f-4c90-8871-bad1fb022639/manoelcampos/cloudsim-plus-v1.2.3.zip\r\n",
      "https://zenodo.org/api/files/acca7c36-ed4d-42a9-93bd-e10d840ebc8c/chrismheiser/lipdnet-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/dd0fb7b0-2603-4e0d-a473-92b3541d30ba/martincallaghan/text_project_1-v1.0.zip\r\n",
      "https://zenodo.org/api/files/ca43e0ec-40f9-4ef2-9f4c-62598557c9af/clulab/eidos-Hackathon-NewMap.zip\r\n",
      "https://zenodo.org/api/files/aef83a38-ded9-49a5-a1b6-17f8e39f01e0/phenoscape-owl-tools-1.4.zip\r\n",
      "https://zenodo.org/api/files/91438144-8167-44f7-a939-27c4261d1a5e/dealii/dealii-v8.5.0-rc1.zip\r\n",
      "https://zenodo.org/api/files/45b1b0f5-d3e5-4d37-96a8-0ac0bb585489/jhpoelen/eol-globi-data-v0.12.4.zip\r\n",
      "https://zenodo.org/api/files/298d4070-7b04-45ae-b4e9-4ec6f797cd7a/okgreece/Alignment-v1.0.zip\r\n",
      "https://zenodo.org/api/files/e91bcf47-ef4b-4250-a112-995e6e625190/xanadu-linux-v0.5.8.zip\r\n",
      "https://zenodo.org/api/files/c2d0bcc4-15e1-4b7a-a2bf-d3c965fbdeb2/ORAC-DR-2015B.zip\r\n",
      "https://zenodo.org/api/files/499dd397-9520-4f37-a653-7f0f14402731/soapdenovo2-v1.0.zip\r\n",
      "https://zenodo.org/api/files/ccae7187-3853-454f-b840-5d70bdd484f9/SAOImageDS9/SAOImageDS9-v8.0rc.zip\r\n",
      "https://zenodo.org/api/files/713c8a6d-2b64-4763-9ff1-210ce4116669/kbdhaar/OCRSEV_Main-v1.0.zip\r\n",
      "https://zenodo.org/api/files/04c9f6eb-1ae6-456f-8c1c-25c121604134/StuntsPT/Structure_threader-1.3.0.zip\r\n",
      "https://zenodo.org/api/files/15c5e3b6-baf9-4fae-9748-fe3f39e79c31/morse-1.3-beta1.zip\r\n",
      "https://zenodo.org/api/files/cce1567b-42b8-4947-bd44-dbb48320b2ba/docmanager-3.2.1.zip\r\n",
      "https://zenodo.org/api/files/d323d2d3-f32c-49b2-acb6-4333eaac1eac/phon-1.7.0alpha3.zip\r\n",
      "https://zenodo.org/api/files/74fe840e-288e-4a7d-9844-a44d20c0de02/InsightSoftwareConsortium/ITK-v4.12.2.zip\r\n",
      "https://zenodo.org/api/files/74a9cd46-232c-43d1-9cb4-c548e230f82a/althonos/pronto-v0.6.2.zip\r\n",
      "https://zenodo.org/api/files/d6cb4124-00fc-4669-94c3-989af4c514a1/PX4/Firmware-v1.6.0-rc3.zip\r\n",
      "https://zenodo.org/api/files/2c75ccc9-bbc7-4f30-82aa-f958d61539e4/FESOM/fesom2-2.0.3.zip\r\n",
      "https://zenodo.org/api/files/aaabdfda-c0e6-4c76-8233-d6385f6929e7/cmssw-CMSSW_7_5_1.zip\r\n",
      "https://zenodo.org/api/files/fedf570f-0208-454f-8293-298ee6717579/BristolTopGroup/DailyPythonScripts-pre-python-package.zip\r\n",
      "https://zenodo.org/api/files/00042555-11ed-4342-81ce-a7e850756a48/jgm/pandoc-2.0.4.zip\r\n",
      "https://zenodo.org/api/files/0bc1dc9f-b611-440f-af0d-c3ca03c0567e/phon-2.0.8.zip\r\n",
      "https://zenodo.org/api/files/5bf12ea8-782e-423e-9cfe-7afee90ba0cf/EGI-Foundation/ansible-role-umd-v0.1.1.zip\r\n",
      "https://zenodo.org/api/files/c5cd30eb-4db2-409b-a70c-17fc7f1200f5/sys-bio/tellurium-2.0.7.zip\r\n",
      "https://zenodo.org/api/files/28d11d59-3939-459d-9372-47b988926c5c/jhpoelen/eol-globi-data-v0.9.18.zip\r\n",
      "https://zenodo.org/api/files/a659260d-8c8c-4638-9d82-775012b4c206/jgm/pandoc-2.0.6.zip\r\n",
      "https://zenodo.org/api/files/f33b0fbc-4205-41e4-b37f-76b879a9930d/rdkit/rdkit-Release_2017_09_3.zip\r\n",
      "https://zenodo.org/api/files/17d7a423-39a4-4fce-b33b-5d9eae241a73/cdk/cdk-cdk-2.2.zip\r\n",
      "https://zenodo.org/api/files/9f394811-a88c-4086-8670-3145894fbcc4/InsightSoftwareConsortium/ITK-v5.0.1.zip\r\n",
      "https://zenodo.org/api/files/3c8338ae-c417-41f6-a689-861bba09da22/StuntsPT/Structure_threader-0.3.2.zip\r\n",
      "https://zenodo.org/api/files/86827817-3159-44e2-afe5-4b28109cd7fe/gscripts-0.1.zip\r\n",
      "https://zenodo.org/api/files/6b30e50d-fb83-46b8-9fb5-aa7c9324c226/cfe-lab/Kive-v0.14.zip\r\n",
      "https://zenodo.org/api/files/2a225341-90a0-4692-94fa-554ee92b9adc/stan-v2.11.0.zip\r\n",
      "https://zenodo.org/api/files/2f2332cb-7e6f-477d-8790-4d87de322cd5/connorcoley/ASKCOS-v0.2.9.zip\r\n",
      "https://zenodo.org/api/files/1fee57a8-aa27-485c-b3bf-4fe06c9b4c1d/ag-gipp/MathMLTools-2.0.2.zip\r\n",
      "https://zenodo.org/api/files/4b056216-3e14-426a-8c55-0a2408cd9f7d/InsightSoftwareConsortium/ITK-v5.0.0.zip\r\n",
      "https://zenodo.org/api/files/6d2ff49e-80bc-4333-85c9-fc990ab5f7ad/kipoi/kipoi-v0.6.7.zip\r\n",
      "https://zenodo.org/api/files/53565158-4180-483d-b3e7-4d842a22ed48/StuntsPT/Structure_threader-v0.2.0.zip\r\n",
      "https://zenodo.org/api/files/45d2328a-8c9b-42ec-a84b-0d3fc4bc08a8/dgarijo/Widoco-v1.4.11.zip\r\n",
      "https://zenodo.org/api/files/8a4e6627-80d8-4824-a933-f5aeb27b0f7e/sys-bio/tellurium-2.0.13.zip\r\n",
      "https://zenodo.org/api/files/624c6f89-7e00-4d68-933b-891593a4dc7f/JavaSimulation.zip\r\n",
      "https://zenodo.org/api/files/05e2b7d1-e4cb-4590-95f1-9cec395099d4/raymondEhlers/reactionPlaneFit-1.2.zip\r\n",
      "https://zenodo.org/api/files/bada7590-514e-465a-a376-a8dcd48f7ffc/jasonclark/linked-people-bioscience-v1.0.zip\r\n",
      "https://zenodo.org/api/files/b30785bd-3d76-49f1-a4a8-d482211ecc87/samaaron/sonic-pi-v2.11.1.zip\r\n",
      "https://zenodo.org/api/files/9745c7ca-328b-4447-b5f9-09afb0d8284c/ncsa/clowder-v1.4.0.zip\r\n",
      "https://zenodo.org/api/files/214c5f26-0346-44e3-aa45-34a03b00fc1b/votca/csg-tutorials-v1.5.zip\r\n",
      "https://zenodo.org/api/files/1dadd68d-674b-49ec-bce4-8b99d5330738/Source%20codes%20%2B%20Related%20data%20%2B%20Scripts.zip\r\n",
      "https://zenodo.org/api/files/e1ba0985-5cdf-4b9c-a4d0-ff2d3a84edf9/pipauwel/IFCtoRDF-IFCtoRDF-0.1.zip\r\n",
      "https://zenodo.org/api/files/e0564f70-9cd8-4611-b173-1029ae7a7961/luca-tubiana/KymoKnot-v1.0.zip\r\n",
      "https://zenodo.org/api/files/c19cde22-847f-4fe2-a051-5c133198f387/PX4/Firmware-v1.6.1.zip\r\n",
      "https://zenodo.org/api/files/372a7bae-900c-442b-8a15-5962de5977ec/NETESOLUTIONS/NETELabs_CaseStudies-v0.9.zip\r\n",
      "https://zenodo.org/api/files/5ca9d2ce-58a2-4977-85e0-56ac159b3f2d/common-workflow-language/cwltool-1.0.20170828135420.zip\r\n",
      "https://zenodo.org/api/files/6f674aa5-da55-498d-a6a6-5a25ea7feda3/PX4/Firmware-v1.7.3.zip\r\n",
      "https://zenodo.org/api/files/cd8172f3-58c1-49c7-a438-989c525584c4/picongpu-beta-rc6.zip\r\n",
      "https://zenodo.org/api/files/c7e5bdce-e892-45c1-a919-dc0c2f75da3b/loprop-v1.0.zip\r\n",
      "https://zenodo.org/api/files/b93b6962-f4a4-4af8-ab95-653838d0b980/Biskit1943/tensorflow-v1.12.0-rc2.zip\r\n",
      "https://zenodo.org/api/files/e9e9b322-0cf6-43e9-b272-65c4f810dab5/jhpoelen/eol-globi-data-v0.9.11.zip\r\n",
      "https://zenodo.org/api/files/0a4a679e-9457-4278-ab16-1c934aac793a/unit-ontology-review-results-v1.2.0.zip\r\n",
      "https://zenodo.org/api/files/d534bd09-63a3-41b0-b754-08ad8cb7a48d/SeedHandling-John-2019-07-04.zip\r\n",
      "https://zenodo.org/api/files/d9787eaf-4ffb-42e3-abf6-236a96777a62/dgarijo/Widoco-v1.3.1.zip\r\n",
      "https://zenodo.org/api/files/c0b9b4e6-7316-4907-94c0-d1e6ccee1dd1/mesa-r3709.zip\r\n",
      "https://zenodo.org/api/files/232472d7-a5b9-4d2a-8ff2-ea146b52e703/jhpoelen/eol-globi-data-v0.8.12.zip\r\n",
      "https://zenodo.org/api/files/099a111a-eb77-44cc-ac9b-5fc1a5ed9ca3/YCML-v0.3.6.pr.zip\r\n",
      "https://zenodo.org/api/files/e8eaf17e-2807-41da-979b-a02a14bcfff6/myvariant.info-master-fb2a871.zip\r\n",
      "https://zenodo.org/api/files/cd662627-34c9-4fe0-82a9-2fe09fad135a/DailyPythonScripts-TOP-12-042_V4.zip\r\n",
      "https://zenodo.org/api/files/53307082-baa7-42ba-ad75-229e2592d9d8/SAOImageDS9/SAOImageDS9-v8.0rc6.zip\r\n",
      "https://zenodo.org/api/files/a0f45f7f-652b-4102-9c61-f71001614459/jgm/pandoc-2.2.3.1.zip\r\n",
      "https://zenodo.org/api/files/0b7dccbe-eaef-4214-bbf6-be6d89ba98ec/cmssw-CMSSW_6_2_0_SLHC26_patch3.zip\r\n",
      "https://zenodo.org/api/files/461152ca-79b5-4592-b500-787a65196b48/SERPENT-master.zip\r\n",
      "https://zenodo.org/api/files/131096f9-e7b8-428f-9f00-fa983485af00/xraypy/xraylarch-0.9.35rc2.zip\r\n",
      "https://zenodo.org/api/files/3035ddf9-38b4-43a0-8ecf-3cb688269121/taverna-scufl2-0.14.0.zip\r\n",
      "https://zenodo.org/api/files/66df01e6-2b5c-40dc-8f4f-01c4bb23b665/SLACKHA/pyJac-v1.0.3.zip\r\n",
      "https://zenodo.org/api/files/b0c7436b-57b7-4a57-a4c6-ad5104f0fbd2/examples.zip\r\n",
      "https://zenodo.org/api/files/4afbdb5c-3eb8-4592-a50e-8250b7a45359/rrdf-rrdf_2.1.2.zip\r\n",
      "https://zenodo.org/api/files/8207dffb-d657-4a17-ba99-e3da21340d17/labra/shaclex-v0.1.00.zip\r\n",
      "https://zenodo.org/api/files/8e15a508-8e2c-4a6a-9c9a-4ba59630a7dd/miledrousset/opentheso-4.4.9.zip\r\n",
      "https://zenodo.org/api/files/07ac81f4-1bc8-428d-b69d-93a9bdb67b51/embryoCropUI_WINDOWS.zip\r\n",
      "https://zenodo.org/api/files/2e206d9d-247a-47be-9cc2-1496169c9ccb/monarch-initiative/mondo-v2018-10-14.zip\r\n",
      "https://zenodo.org/api/files/0960fe87-b620-4c06-865e-4ebe71025d14/DailyPythonScripts-AN-14-071-v3.zip\r\n",
      "https://zenodo.org/api/files/e19e7654-c71f-46ca-9244-d2e6d2804392/ElektraInitiative/libelektra-0.8.22.zip\r\n",
      "https://zenodo.org/api/files/9fdfc756-edc1-4e3f-86c9-cbe2a6f106aa/docmanager-3.3.2.zip\r\n",
      "https://zenodo.org/api/files/8e7a0425-d38d-4cb1-b94e-3d1af5d9272b/SDG-InterfaceOntology/sdgio-v2018-05-11.zip\r\n",
      "https://zenodo.org/api/files/216ef8b5-c97a-425f-a905-da5f0bae60c5/matthiaskoenig/cy3sbml-v0.2.5.zip\r\n",
      "https://zenodo.org/api/files/1e8113cd-201f-429a-aa9e-583e9f357a44/DawnDiamond-1.7.1.v20141201-0949-linux32.zip\r\n",
      "https://zenodo.org/api/files/193b71c8-72d5-472e-a6e2-ade8a31ebb20/dartsim/dart-v6.3.1.zip\r\n"
     ]
    }
   ],
   "source": [
    "!cat data/metadata-files/rdf/urls.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## JSON files\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "! egrep -ri '(json|jsonld)$' data/*/listing > data/metadata-files/json/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "data/dataset/listing/zazo.txt:data/2017-05-18-145157833880/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-05-18-145157833880/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-05-18-152118060656/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-05-18-152118060656/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-01-102743523980/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-01-102743523980/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-06-145135235899/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-06-145135235899/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-06-150808383862/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-06-150808383862/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-101750079399/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-101750079399/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-104553891616/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-104553891616/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-110339927658/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-07-110339927658/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-08-111858535617/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-08-111858535617/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-08-114451734280/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-08-114451734280/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-09-105136319110/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-09-105136319110/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-102616090964/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-102616090964/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-112002661195/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-112002661195/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-140651300755/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-140651300755/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-143746652201/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-12-143746652201/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-102226367218/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-102226367218/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-110008089945/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-110008089945/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-113301977235/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-113301977235/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-140354392253/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-13-140354392253/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-14-100555273347/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-14-100555273347/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-16-103003933857/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-16-103003933857/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-16-110134843091/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-16-110134843091/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-103539303567/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-103539303567/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-105505400084/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-105505400084/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-112654518151/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-112654518151/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-140956193720/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-140956193720/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-145454904791/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-20-145454904791/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-21-142552882405/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-21-142552882405/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-21-144653168299/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-21-144653168299/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-101045562870/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-101045562870/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-111246014789/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-111246014789/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-141304357365/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-22-141304357365/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-23-104918496756/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-23-104918496756/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-23-115504081145/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-23-115504081145/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-104220488114/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-104220488114/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-112331105839/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-112331105839/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-144602575734/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-26-144602575734/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-27-105946305604/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-27-105946305604/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-27-143652394670/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-27-143652394670/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-28-110943922717/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-28-110943922717/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-29-105524492596/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-06-29-105524492596/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-102437565816/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-102437565816/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-113314145692/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-113314145692/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-120041673042/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-05-120041673042/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-093740908665/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-093740908665/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-101405806926/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-101405806926/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-110834943320/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-110834943320/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-115110822277/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-115110822277/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-133314899095/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-133314899095/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-140054776209/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-140054776209/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-143156556458/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-06-143156556458/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-094957490615/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-094957490615/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-102026329707/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-102026329707/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-113218785067/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-113218785067/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-141201836000/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-07-141201836000/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-104707663131/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-104707663131/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-114625404870/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-114625404870/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-142415953974/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-142415953974/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-144835853060/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-10-144835853060/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-100510760140/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-100510760140/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-104650618055/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-115151188934/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-115151188934/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-135134002524/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-135134002524/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-143529518329/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-11-143529518329/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-12-101454960354/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-12-101454960354/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-13-103326543970/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-13-103326543970/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-13-133638108581/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-13-133638108581/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-103610065352/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-103610065352/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-113137465891/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-113137465891/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-134029609907/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-134029609907/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-140952276828/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-14-140952276828/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-101105152664/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-101105152664/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-120012987344/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-120012987344/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-135017208460/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-135017208460/visual_tracking.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-142114896097/freeplay.poses.json\r\n",
      "data/dataset/listing/zazo.txt:data/2017-07-17-142114896097/visual_tracking.poses.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/comments.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/delimiters.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/interpolation.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/inverted.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/lambdas.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/partials.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/whisker/specs/sections.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/comments.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/delimiters.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/interpolation.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/inverted.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/lambdas.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/partials.json\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/whisker/specs/sections.json\r\n",
      "data/dataset/listing/zbtb.txt:CORALCorpus/HTML/package-lock.json\r\n",
      "data/dataset/listing/zbtb.txt:CORALCorpus/HTML/package.json\r\n",
      "data/dataset/listing/zatw.txt:Artists.json\r\n",
      "data/dataset/listing/zatw.txt:Artworks.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func_se/func_00.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func_se_op/func_00.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_06.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_04.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_03.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_01.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_07.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_05.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/func/func_02.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_pdw.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_inv1_phase.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_t1.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_inv1.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_pdw_phase.json\r\n",
      "data/dataset/listing/zbna.txt:sub-04/anat/mp2rage_uni.json\r\n",
      "data/dataset/listing/zayo.txt:weecology-portalPredictions-821c804/.zenodo.json\r\n",
      "data/dataset/listing/zbjc.txt:weecology-portalPredictions-7706ce5/.zenodo.json\r\n",
      "data/dataset/listing/zbbe.txt:weecology-PortalData-c9ca1a8/.zenodo.json\r\n",
      "data/dataset/listing/zamb.txt:emmanuel-senft-data-foodchain-3089664/supervisor.json\r\n",
      "data/dataset/listing/zbbn.txt:ricklupton-uk-steel-model-9303f6e/.zenodo.json\r\n",
      "data/dataset/listing/zbbn.txt:ricklupton-uk-steel-model-9303f6e/datapackage.json\r\n",
      "data/dataset/listing/sk.txt:Artists.json\r\n",
      "data/dataset/listing/sk.txt:Artworks.json\r\n",
      "data/dataset/listing/qo.txt:persons2/training-bis/persons2-input.json\r\n",
      "data/dataset/listing/qo.txt:persons2/training/persons2-input-filtered.json\r\n",
      "data/dataset/listing/qo.txt:persons2/training/persons2-input.json\r\n",
      "data/dataset/listing/qo.txt:persons2/validation-bis/persons2-input.json\r\n",
      "data/dataset/listing/qo.txt:persons2/validation/persons2-input-filtered.json\r\n",
      "data/dataset/listing/zahn.txt:zenodo_open_metadata_06_03_2017.json\r\n",
      "data/dataset/listing/ek.txt:Fe-Example/1/parameters.json\r\n",
      "data/dataset/listing/ek.txt:Fe-Example/2/parameters.json\r\n",
      "data/dataset/listing/ek.txt:Fe-Example/3/parameters.json\r\n",
      "data/dataset/listing/zbqf.txt:weecology-portalPredictions-4e58c71/.zenodo.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Brain_Trauma.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Brain_Traumaa.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Brain_Traumab.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Brain_Trauma_e1.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Stroke_Angio.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Stroke_Angio_e1.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Stroke_Angio_e1a.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_CT_Stroke_Angio_e1b.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_DWI_Tra.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_DWI_Traa.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_FLAIR_cor.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_t1_se_sag.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_T2_FL_2d_Hemo.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_t2_tse_tra.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15a.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15b.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15c.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15d.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15e.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15f.json\r\n",
      "data/dataset/listing/zbjj.txt:pat-21/func/pat-21_MR_TOF_3D_Dec15g.json\r\n",
      "data/dataset/listing/nv.txt:LANL_Mustang_parquet/workload/schema-1.0/generic_information.json\r\n",
      "data/dataset/listing/zazr.txt:Data_and_Analyses/CSFDataAnalysis/jsonlab_0.9.0/jsonlab/examples/example1.json\r\n",
      "data/dataset/listing/zazr.txt:Data_and_Analyses/CSFDataAnalysis/jsonlab_0.9.0/jsonlab/examples/example2.json\r\n",
      "data/dataset/listing/zazr.txt:Data_and_Analyses/CSFDataAnalysis/jsonlab_0.9.0/jsonlab/examples/example3.json\r\n",
      "data/dataset/listing/zazr.txt:Data_and_Analyses/CSFDataAnalysis/jsonlab_0.9.0/jsonlab/examples/example4.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/regions/L1__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.01.01/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.04.00.t/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.04.00.t/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.04.00.t/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.04.00.t/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.04.00.t/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/regions/L1__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J115/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.03.00.t/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.03.00.t/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.03.00.t/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.03.00.t/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.03.00.t/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/regions/L1__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/YST/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/regions/L1__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.02.00/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J123/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J123/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J123/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J123/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/J123/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.00.00/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.00.00/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.00.00/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.00.00/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/N.00.00/info.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/regions/L2__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/regions/L4__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/regions/consensus_regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/regions/L3__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/regions/L1__regions.json\r\n",
      "data/dataset/listing/zava.txt:WEBSITE/K53/info.json\r\n",
      "data/dataset/listing/zacm.txt:matthewgamble-clinicalcodes.article5-8e64a7b/.ro/manifest.json\r\n",
      "data/dataset/listing/zacm.txt:matthewgamble-clinicalcodes.article5-8e64a7b/annotations/articlecomments.json\r\n",
      "data/dataset/listing/zacm.txt:matthewgamble-clinicalcodes.article5-8e64a7b/annotations/articledescription.json\r\n",
      "data/dataset/listing/zacm.txt:matthewgamble-clinicalcodes.article5-8e64a7b/codelists/codelists.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke).json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)a.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)b.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1a.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1b.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1c.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1d.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Plain_Brain_(Stroke)_e1e.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1a.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1b.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1c.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1d.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_CT_Stroke_Angio_e1e.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_cor.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_dADC.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_DTI_calib_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_DTI_P6d_noniso_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_eEADC.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_facFA_Color.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_faFA.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_FLAIR_10800_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_isoDWI_b1000_iso.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_LT.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_MIP_s3DI_MC_resol_match_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_RT.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_s3DI_MC_resol_match_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_SWIp_fast_SENSE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_SWIp_MIP.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_TRA.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_TUMBLE.json\r\n",
      "data/dataset/listing/zbyh.txt:pat-15/func/pat-15_MR_WIP_T1W3Drapid_SENSE.json\r\n",
      "data/dataset/listing/zajr.txt:kuopio/routes.geojson\r\n",
      "data/dataset/listing/zajr.txt:kuopio/sections.geojson\r\n",
      "data/dataset/listing/zajr.txt:kuopio/stops.geojson\r\n",
      "data/dataset/listing/zepd.txt:workflowhub_epigenomics_dataset-hep_grid5000_schema-0-2_epigenomics-hep-g5k-run001_parquet/workload/schema-1.0/generic_information.json\r\n",
      "data/dataset/listing/zfpq.txt:globalbioticinteractions-Catalogue-of-Afrotropical-Bees-b82c185/globi.json\r\n",
      "data/dataset/listing/fm.txt:weecology-PortalData-f6b9d5c/.zenodo.json\r\n",
      "data/dataset/listing/zbhb.txt:weecology-forecasts-08e957c/.zenodo.json\r\n",
      "data/dataset/listing/zazi.txt:sem_stemmed.json\r\n",
      "data/dataset/listing/zazi.txt:sem_unstemmed.json\r\n",
      "data/dataset/listing/zazi.txt:sur_stemmed.json\r\n",
      "data/dataset/listing/zazi.txt:sur_unstemmed.json\r\n",
      "data/dataset/listing/zazi.txt:template.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006835.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010519.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010336.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009616.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013126.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001966.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN554889.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003540.2-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002791.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM420293.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011112.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007699.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015219.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003899.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011522.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000474.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001641.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010114.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012095.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010334.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP012330.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ311875.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012382.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010849.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN831039.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014960.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-31.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002883.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009914.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE017283.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-34.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP009493.1-26.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP006618.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM408590.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001737.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FO117623.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012382.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009407.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011545.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009716.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014960.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006850.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001641.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011341.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000431.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000580.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002885.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003322.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010332.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009927.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012837.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE572590.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003987.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006259.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010818.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006871.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002299.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012136.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013260.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP006618.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009483.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003061.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003214.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003213.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003207.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002299.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN868938.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013142.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE017283.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003324.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003699.2-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM711867.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009500.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003234.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009493.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003540.2-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-35.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013219.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000454.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010849.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012022.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000384.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001700.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013738.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003215.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007809.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003652.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012750.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP009493.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012150.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013261.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010407.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015219.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011522.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP014547.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001809.2-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000854.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014958.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003275.1-37.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007219.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014959.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003788.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002040.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN554889.1-30.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP014719.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003323.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012382.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012095.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009110.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001686.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014950.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003407.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014196.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006936.2-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010849.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FM211192.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG917972.2-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009493.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004370.1-23.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012181.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003323.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003077.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012949.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013142.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009755.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011868.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012070.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010114.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003949.1-23.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010797.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009447.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003900.2-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010339.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006259.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012382.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001778.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003053.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009613.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000384.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN554889.1-35.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013741.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015098.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002857.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001662.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007809.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002593.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011664.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013263.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-33.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010340.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009408.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007574.1-35.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010338.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003491.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG917972.2-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011522.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002801.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FR878060.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009124.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013745.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010873.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN881739.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011489.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001663.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000249.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009754.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006850.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012022.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000481.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005929.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014954.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003053.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003215.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011309.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP009493.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012573.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002666.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007219.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003699.2-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003347.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002251.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005928.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006764.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FO117623.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN554889.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014485.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014313.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003275.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003761.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007220.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002638.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-35.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE000516.2-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014950.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001814.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009124.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010519.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003876.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007255.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FR878060.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011522.1-36.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006842.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN881739.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003777.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014518.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004047.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013129.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001683.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003119.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010337.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015098.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013142.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003699.2-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002638.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007803.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003761.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003876.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006871.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010329.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-30.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP014719.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013738.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014952.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001641.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002907.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013743.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003494.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007803.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009427.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003987.1-30.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007809.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009243.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005080.1-34.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000854.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007574.1-30.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003491.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007255.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014952.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011492.1-28.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001095.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP008957.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG917972.2-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002299.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000518.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002475.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006578.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015220.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003211.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003900.2-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000717.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003215.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012949.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015098.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009927.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010071.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003216.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002994.1-33.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014485.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011269.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010071.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011542.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-42.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014959.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009615.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007219.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009481.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011295.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002801.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014485.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003410.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FO203431.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN563149.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009408.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002665.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000820.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006259.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001662.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009101.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014518.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009426.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003761.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009616.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012150.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003210.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004370.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002162.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000511.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CU458896.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003729.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002040.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011312.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011862.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CU458896.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014209.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003152.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE016958.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014956.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007220.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011022.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000656.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002993.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000511.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012506.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000480.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003385.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003777.1-28.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009482.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE572590.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001700.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003877.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP006618.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009101.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002095.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM412059.2-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001809.2-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009407.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003324.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP008744.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012506.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003322.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013219.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000717.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001778.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000580.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000820.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003053.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009499.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP011115.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013743.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004085.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003275.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012506.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-34.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009494.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010849.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002917.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006996.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG813240.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AL123456.3-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013738.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-38.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-28.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000511.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003697.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-52.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000249.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005929.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006936.2-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002994.1-35.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000511.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE000516.2-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010113.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003322.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010519.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009494.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010332.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014960.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014513.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003949.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012382.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001736.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010818.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002884.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013049.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014951.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-46.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-31.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011341.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002993.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001642.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CU458896.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014951.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015220.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN881739.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002993.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP011115.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP008923.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002885.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011545.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001736.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010519.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012837.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003876.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FO082843.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003053.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005386.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003729.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM408590.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002385.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000431.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011295.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FN554889.1-36.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003062.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HE804045.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014485.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010339.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009426.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003169.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009100.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FR878060.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000474.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010849.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007803.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007699.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001658.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014950.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006716.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012949.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000580.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP012555.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000511.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010338.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-50.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006996.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015098.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014952.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP008944.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003788.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013219.1-23.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006842.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001976.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP015098.1-26.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003324.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009124.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006850.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP012284.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005959.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011530.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-26.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010113.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010331.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012194.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002801.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014957.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009616.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003900.2-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013743.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP012204.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004374.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013745.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG813240.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012095.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010114.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009407.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014959.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003077.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001663.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002047.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN881739.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AM412059.2-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007027.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012095.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000717.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003078.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003152.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000325.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010340.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000518.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010329.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003206.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-33.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011664.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003385.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012044.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007699.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011022.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002786.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011773.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007699.1-28.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003211.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009427.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002801.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002000.1-23.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007574.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014958.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-20.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012354.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005286.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003421.2-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007790.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003987.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014241.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009615.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AL123456.3-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-37.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009481.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003876.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001663.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP008924.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002871.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009447.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007722.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001802.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011492.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000431.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009100.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006996.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003788.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003062.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/BA000030.4-36.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009449.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-39.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009495.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE000516.2-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003053.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009616.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013260.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/FQ859185.1-32.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000249.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000518.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/HG813240.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003119.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012044.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011913.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003248.2-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006259.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005928.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003987.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010827.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014955.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002994.1-29.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CU458896.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004370.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004048.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003899.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006259.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003990.1-24.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002040.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014959.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003407.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001630.1-13.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000850.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002280.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011883.2-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009407.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-21.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002475.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002871.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011492.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007809.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014956.1-18.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007722.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010337.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001736.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP005928.1-17.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002040.1-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012090.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009480.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002097.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002883.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009447.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011913.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP014573.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003385.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN881739.1-34.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006272.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP012555.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014475.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009483.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/LN997842.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002786.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003949.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003169.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010968.1-27.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013219.1-22.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002786.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007155.1-31.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007803.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009438.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP004048.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001631.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP010989.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000854.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002638.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AE016822.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003720.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000854.1-2.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000386.1-4.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP006835.1-10.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002884.1-15.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009495.1-14.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003421.2-6.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003987.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-19.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP012752.1-30.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001662.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000518.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP007699.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP011295.1-16.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP003219.1-34.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP002896.1-11.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP000479.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009481.1-7.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP013297.1-3.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP014954.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP001663.1-5.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009483.1-12.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009480.1-9.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009426.1-1.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP011115.1-25.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/AP010918.1-8.json\r\n",
      "data/dataset/listing/xf.txt:antiSMASH_Actinobacterial_BGCs/bgcJsonOutput/CP009495.1-10.json\r\n"
     ]
    }
   ],
   "source": [
    "! head -n 1000 data/metadata-files/json/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! grep jsonld data/metadata-files/json/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! egrep -i 'meta.?data' data/metadata-files/json/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! grep -i manifest data/metadata-files/json/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://zenodo.org/api/files/602ad012-6cad-4e1c-9e13-ffaf65b282b6/raptr-manuscript.zip\r\n",
      "https://zenodo.org/api/files/792b2464-eedf-427f-8166-883a85de8b5d/Fe-Example_input_files.zip\r\n",
      "https://zenodo.org/api/files/ec40beec-f99c-4f95-914e-dcc329df3a84/weecology/PortalData-1.104.0.zip\r\n",
      "https://zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/rome.zip\r\n",
      "https://zenodo.org/api/files/579d4580-d2b4-4441-ab11-3a9bdbf8aa89/CN-TU/nta-meta-analysis-2018.11.zip\r\n",
      "https://zenodo.org/api/files/5d6ce4de-f954-4e1d-9285-4fce0ef0a146/Collocated%20Datesets%20for%20Terra%20and%20Aqua.zip\r\n",
      "https://zenodo.org/api/files/5935dbec-ac23-4b32-8421-4b3bf07e3f75/LANL_Mustang_parquet.zip\r\n",
      "https://zenodo.org/api/files/217beb43-3a3a-429b-86cf-a154ded2f6e5/graphs_json.zip\r\n",
      "https://zenodo.org/api/files/a2fec3ee-69f7-467e-aeba-f9d8bab7853d/persons2.zip\r\n",
      "https://zenodo.org/api/files/706a040e-d9be-41c7-8677-cd94d52a1fb8/data_metrics.zip\r\n",
      "https://zenodo.org/api/files/8b54bee9-9932-4f9d-8cbb-61f83c5ccf56/collection-v1.11.zip\r\n",
      "https://zenodo.org/api/files/335b11e6-9ee3-4fc6-b892-4b650bcfe02b/collection-v1.10.zip\r\n",
      "https://zenodo.org/api/files/7d1192dc-a006-4ee0-a718-2b894fec7a6e/weecology/forecasts-2018-02-16.zip\r\n",
      "https://zenodo.org/api/files/ed4de091-d86d-465a-bc3b-9d70e12a77b4/antiSMASH_Actinobacterial_BGCs.zip\r\n",
      "https://zenodo.org/api/files/e0425375-1288-4105-b970-08ebd8c654a8/dataset_for_validation_experiments_mitmprobe.zip\r\n",
      "https://zenodo.org/api/files/9712835c-b4d6-4fcc-b0ec-dba4f346dd40/clinicalcodes.article5-v0.1.0.zip\r\n",
      "https://zenodo.org/api/files/e6b5c812-92ab-4552-8a03-1782bc8245bc/wfschneidergroup/SI-mehta-plasma-N2-vibrations-0.5.zip\r\n",
      "https://zenodo.org/api/files/77d2fd12-585f-4e94-a5ea-56ba5f9e942d/zenodo_open_metadata_06_03_2017.zip\r\n",
      "https://zenodo.org/api/files/c101d543-3be4-4734-91bc-385cbc8123f3/gbv/kos-registry-20190603.zip\r\n",
      "https://zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/kuopio.zip\r\n",
      "https://zenodo.org/api/files/3500a1bc-360a-45f4-8d31-4b6c61d942bb/Pegasus_P7_parquet.zip\r\n",
      "https://zenodo.org/api/files/64b75e05-2673-4a59-8425-97be3d5b3ca0/emmanuel-senft/data-foodchain-v1.0.zip\r\n",
      "https://zenodo.org/api/files/59a30a09-aa06-4799-ad5c-ecc010d9724d/collection-v1.25.zip\r\n",
      "https://zenodo.org/api/files/a3b36c12-4a7b-4fa8-9172-d2be937f8cf1/WEBSITE_basic.zip\r\n",
      "https://zenodo.org/api/files/a1fa0f27-cbc8-4eb5-92ce-30c8076e3865/Supplementary%20files.zip\r\n",
      "https://zenodo.org/api/files/742ff057-9769-4469-8c66-15b96e672336/WL04.zip\r\n",
      "https://zenodo.org/api/files/6cce1121-0f26-4516-88b7-17bb1663a0c8/survey.zip\r\n",
      "https://zenodo.org/api/files/14466297-96dc-4445-83d0-4c3299eb42c4/weecology/PortalData-1.98.0.zip\r\n",
      "https://zenodo.org/api/files/a191b63a-9d66-4c5d-81ac-18987486d7b2/weecology/portalPredictions-2018-04-27.zip\r\n",
      "https://zenodo.org/api/files/0c32f27d-01c5-412a-bf6c-a8fc42f55cc4/SeminalSurveyDBLP.zip\r\n",
      "https://zenodo.org/api/files/4222b3e2-3777-454f-b1c2-95e264431b46/dataset-nobagfiles.zip\r\n",
      "https://zenodo.org/api/files/17542c5f-9242-4403-a6cb-8ec034c04c9f/Data_and_Analyses.zip\r\n",
      "https://zenodo.org/api/files/05df56e1-f3e3-4728-a09e-b6b4f0d0a340/weecology/PortalData-1.72.0.zip\r\n",
      "https://zenodo.org/api/files/b2572be2-9276-4433-99e5-f7bc030ff081/ricklupton/uk-steel-model-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/10cfa833-96ce-43c8-a01d-a07675e842d4/domain_split.zip\r\n",
      "https://zenodo.org/api/files/b60d6c09-61d5-47c3-b4d4-cfc06c6a3b80/weecology/forecasts-2018-10-19.zip\r\n",
      "https://zenodo.org/api/files/72dd54ca-efe0-4be7-a275-928fa43912cb/weecology/portalPredictions-2018-09-07.zip\r\n",
      "https://zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_21.zip\r\n",
      "https://zenodo.org/api/files/4fcac888-9cf5-44b9-b0d5-94ceba1f23dc/lingpy/language-island-paper-v1.0.zip\r\n",
      "https://zenodo.org/api/files/b37db781-e7a5-4c00-9750-6bd8c2ffbcd2/sub-04.zip\r\n",
      "https://zenodo.org/api/files/875d5f34-367a-4cab-b243-a20df1ead590/weecology/portalPredictions-2018-03-16.zip\r\n",
      "https://zenodo.org/api/files/bbc7f075-eac0-4968-851f-0a475a09819b/ComplexNetTSP/MultilayerParis-v1.0.zip\r\n",
      "https://zenodo.org/api/files/abb3d7fa-d2de-43bd-9423-269746cf8b67/weecology/portalPredictions-2018-03-09.zip\r\n",
      "https://zenodo.org/api/files/021841c1-717b-456c-8f31-f7968f4122b5/CORALCorpus.zip\r\n",
      "https://zenodo.org/api/files/7ce871ea-5f97-43db-853a-ba1b4ddf07fd/askalon-new_ee26_parquet.zip\r\n",
      "https://zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_15.zip\r\n",
      "https://zenodo.org/api/files/c0e7a65b-97e2-4d61-9b6b-150d6933327f/MathStackExchangeAPI_Part_2_TimeStamps_1484447260_1512760268.json.zip\r\n",
      "https://zenodo.org/api/files/d28a40ef-f657-4c4b-945b-c4f2b805b81c/workflowhub_epigenomics_dataset-hep_grid5000_schema-0-2_epigenomics-hep-g5k-run001_parquet.zip\r\n",
      "https://zenodo.org/api/files/4b96de2a-2a4f-4078-83ec-098c28098863/globalbioticinteractions/Catalogue-of-Afrotropical-Bees-v0.1.zip\r\n",
      "https://zenodo.org/api/files/ff29524f-d899-4699-974f-584f5247b19e/artifact.zip\r\n",
      "https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Sending%20Christmas%20Greetings%20IaaS.zip\r\n",
      "https://zenodo.org/api/files/621c107b-e0e8-416d-be69-d6d0ce82bd9c/datacarpentry/R-ecology-lesson-v2017.04.3.zip\r\n",
      "https://zenodo.org/api/files/e5a55eb3-b97e-4591-99f9-063400e0d852/glo-d-i-um/glo-d-i-um.github.io-1.0.zip\r\n",
      "https://zenodo.org/api/files/5d1dc30d-0d3e-48a2-a335-15491bd55b0e/terraref/documentation-v0.01.zip\r\n",
      "https://zenodo.org/api/files/a30fb1b2-f042-4cf8-a1ed-6ff79c076deb/swcarpentry/python-novice-gapminder-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/25c965d3-18fb-4737-b007-c215a37319be/analyses.zip\r\n",
      "https://zenodo.org/api/files/6739c06c-f1a7-4a98-8e4f-f012fd023b03/datacarpentry/r-raster-vector-geospatial-v2018.08.2.zip\r\n",
      "https://zenodo.org/api/files/f3ce341c-70ab-4acb-95e3-dcaeaf0f930d/via-src-2.0.0.zip\r\n",
      "https://zenodo.org/api/files/34f5cc18-c5e8-4e08-af74-1f7f717bff13/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/ba8f9c1c-a22b-4254-91f1-65d215a80656/Diaspora_proof_of_concept.zip\r\n",
      "https://zenodo.org/api/files/3b2af3a0-ec88-4eaf-8738-1a7d27609e43/CIDER-Seq%20Results.zip\r\n",
      "https://zenodo.org/api/files/b856436c-80a0-449d-a44f-447812c06fbd/datacarpentry/spreadsheets-socialsci-v2018.05-2.zip\r\n",
      "https://zenodo.org/api/files/2ac05097-56ee-4a31-bbf6-b28f25c70ed8/Docker_Analysis.zip\r\n",
      "https://zenodo.org/api/files/9d2e22e0-b077-4844-96ee-09f2556e1310/2019-05-23_HandsOn_GrainBoundary1.zip\r\n",
      "https://zenodo.org/api/files/aceb9a66-c991-4d19-a299-0c64061e1f19/larmarange/analyse-R-2018-12-08.zip\r\n",
      "https://zenodo.org/api/files/a991f7bc-f870-42ff-8ed1-9613fe3f9c46/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/ded35234-409b-4dac-be09-979db1e92ab8/ro-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/239861a2-8ee1-4129-879f-26119cb0c2bc/1_orochi-0.1.zip\r\n",
      "https://zenodo.org/api/files/6fd24302-64d9-4464-83dc-87ad48d54744/models.zip\r\n",
      "https://zenodo.org/api/files/1bac5a90-319a-4d22-a462-15ca96c5f080/hydrosoc/rhydro_EGU19-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/3ae0de4d-2e4e-44f0-bace-84cd4dff924b/datacarpentry/r-intro-geospatial-v2018.08.1.zip\r\n",
      "https://zenodo.org/api/files/06562a9c-22e1-4c16-af79-efc21cec2b89/ro-crate-0.2.1.zip\r\n",
      "https://zenodo.org/api/files/5a46c61f-c44c-4530-a9db-4a40fcb4f071/lazappi/phd-thesis-2019-05-08.zip\r\n",
      "https://zenodo.org/api/files/8c8516ea-3412-4bb9-849d-5a4ef91a6f59/datacarpentry/socialsci-workshop-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/9fef9cdb-5901-48e0-ad55-63328bfaf7b9/adamisntdead/higher-shapes-1.0.1.zip\r\n",
      "https://zenodo.org/api/files/9ddb5281-3ddd-4bd4-9746-fa74089c8b3a/swcarpentry/git-novice-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/92d8577a-bf28-452c-847f-3a542087daeb/Open-Scholarship-Strategy/indexed-2.04.zip\r\n",
      "https://zenodo.org/api/files/7968d3ec-1501-4bd2-8bd3-3d934dd492a2/datacarpentry/OpenRefine-ecology-lesson-v2017.04.0.zip\r\n",
      "https://zenodo.org/api/files/4f58ed6e-f2b8-449a-8d38-2aa8de8f0436/sensebox-binder.zip\r\n",
      "https://zenodo.org/api/files/0a61420f-dcd0-42a9-8cb1-b80a27533210/result.zip\r\n",
      "https://zenodo.org/api/files/dc18a22f-3b8a-460d-acdf-667f4295f09c/alt-pub-alt-pub_v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/d8f24c9f-d583-4fdd-bbb9-008342056fa0/onecodex/bacillus-anthracis-panel-jan2019.zip\r\n",
      "https://zenodo.org/api/files/687b0e16-5d65-4def-8a7e-e301845a27d4/datacarpentry/python-ecology-lesson-es-v2019.01.0.zip\r\n",
      "https://zenodo.org/api/files/75358699-957e-49c0-8d1f-d885b9a9acad/programminghistorian/jekyll-2017-11.zip\r\n",
      "https://zenodo.org/api/files/0598fa8e-78b7-4826-a412-9cd5dd37c7db/UofTCoders/rcourse-v2.2.0.zip\r\n",
      "https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/RunTimeResearchObject-f0b553d37e4255a3291393948f3e308bd88ed301.zip\r\n",
      "https://zenodo.org/api/files/5a570ce1-9182-4931-929b-cd24fc663c13/common-workflow-language/user_guide-v0.3.zip\r\n",
      "https://zenodo.org/api/files/966442f8-3955-4ab1-8706-fd1e0a84b300/Bachelorarbeit.zip\r\n",
      "https://zenodo.org/api/files/6707b75d-848d-46f5-8cbf-36f7047a6458/data.zip\r\n",
      "https://zenodo.org/api/files/94257e8d-bb76-45df-8448-68b04dfeb2bc/v.2018-02-28.zip\r\n",
      "https://zenodo.org/api/files/cb9df964-0c4e-4829-a6a4-b9739ccf3c19/DMPs.zip\r\n",
      "https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Send%20Invoice%20PaaS.zip\r\n",
      "https://zenodo.org/api/files/b607155e-c427-4380-86fd-4667ee10fa4f/tsamsonov/r-geo-course-v0.1-alpha.zip\r\n",
      "https://zenodo.org/api/files/1a03eb70-03a2-454c-8f8c-a03c4775ea44/ICSE-data.zip\r\n",
      "https://zenodo.org/api/files/0dbaadcb-0029-45e9-b25f-8f426e54d1d7/AnnexesNumeriques.zip\r\n",
      "https://zenodo.org/api/files/e934c4b3-70ae-4214-b045-10e166125907/swcarpentry/r-novice-gapminder-es-v2018.05-1.zip\r\n",
      "https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/cwltool-61797e04db7db33486d6074ec4ecc7dadd20d5e6.zip\r\n",
      "https://zenodo.org/api/files/bf6bb00d-dd26-4a6f-8924-95a16d8ce24d/Duerfeld_Stein_2018_ID%2B_An_Interdisciplinarity_Model_Website_Code.zip\r\n",
      "https://zenodo.org/api/files/c8094de0-dd94-4284-b860-4f375c76f10b/datacarpentry/openrefine-socialsci-v2018.05-3.zip\r\n",
      "https://zenodo.org/api/files/ae20c12b-5380-496d-9569-d57c5f5bfcca/ensembleTopic-1.0.zip\r\n",
      "https://zenodo.org/api/files/8d80e007-d1a7-4634-9652-d810839251fc/Results.zip\r\n",
      "https://zenodo.org/api/files/2d2652c6-c555-43e4-aa87-223d3da93b54/swcarpentry/make-novice-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/75e63414-b867-4a08-af77-69ae4c058c27/Coupette_Juristische_Netzwerkforschung_Online-Appendix.zip\r\n",
      "https://zenodo.org/api/files/8e5381ec-98e4-4232-a816-71ffca086162/UofTCoders/rcourse-v2.1.0.zip\r\n",
      "https://zenodo.org/api/files/8c9a9852-1516-4dde-b7fe-61ebc7325ba7/trallard/ML_retreat_presentation-1.1.zip\r\n",
      "https://zenodo.org/api/files/c037e866-1fcd-4c7a-ab00-fc18061527bc/ore-0.9.0.zip\r\n",
      "https://zenodo.org/api/files/3a13c696-b652-4883-8323-00bb3c8f76a0/supplementary_material.zip\r\n",
      "https://zenodo.org/api/files/ff0c635a-6166-4159-826a-5bdcfa68bfd2/gds15-v0.9.zip\r\n",
      "https://zenodo.org/api/files/677eec6c-cbd3-4e93-823b-bde72f7e2319/labordynamicsinstitute/replicability-presentation2019-v20190328.zip\r\n",
      "https://zenodo.org/api/files/dbedfef0-2d4c-4183-8373-85887afb76ab/savmap_dataset_v2.zip\r\n",
      "https://zenodo.org/api/files/3db8c7f1-5d79-4ec0-ab3f-eff34d09e58f/2018-10-19_MD-Dislocation-Motion-in-Silicon.zip\r\n",
      "https://zenodo.org/api/files/f1dfc12c-d9ba-4eed-987d-9eeab82b48fb/pddi-cds-level1-implementation.zip\r\n",
      "https://zenodo.org/api/files/aed46454-e7fd-4615-bd88-d038c0a56e59/WG3-MetadataSpecifications-v1.zip\r\n",
      "https://zenodo.org/api/files/2d2c8700-631b-4d19-9c7b-477be6931bda/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/c562c8ba-0b51-49db-82eb-3d708c86c635/Modellbildung-und-Simulation-0.4.zip\r\n",
      "https://zenodo.org/api/files/f8fa5106-60d0-4ffb-b2e4-bb6f15a41d88/models.zip\r\n",
      "https://zenodo.org/api/files/50f7df84-b33c-4b51-919e-d46b3dbc270c/LibraryCarpentry/lc-git-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/26de068e-9f77-4e6c-bcac-b371233f36bc/annakrystalli/dataspice-tutorial-v1.0.1.zip\r\n",
      "https://zenodo.org/api/files/38c137d7-57d7-4663-9395-0efb2cbd8086/2018-11-29_HandsOn_MD-simulations-fracture.zip\r\n",
      "https://zenodo.org/api/files/984d1ff2-9f6b-4c0e-b302-883fffe72a2a/swcarpentry/r-novice-gapminder-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/29580aac-b7e8-48aa-9d5a-4390e0326924/stalkcd.zip\r\n",
      "https://zenodo.org/api/files/9747960f-21aa-43b4-a092-104b5bb7ad24/13072_2018_177_MOESM11_ESM.zip\r\n",
      "https://zenodo.org/api/files/5e90446c-d14d-4868-aff7-35de0ce82545/ck-repository-scc18-20181219.zip\r\n",
      "https://zenodo.org/api/files/9ec9ea7c-b201-4255-81e5-c017a4261d63/datacarpentry/r-raster-vector-geospatial-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/a621c8dc-57a3-4bdf-8cf0-e926d4b502c4/datacarpentry/sql-ecology-lesson-v2017.04.0.zip\r\n",
      "https://zenodo.org/api/files/d336e385-299c-4c6a-a227-5707c63dee10/5cb4ffead9323600016c4d4c.zip\r\n",
      "https://zenodo.org/api/files/e753f95f-68de-4dfe-b96e-13c50fe380e2/reza11617/finiteElement-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/eff09508-5b24-4940-99dc-a37a21bf284f/PerTract-Evaluation.zip\r\n",
      "https://zenodo.org/api/files/953dfb18-6160-457e-9c79-7f7f24068ae7/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/ec568796-c079-46f1-806d-732a1964428f/Summer%20school%20Glasgow%20June%202019.zip\r\n",
      "https://zenodo.org/api/files/3a3f28e8-cbc3-491b-86f6-9dd78c4a7f3e/JRC-PPDB-OPEN.ver0.91.zip\r\n",
      "https://zenodo.org/api/files/0b16de75-9b3c-444a-beb0-48ba44e051b7/slisovski/TheGeolocationManual-v1.0.zip\r\n",
      "https://zenodo.org/api/files/9fcf6990-4b0c-4374-81a4-578a63f7f790/AVTrack360_files.zip\r\n",
      "https://zenodo.org/api/files/94cdef82-f63d-4a71-b3fd-da7cd96ad7bf/reusabledata/reusabledata-v1.0.zip\r\n",
      "https://zenodo.org/api/files/ff5b31bd-9ee8-449d-a215-54d0353103be/datacarpentry/R-ecology-lesson-v2017.04.2.zip\r\n",
      "https://zenodo.org/api/files/7eaba6d0-fdb9-4e87-84b4-079648896aac/datacarpentry/organization-geospatial-v2018.08.1.zip\r\n",
      "https://zenodo.org/api/files/8d4231d8-f80f-4250-a901-f9c6db290715/supplemental_data.zip\r\n",
      "https://zenodo.org/api/files/c8a1859c-7c59-43ef-acc4-1a9a71fe500b/datacarpentry/geospatial-workshop-v2018.08.1.zip\r\n",
      "https://zenodo.org/api/files/5e90446c-d14d-4868-aff7-35de0ce82545/ck-snapshot-scc18-20181219.zip\r\n",
      "https://zenodo.org/api/files/b59dceca-9891-4825-baf8-1b6b1d316ded/Multiobjective_Optimization.zip\r\n",
      "https://zenodo.org/api/files/e2850bd6-838f-4d44-813e-fabf3f5a6197/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/e33b0961-f9a9-4513-92b2-8c2b14d9bc7a/gorlapraveen/The-Learning-Documentation-Project-1.0.2.zip\r\n",
      "https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/After%20sales%20repair%20management.zip\r\n",
      "https://zenodo.org/api/files/7de88097-9545-46f4-bbb1-bf28a15cdd44/datacarpentry/r-socialsci-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/8997ed4f-c473-4ed7-b1a6-f5ccb0f44f56/4.4.java-custom-rules-template.zip\r\n",
      "https://zenodo.org/api/files/6450de03-a282-4d3f-8fbf-5373d9406406/datacrate-RO-2018.zip\r\n",
      "https://zenodo.org/api/files/c62a939a-9759-4089-b1aa-c7be60d23e18/darribas/gds18-v4.1.zip\r\n",
      "https://zenodo.org/api/files/4e8b35f0-cd55-4901-8908-88185c6ad7da/asancpt/Rprogramming-v0.1.zip\r\n",
      "https://zenodo.org/api/files/07da2a8d-6de2-42ee-b860-9d607fe93261/MoMEMta/momemta.github.io-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/9100367b-fa9d-4e8c-8ece-03644c56853a/datacarpentry/shell-genomics-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/b073d1d0-6b77-4d21-8482-4fdc40c147de/rubenarslan/ovulatory_shifts-v3.0.0.zip\r\n",
      "https://zenodo.org/api/files/87955706-a7a7-477c-b563-ed7854eca077/o2r_project_website_and_blog_git-repository.zip\r\n",
      "https://zenodo.org/api/files/eddc508d-0ec3-436e-a6c6-c4ade907f64b/swcarpentry/sql-novice-survey-v2019.07.1.zip\r\n",
      "https://zenodo.org/api/files/1a8e8cde-605a-409a-87c9-2043b575e346/felixlohmeier/bibliotheks-und-archivinformatik-v1.0.zip\r\n",
      "https://zenodo.org/api/files/73e20f1f-632b-49ce-ace9-ddeb21e85041/LSDtopotools/LSDTT_documentation-v2.0.zip\r\n",
      "https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Facebook%20Social%20Media%20Campaign.zip\r\n",
      "https://zenodo.org/api/files/93af605e-e301-4bea-9bba-9490e2f06d6e/SAOImageDS9/SAOImageDS9-v8.0rc2.zip\r\n",
      "https://zenodo.org/api/files/199955ac-a752-42aa-a61d-a7fbe9298b3d/scikit-hep/uproot-3.6.1.zip\r\n",
      "https://zenodo.org/api/files/9362ca02-1b0c-412f-8376-140f826e8804/materialsvirtuallab/flamyngo-v0.9.1.zip\r\n",
      "https://zenodo.org/api/files/5ea69f7e-29e5-4013-84cb-e16a21dd925c/hls-fpga-machine-learning/hls4ml-v0.1.5.zip\r\n",
      "https://zenodo.org/api/files/149eeeef-e6c9-4b06-b10b-7451cd36742e/pydata/xarray-v0.12.1.zip\r\n",
      "https://zenodo.org/api/files/07fc92b2-165f-4422-b9df-6648959b784f/CeON/CERMINE-CERMINE-1.12.zip\r\n",
      "https://zenodo.org/api/files/791f43de-2de6-48ac-916c-722b63d82bd1/avilab/vs-v0.9.zip\r\n",
      "https://zenodo.org/api/files/9814ef17-b8f4-4581-86a6-c8c5fe2c4a51/digitallinguistics/spec-v0.10.2.zip\r\n",
      "https://zenodo.org/api/files/ff41ab02-0782-4503-8225-a24b9042a273/templateflow/python-client-0.4.1rc2.zip\r\n",
      "https://zenodo.org/api/files/d60bb9ea-ae3e-4187-a175-ec886df1a5c4/iAnnotateSV-1.0.6.zip\r\n",
      "https://zenodo.org/api/files/00ef40ed-9d26-4742-a13f-c4385f161b03/ncsa/clowder-v1.6.2.zip\r\n",
      "https://zenodo.org/api/files/3b00c223-40ac-4122-b482-af11c60ba3d3/phenomecentre/nPYc-Toolbox-v1.2.zip\r\n",
      "https://zenodo.org/api/files/f79d369a-d343-408d-acb7-1579f515e3f5/PolMine/polmineR-v0.7.11.zip\r\n",
      "https://zenodo.org/api/files/539b16a5-8cd4-4293-b160-d12a12ab3332/OpenSourceEconomics/respy-v1.2.1.zip\r\n",
      "https://zenodo.org/api/files/c602a883-63fb-4094-9191-1591464e0750/arXiv/arxiv-base-0.15.8.zip\r\n",
      "https://zenodo.org/api/files/c61975d5-d1c2-401d-86b9-81e3a4958661/ERA-URBAN/fm128_radar-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/47eb5150-e058-4a08-b669-a5f56471309d/scikit-hep/uproot-3.4.1.zip\r\n",
      "https://zenodo.org/api/files/9fe11393-41b9-4d86-8788-2e732ee86a53/IbrahimTanyalcin/taskq-2.0.7.zip\r\n",
      "https://zenodo.org/api/files/36c8bf34-74ba-4904-bad2-2a92c1b45d53/AMReX-Astro/mini-Castro-19.07.zip\r\n",
      "https://zenodo.org/api/files/36de8dd1-c0d7-4f08-ae61-824237db1eed/MosquitoAlert/Data-v1.134.2017.zip\r\n",
      "https://zenodo.org/api/files/a124ee9a-7098-4300-b6b1-84e33431b9f3/the-rocci-project/keystorm-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/445939db-5c19-4112-a126-935eea09db6e/seemoo-lab/seemoo-mobile-sensing-1.0.zip\r\n",
      "https://zenodo.org/api/files/af11f307-507b-444b-9f18-9133d7e52754/Ternary-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/307b3d6d-859f-4900-b210-1732d9c2fe78/ga4gh/dockstore-1.5.0-beta.4.zip\r\n",
      "https://zenodo.org/api/files/f03c8382-4ea0-42f5-ab51-2f29665358f6/DamCB/tyssue-0.4.0.zip\r\n",
      "https://zenodo.org/api/files/b206fa7e-23a8-4c1a-9a4c-9d8c6d696b1d/etheleon/blast2lcaPlus-0.0.1.zip\r\n",
      "https://zenodo.org/api/files/68105dfe-915b-4230-a250-5db2ddfa69b5/MosquitoAlert/Data-v1.227.2019.zip\r\n",
      "https://zenodo.org/api/files/8de6f138-8f6a-4cc1-8848-5f38c04447e9/qmflows-0.2.zip\r\n",
      "https://zenodo.org/api/files/c40dd839-42e9-4e54-9c6e-95e99ce93989/sfstoolbox/theory-3.1.1.zip\r\n",
      "https://zenodo.org/api/files/824b3d3c-c591-4741-a863-193c793aa67f/common-workflow-language/cwlprov-0.6.0.zip\r\n",
      "https://zenodo.org/api/files/8db82a82-3162-4ef6-b860-46b25121ccd5/LibCrowds/vue-pybossa-frontend-v1.0.0-beta.2.zip\r\n",
      "https://zenodo.org/api/files/61df3fa6-eed3-4017-b965-38d44bc6d2ee/giuseppelettieri/linux-pspat-v4.13_pspat-0.9.zip\r\n",
      "https://zenodo.org/api/files/df9d6b67-054c-4419-8e0a-a886caf4afe0/cmor-3.1.1.zip\r\n",
      "https://zenodo.org/api/files/2bf34cd6-be96-4095-b95d-0dd66831f1db/NHERI-SimCenter/EE-UQ-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/be0ded80-2ab4-4496-86a6-d499a51923cf/makammusicbrainz-v1.2.0.zip\r\n",
      "https://zenodo.org/api/files/9d74f082-9091-4f9d-859c-d4a5f9a357fe/tmt96/rs-object-space-0.1.zip\r\n",
      "https://zenodo.org/api/files/0a69dcae-c20b-4e32-b772-e2d2cffc0278/MDSLab/wstun-v1.0.9.zip\r\n",
      "https://zenodo.org/api/files/6d409627-3ccb-4bb2-acf4-5d0b71d9aa08/tilde-v0.7.0.zip\r\n",
      "https://zenodo.org/api/files/c10b652e-b679-4d96-ab14-88a422935194/bjodah/chempy-v0.7.0.zip\r\n",
      "https://zenodo.org/api/files/94245c46-bbe4-4a47-8f1d-46267225f29a/didillysquat/SymPortal_framework-v0.3.5.zip\r\n",
      "https://zenodo.org/api/files/7e88dd17-8b3c-44ac-8e4c-d941549a7346/hillR.zip\r\n",
      "https://zenodo.org/api/files/3f7aaa9a-45f9-4971-95f0-c960152761dc/cpmpercussion/PhaseRings-v1.2.1.zip\r\n",
      "https://zenodo.org/api/files/c3c57820-f38a-4a8a-bd32-0ef08074f7fc/puppet-gridftp-v0.2.0.zip\r\n",
      "https://zenodo.org/api/files/c7ff7bce-52d0-49ca-a4e6-6f40a9ea9c3b/gvcm/CWLY-v1.3.zip\r\n",
      "https://zenodo.org/api/files/b2d1fab6-fcfe-4514-96a5-480e0eeec39c/microbedb-microbedb-eaa1ca3f416c.zip\r\n",
      "https://zenodo.org/api/files/bb839f68-b036-402f-8986-77c4769eb13c/PerseusDL/canonical-latinLit-0.0.66.zip\r\n",
      "https://zenodo.org/api/files/e7462635-c686-47c4-adc3-4602837f0152/MosquitoAlert/Data-v1.187.2019.zip\r\n",
      "https://zenodo.org/api/files/27f0fc04-da57-4d83-99f7-4b8244c64f03/dib-lab/sourmash-v2.0.0-rc1.zip\r\n",
      "https://zenodo.org/api/files/6cc6ec93-47d7-4578-9a03-86c222318b72/Miccighel/Readersourcing-2.0-RS_Rate-v1.0.4-alpha.zip\r\n",
      "https://zenodo.org/api/files/de12719d-a155-4237-a84f-c424a70f16dd/PerseusDL/canonical-latinLit-0.0.247.zip\r\n",
      "https://zenodo.org/api/files/cfcebe14-7da9-4a07-a7c8-455b7ee0608c/geodynamics/aspect-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/ab11a2a9-094d-438a-bc35-213d6c5a4914/datacurationproject/CatanieDataLoad-v3.2.zip\r\n",
      "https://zenodo.org/api/files/f9f1795e-0d67-47b5-8e90-7611e090c9a3/MosquitoAlert/Data-v1.064.2019.zip\r\n",
      "https://zenodo.org/api/files/f8638328-341b-4bad-8be3-3980af6dbc5b/DRONE-Proj/DRONE-v1.0.zip\r\n",
      "https://zenodo.org/api/files/d495906e-801d-469e-bc62-94628dd6310d/FAIRMetrics/Metrics-v1.0.3.zip\r\n",
      "https://zenodo.org/api/files/7468fba9-a2ca-4938-a38b-ef0ef974944a/clld/clld-v4.1.1.zip\r\n",
      "https://zenodo.org/api/files/d49eaf79-1e41-450a-bef9-ad4c3db9ea1f/ctsit/linear_data_entry_workflow-2.5.1.zip\r\n",
      "https://zenodo.org/api/files/cc97afae-08e5-4f26-b1d2-2199d0238ce7/puppet-lcgdm-v0.3.2.zip\r\n",
      "https://zenodo.org/api/files/5a5b5397-9927-4e21-a9d3-28e71ce62f10/ga4gh/dockstore-1.4.0-beta.0.zip\r\n",
      "https://zenodo.org/api/files/7cb31017-718f-4858-949f-baad4563cc3d/PJK/comm-avoiding-cuts-cc-v0.1.4.zip\r\n",
      "https://zenodo.org/api/files/c84138f6-f4a0-42f7-9466-5a8874053a99/CamHD-Analysis/CamHD_motion_metadata-v0.0.3.zip\r\n",
      "https://zenodo.org/api/files/65c91d80-6c70-46b4-a80a-cc5079351af4/MosquitoAlert/Data-v1.056.2017.zip\r\n",
      "https://zenodo.org/api/files/600d2594-cd06-49ed-874c-976bd880cb53/cytoscape/py2cytoscape-0.7.0.zip\r\n",
      "https://zenodo.org/api/files/3c7673f2-bf1f-4c90-8871-bad1fb022639/manoelcampos/cloudsim-plus-v1.2.3.zip\r\n",
      "https://zenodo.org/api/files/fe49fe93-20ff-42d1-a448-c662f43eeb87/digicademy/cobj_xslt-v1.9.0.zip\r\n",
      "https://zenodo.org/api/files/acca7c36-ed4d-42a9-93bd-e10d840ebc8c/chrismheiser/lipdnet-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/dd0fb7b0-2603-4e0d-a473-92b3541d30ba/martincallaghan/text_project_1-v1.0.zip\r\n",
      "https://zenodo.org/api/files/3a9ee784-61a6-48cd-bd42-1902a3a4d958/juliasilge/tidytext-0.2.1.zip\r\n",
      "https://zenodo.org/api/files/8c46e152-e2a9-44dd-b155-ada3b61de625/pymatgen-v3.3.4.zip\r\n",
      "https://zenodo.org/api/files/ca43e0ec-40f9-4ef2-9f4c-62598557c9af/clulab/eidos-Hackathon-NewMap.zip\r\n",
      "https://zenodo.org/api/files/2c9e4916-9f0f-458a-af37-90bdb9d8961a/MPAS-Dev/MPAS-Model-v6.3.zip\r\n",
      "https://zenodo.org/api/files/a3176cb7-6196-436f-b581-9d0c65971957/Mambu38/CALOA-v0.23.0.2.zip\r\n",
      "https://zenodo.org/api/files/99176ad9-52fd-4a53-a2f2-9608e7dde936/MosquitoAlert/Data-v1.358.2018.zip\r\n",
      "https://zenodo.org/api/files/28106d59-a203-4832-a06f-25141a64a8e5/Capitains/Sparrow-1.2.2.zip\r\n",
      "https://zenodo.org/api/files/7745a133-7c4b-422d-9a3a-410764f0c835/nomisr-0.2.0.zip\r\n",
      "https://zenodo.org/api/files/876e9935-a192-458d-bf24-f56c99ac4c0a/VariantEffect/mavedb-v1.4.1-alpha.zip\r\n",
      "https://zenodo.org/api/files/c2b228f3-7846-40d0-99b2-0992f7b898ce/cggh/panoptes-2.0.beta3.zip\r\n",
      "https://zenodo.org/api/files/74b5ba67-653d-4e7e-99b8-ed55989b2123/materialsproject/pymatgen-v2018.4.20.zip\r\n",
      "https://zenodo.org/api/files/943531dc-34a8-4143-bfe8-c48367aaf60a/cytoscape/cytoscape.js-v3.9.1.zip\r\n",
      "https://zenodo.org/api/files/0dd09d1b-8309-419d-a30e-cf2695da1b3a/kipoi/models-v2019-03-17.zip\r\n",
      "https://zenodo.org/api/files/45b1b0f5-d3e5-4d37-96a8-0ac0bb585489/jhpoelen/eol-globi-data-v0.12.4.zip\r\n",
      "https://zenodo.org/api/files/bb3bc232-2cd5-415d-a2e7-c36271bfd767/globalbioticinteractions/hechinger2011-v0.1.zip\r\n",
      "https://zenodo.org/api/files/9b54d1b7-4501-4d13-a06b-d768554e1005/higlass/higlass-v1.6.2.zip\r\n",
      "https://zenodo.org/api/files/843e382f-4777-43c7-a298-90bdcaa14ff9/lmfit/lmfit-py-0.9.11.zip\r\n",
      "https://zenodo.org/api/files/bb184f3b-e765-4407-a5f0-81cd9a0b4ac3/justinlittman/TweetSets-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/32b2fd80-e813-4493-ac75-98412a1bc060/simX-master.zip\r\n",
      "https://zenodo.org/api/files/09120dc6-8675-4495-88b0-fc60c08d4200/scikit-hep/uproot-3.8.0.zip\r\n",
      "https://zenodo.org/api/files/0407706f-29de-492f-b03e-b03710fb3919/kipoi/models-v2019-06-09.zip\r\n",
      "https://zenodo.org/api/files/298d4070-7b04-45ae-b4e9-4ec6f797cd7a/okgreece/Alignment-v1.0.zip\r\n",
      "https://zenodo.org/api/files/38304926-edaf-462f-afa8-88f2b1c7a995/globalbioticinteractions/gray2015-v0.2.zip\r\n",
      "https://zenodo.org/api/files/aa8359d4-05e5-426e-b257-1cbdab4be6fe/Capitains/Hook-1.0.1.zip\r\n",
      "https://zenodo.org/api/files/882b42c3-21fc-4cad-81e0-1722c253fe03/ricklupton/d3-sankey-diagram-v0.4.1.zip\r\n",
      "https://zenodo.org/api/files/c5965d56-bbf8-427d-8a12-bd1bc4edf764/flav-io/flavio-v0.21.1.zip\r\n",
      "https://zenodo.org/api/files/73ac6b3f-199c-41a6-bcad-fc669aaf609d/clics/clics2-v0.1.zip\r\n",
      "https://zenodo.org/api/files/6e3b1dbb-f82b-433a-ad28-89a22c85618f/nodefony/nodefony-1.0.2-beta.zip\r\n",
      "https://zenodo.org/api/files/7a060b80-130d-47e9-a8ef-6af26cbe5e58/MosquitoAlert/Data-v1.327.2018.zip\r\n",
      "https://zenodo.org/api/files/368b0152-ccb1-4b36-b264-a878ef73b273/jfy133/MEx-IPA-0.4.4.zip\r\n",
      "https://zenodo.org/api/files/c91ccb36-9421-408b-95db-5b40b7f5d945/oscarbranson/latools-0.3.8.zip\r\n",
      "https://zenodo.org/api/files/fb9ed0e7-2f7c-4127-aec0-2e98a5080f6a/ga4gh/dockstore-1.4.0-alpha.7.zip\r\n",
      "https://zenodo.org/api/files/fc09d04f-f26f-440d-922d-f990d34d0c4f/rOCCI-server-v1.1.2.zip\r\n",
      "https://zenodo.org/api/files/ce6c8f6b-538b-4980-8704-388ae84c9ea6/GoMexSI/baremore-0.1.zip\r\n",
      "https://zenodo.org/api/files/c332eccc-0e18-4160-9b33-320096cc73e5/MosquitoAlert/Data-v1.137.2018.zip\r\n",
      "https://zenodo.org/api/files/90db1dc9-d0f0-403d-bc1f-dab500ead43d/flavio-v0.6.0.zip\r\n",
      "https://zenodo.org/api/files/f5b1bc88-f42b-455f-a443-ae8719a598e6/MosquitoAlert/Data-v1.289.2016.zip\r\n",
      "https://zenodo.org/api/files/c0b4775c-ed94-435d-91b0-e5856e62112d/sasview-v3.3.1.zip\r\n",
      "https://zenodo.org/api/files/fc4f606f-f985-4edf-afd9-15fbae03db37/MobleyLab/FreeSolv-v0.5.zip\r\n",
      "https://zenodo.org/api/files/671d183a-187e-4775-bf72-b49f7ccb742e/materialsproject/pymatgen-v2017.9.1.zip\r\n",
      "https://zenodo.org/api/files/9a1a750d-1104-440b-862b-cc3c2f4470a4/Zhong-Lab-UCSD/Genomic-Interactive-Visualization-Engine-v0.1.1.zip\r\n",
      "https://zenodo.org/api/files/a99c1879-28c3-46ed-8443-f17746591d90/MosquitoAlert/Data-v1.339.2016.zip\r\n",
      "https://zenodo.org/api/files/452834ee-f898-47a8-a733-cd0e1f7e7a30/materialsproject/fireworks-v1.7.5.zip\r\n",
      "https://zenodo.org/api/files/ad311bf7-9806-4fb6-a4bc-59ccdd44d24c/MOG-Public-2.31.zip\r\n",
      "https://zenodo.org/api/files/b26dfa4e-7cc6-4101-83d1-380dc198e5cb/matthiaskoenig/pkdb-v0.4.0.zip\r\n",
      "https://zenodo.org/api/files/9cb12f86-1a62-4005-af60-65f4dcbdafb9/materialsproject/fireworks-v1.3.8.zip\r\n",
      "https://zenodo.org/api/files/0ce08862-1075-4201-9420-b30a863de7ff/avilab/vs-v0.99.3.zip\r\n",
      "https://zenodo.org/api/files/0b90a541-1363-43e9-b8d1-805ac4258b73/MetSCalc/metscalc-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/6d1d6c86-e1e0-49a9-9acf-206178764ab4/pymatgen-v3.3.1.zip\r\n",
      "https://zenodo.org/api/files/b1537f03-4353-4652-9537-ce31b67ee6a7/galantelab/sandy-v0.22.zip\r\n",
      "https://zenodo.org/api/files/fe570da3-f17c-421a-83c8-832ec198d005/CESNET/proxystatistics-simplesamlphp-module-v1.5.0.zip\r\n",
      "https://zenodo.org/api/files/77aec9d0-a877-4c03-aff1-8142048cb047/pymatgen-v3.3.0.zip\r\n",
      "https://zenodo.org/api/files/8fb3691f-d64f-48eb-afc7-0d973fc4a4cc/Miccighel/Readersourcing-2.0-RS_Rate-v1.0.5-alpha.zip\r\n",
      "https://zenodo.org/api/files/01727b2b-b3af-43e9-a5fa-e11e9d2d23af/Uko/Renraku-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/4cdebeaa-7958-4ace-9846-6b7f0f31e61c/nismod/ukpopulation-1.0.2.zip\r\n",
      "https://zenodo.org/api/files/bf2bdcd1-f829-45b5-93bc-ba08dd2ad22d/PerseusDL/canonical-greekLit-0.0.1379.zip\r\n",
      "https://zenodo.org/api/files/1a08b550-a68e-485a-b77a-d448e489366c/NLeSC/spot-desktop-app-v0.2.0-beta.zip\r\n",
      "https://zenodo.org/api/files/0b10699b-6f20-4f86-985e-d6c692840b72/sotorrent/db-scripts-sotorrent_2018-06-17.zip\r\n",
      "https://zenodo.org/api/files/9b10b2a7-4a19-4857-ab18-cfee82a1a207/MosquitoAlert/Data-v1.305.2017.zip\r\n",
      "https://zenodo.org/api/files/9c293f61-3fd0-400a-a6be-28aa05b14b63/TactileUniverse/3D-Printed-Galaxy-Software-V3.3.zip\r\n",
      "https://zenodo.org/api/files/adfdee2f-8b29-464a-8c6b-832d74086a6d/ga4gh/dockstore-ui-1.1-rc.1.zip\r\n",
      "https://zenodo.org/api/files/ccae7187-3853-454f-b840-5d70bdd484f9/SAOImageDS9/SAOImageDS9-v8.0rc.zip\r\n",
      "https://zenodo.org/api/files/c7056e3f-7884-40d9-a92a-5d0856ac5eca/MosquitoAlert/Data-v1.275.2016.zip\r\n",
      "https://zenodo.org/api/files/8dcdebd4-cc35-4c9f-8af8-b1dcd033909f/seqware-1.2.0-alpha.0.zip\r\n",
      "https://zenodo.org/api/files/6b96baef-2a39-4ce0-be31-50c1a92abe95/brianjimenez/lightdock-0.5.5.zip\r\n",
      "https://zenodo.org/api/files/9a88afd8-1614-4284-a7f7-52fb85b77f6d/materialsproject/pymatgen-v2019.5.1.zip\r\n",
      "https://zenodo.org/api/files/fb894ce9-4fef-402a-8d8a-969cca580632/datacarpentry/R-ecology-lesson-v2019.06.1.zip\r\n",
      "https://zenodo.org/api/files/8c0d30ed-ca0e-46b4-bf74-beace6869deb/projectAndvariLOD-v1.0.zip\r\n",
      "https://zenodo.org/api/files/8e31d476-f183-4f39-994c-d925b1dddc67/RinSim-v3.2.0.zip\r\n",
      "https://zenodo.org/api/files/f51bc320-1c40-495f-b39c-67f6d639026e/jdidion/xphyle-4.1.0.zip\r\n",
      "https://zenodo.org/api/files/1026f374-8ee4-4c78-a9dd-870e82bb4045/clld-v0.25.zip\r\n",
      "https://zenodo.org/api/files/8c57aaea-23cb-405c-9e52-e4124a475b91/MosquitoAlert/Data-v1.354.2016.zip\r\n",
      "https://zenodo.org/api/files/b05c9bff-de8b-4f09-84ef-39adeedba219/MosquitoAlert/Data-v1.079.2017.zip\r\n",
      "https://zenodo.org/api/files/8a3db2cc-1dd9-44dd-823d-15fcc0e4a647/sebastiansauer/data_se-v1.0.zip\r\n",
      "https://zenodo.org/api/files/1730ed80-3c81-4bf1-8612-48edfae88325/samapriya/Planet-Pipeline-v1.0.zip\r\n",
      "https://zenodo.org/api/files/88e6a846-b252-46a3-be6e-76f07e30bfda/dockstore/dockstore-ui2-2.3.0-rc.4.zip\r\n",
      "https://zenodo.org/api/files/0db670aa-3384-443e-b453-0e54cb965bce/JedS6391/LGP-5.0.zip\r\n",
      "https://zenodo.org/api/files/2e78f02d-cf4a-4fb1-b430-1efb00033343/poldracklab/fmriprep-1.3.0.zip\r\n",
      "https://zenodo.org/api/files/7e4e75dc-73c4-4260-a1d6-a5cf876645ef/rnabioco/valr-v0.5.0.zip\r\n",
      "https://zenodo.org/api/files/1ad54d0c-c9a8-4a07-9b8f-e21f3bc1c6fe/aleneum/kognijs-animate-v0.2.5.zip\r\n",
      "https://zenodo.org/api/files/5fc4cd18-484d-461c-80c6-baef7ae76f54/gorescript-v1.1.5.zip\r\n",
      "https://zenodo.org/api/files/afa625af-100a-48f8-86d2-acd2a60015fb/sbpy-0.2.zip\r\n",
      "https://zenodo.org/api/files/9240ec16-b36f-4fd2-af23-0538f0931c1a/shogun-shogun_4.1.0.zip\r\n",
      "https://zenodo.org/api/files/cac3e1eb-593a-4a29-8a4d-960042b742b4/bety-betydb_4.8.zip\r\n",
      "https://zenodo.org/api/files/a15e3a9e-daa8-4fb5-a445-c1a11be61ee6/MosquitoAlert/Data-v1.029.2018.zip\r\n",
      "https://zenodo.org/api/files/795b2367-70ee-453f-889f-f938930fa0da/bmn-0.1.1.zip\r\n",
      "https://zenodo.org/api/files/b6459b2d-9f98-4310-93c2-1a5aa69ff3bc/citation-file-format/cff-converter-python-0.0.5.zip\r\n",
      "https://zenodo.org/api/files/86946453-358c-433a-a963-dc6a3a864190/GarmanGroup/RADDOSE-3D-v3.0.794.zip\r\n",
      "https://zenodo.org/api/files/f6be33a6-e06f-4fc0-8545-2e3c5d805d99/ecell/ecell4_base-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/8c1af7a3-80c7-40bd-86d1-488487f697ed/failys/cairis-1.6.4.zip\r\n",
      "https://zenodo.org/api/files/11bf861c-0342-4cdb-b588-26157fbd462b/jakelever/cancermine-v1.0.1.zip\r\n",
      "https://zenodo.org/api/files/2c6296d2-44b2-41e4-8fc7-6dd16a6a8b67/Molmed/checkQC-v3.zip\r\n",
      "https://zenodo.org/api/files/4201e683-b0a6-4eba-ad7f-a4bc154de594/inventionate/Synthesise-v2.3.11.zip\r\n",
      "https://zenodo.org/api/files/c4bf6fe0-5f1e-4cec-b66d-bce8f79bb0e3/RubenVerborgh/SPARQL.js-v2.0.1.zip\r\n",
      "https://zenodo.org/api/files/a8b921fa-d518-4e6d-aaf0-7df9b8f6b01b/proycon/clam-v2.3.6.zip\r\n",
      "https://zenodo.org/api/files/ccf39fa3-02d1-412c-9b57-415a789b95cd/Planeshifter/isle-editor-v0.19.1.zip\r\n",
      "https://zenodo.org/api/files/f7597793-b9c9-4215-9d82-3ff4c4123128/Nino-cunei/oldbabylonian-v1.4.zip\r\n",
      "https://zenodo.org/api/files/f6aaa92a-9d20-4f14-858a-9ccb8c6132f7/MosquitoAlert/Data-v1.309.2016.zip\r\n",
      "https://zenodo.org/api/files/42a62a87-87b3-448d-bc88-a7b9b516b546/INCF/BIDS-Starter-Kit-v1.0.2.zip\r\n",
      "https://zenodo.org/api/files/3c2ac657-e3af-4f6c-9f66-5dd18167a5e3/KHP-Informatics/HipDynamics-v1.0.zip\r\n",
      "https://zenodo.org/api/files/01076c10-1234-44cc-b637-44206b4a3952/abntex/limarka-v1.0.1.zip\r\n",
      "https://zenodo.org/api/files/a0bb3dba-ec0e-4b15-8ad1-109781826274/camillescott/shmlast-v1.3.zip\r\n",
      "https://zenodo.org/api/files/d80f7014-0f87-48a0-97da-9882f90ccad3/ctsit/form_render_skip_logic-3.3.1.zip\r\n",
      "https://zenodo.org/api/files/19bb245b-1c79-43b2-b766-10c002f4baf8/guaix-ucm/megaradrp-v0.6.1.zip\r\n",
      "https://zenodo.org/api/files/2273df17-3c85-4ff3-a072-46b72a908485/MosquitoAlert/Data-v1.021.2019.zip\r\n",
      "https://zenodo.org/api/files/727d3169-8319-4384-8b5d-d0474aec9dc4/plasticityai/magnitude-0.1.59.zip\r\n",
      "https://zenodo.org/api/files/f0144220-9255-4d55-9467-1bcc3f5c6832/PerseusDL/canonical-greekLit-0.0.1407.zip\r\n",
      "https://zenodo.org/api/files/3af2459f-74ab-4688-9668-5209b5430134/EC-Earth/ece2cmor3-0.9.0.zip\r\n",
      "https://zenodo.org/api/files/35c80a21-beff-4d2c-80c3-206a2922e6d6/ailiop/idvf-1.0.3b.zip\r\n",
      "https://zenodo.org/api/files/c7075429-6376-4f8a-9f44-98a47b766d17/eweitz/ideogram-v0.6.0.zip\r\n",
      "https://zenodo.org/api/files/88cd0dc3-6e2f-42c4-9cc9-b051438b0e58/cytoscape/cytoscape.js-v3.8.2.zip\r\n",
      "https://zenodo.org/api/files/5187ffbd-4ddb-40b6-8b33-c364e5055cf5/ctmixtures-v2.3.zip\r\n",
      "https://zenodo.org/api/files/696116d0-9901-4371-90b6-2a29691f239e/BD2KGenomics/dcc-action-service-1.0.9.zip\r\n",
      "https://zenodo.org/api/files/851ae981-aab4-49ff-a5ef-444b67719f68/Starfish-0.2.1.zip\r\n",
      "https://zenodo.org/api/files/152159bd-933e-49a3-b035-b19d34187d73/africanmathsinitiative/R-Instat-v0.4.14.zip\r\n",
      "https://zenodo.org/api/files/9a700645-f032-4b65-8eb3-27d3d6c3db2b/CClusTera-v1.0.zip\r\n",
      "https://zenodo.org/api/files/fe15f20c-a6ca-42a5-9427-5fe479f68be6/pymatgen-v3.3.0.zip\r\n",
      "https://zenodo.org/api/files/d4cec331-ba63-44d3-aa19-40995ca1ce9f/yeatmanlab/AFQ-Browser-MSexample-paper.zip\r\n",
      "https://zenodo.org/api/files/bb4611f4-1399-4e85-bf77-05cf27518291/cern-it-sdc-id/puppet-dpm-v0.5.2.zip\r\n",
      "https://zenodo.org/api/files/87850acc-e6fb-4168-be79-13cceb1dd03a/ecell/ecell4_base-test3.zip\r\n",
      "https://zenodo.org/api/files/22852208-f2b1-4e9c-bd64-daa839d51c93/edoddridge/aronnax-0.3.0.zip\r\n",
      "https://zenodo.org/api/files/0bc15222-66a7-4a77-b31b-5d45ecd62972/dockstore/dockstore-1.7.0-beta.1.zip\r\n",
      "https://zenodo.org/api/files/7e80d519-b29a-4386-94ba-81a17bb1eb1b/nf-core/hlatyping-1.1.3.zip\r\n",
      "https://zenodo.org/api/files/f9f4e6ac-5bff-4b4e-b487-e72d0b9c9556/gwu-libraries/sfm-weibo-harvester-1.6.0.zip\r\n",
      "https://zenodo.org/api/files/52e55e65-702f-4a82-8c5b-7220c57f85b1/materialsproject/fireworks-v1.6.9.zip\r\n",
      "https://zenodo.org/api/files/9aafdae4-96b7-47d2-8b2b-4e75bd059610/scikit-hep/uproot-3.4.9.zip\r\n",
      "https://zenodo.org/api/files/bf76330c-263d-4f38-a3ac-3aa50935c96d/MRN-Code/coinstac-v4.2.3.zip\r\n",
      "https://zenodo.org/api/files/7751c494-0d46-4f6d-a021-9e41551b33ec/geodynamics/pylith-v2.2.2.zip\r\n",
      "https://zenodo.org/api/files/3a036565-8b42-458a-b5bb-6607adb5041a/PerseusDL/canonical-greekLit-0.0.380.zip\r\n",
      "https://zenodo.org/api/files/8f156cab-f6b9-438e-bc5f-d7d940a7f93c/heileman/CurricularAnalytics.jl-v0.2.5.zip\r\n",
      "https://zenodo.org/api/files/04464d8e-8f5a-4a20-bf31-a700922b7733/elliothe/CVPR_2019_PNI-v1.0.zip\r\n",
      "https://zenodo.org/api/files/aaabdfda-c0e6-4c76-8233-d6385f6929e7/cmssw-CMSSW_7_5_1.zip\r\n",
      "https://zenodo.org/api/files/fedf570f-0208-454f-8293-298ee6717579/BristolTopGroup/DailyPythonScripts-pre-python-package.zip\r\n",
      "https://zenodo.org/api/files/b8fe9121-00b5-4997-85d6-f174e283e8be/mikldk/malan-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/3cfa903e-410e-407a-aeef-c393477a14c2/IndrajeetPatil/ggstatsplot-0.0.9.zip\r\n",
      "https://zenodo.org/api/files/3364f0bf-47fe-4683-a5a5-a78d31c38a79/ChihyingDeng/MIMIC_CXR_multilabel-v1.0.zip\r\n",
      "https://zenodo.org/api/files/316a6928-8f79-4474-94c3-28aa6ee83aba/arrgh-1.0.1.zip\r\n",
      "https://zenodo.org/api/files/3df0e24a-ef7e-4564-83dc-bbfdb48272eb/merliseclyde/BAS-v1.5.2.zip\r\n",
      "https://zenodo.org/api/files/72303e03-23a8-4bbc-95a8-ffa90827be0b/pybel/pybel-v0.9.0-dev.zip\r\n",
      "https://zenodo.org/api/files/74881f01-3a3e-48f6-9902-deeda46d5deb/symbtrdataextractor-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/14b9e2d1-dfd5-41e0-af92-b37ad4dc0c6e/lexibank/sagartst-v1.1.0.zip\r\n",
      "https://zenodo.org/api/files/f30f4922-ed60-4f19-a7cd-ce118847d6c7/MosquitoAlert/Data-v1.342.2016.zip\r\n",
      "https://zenodo.org/api/files/28d11d59-3939-459d-9372-47b988926c5c/jhpoelen/eol-globi-data-v0.9.18.zip\r\n",
      "https://zenodo.org/api/files/27cec760-140a-484d-8a60-223f4c67e20d/PerseusDL/canonical-greekLit-0.0.1232.zip\r\n",
      "https://zenodo.org/api/files/07b3b2de-7489-4669-9deb-da33a6349dd4/johnsamuelwrites/wdprop-v0.3.zip\r\n",
      "https://zenodo.org/api/files/869e99d6-e4c9-4fe8-9712-fdec0675f169/thomaskrause/graphANNIS-v0.1.0.zip\r\n",
      "https://zenodo.org/api/files/bd837d8b-f947-48c7-8d0e-120ba6b280fc/xenon-middleware/xenon-3.0.0.zip\r\n",
      "https://zenodo.org/api/files/72e38edd-3ac0-445a-9784-780a4186a5cc/oborel/obo-relations-v2018-07-29.zip\r\n",
      "https://zenodo.org/api/files/091d0b8a-2ba0-4d67-a167-ac3856c46cdb/sidbdri/LAB-AID-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/303346ee-549f-4651-9132-a6bef34870f4/sylabs/singularity-v3.3.0-rc.4.zip\r\n",
      "https://zenodo.org/api/files/ca5087a7-0e74-400c-adb7-ff303ada5bc8/sola-da/NL2Type-icse2019artifact.zip\r\n",
      "https://zenodo.org/api/files/6f7a8ee3-fb0a-4ce8-9a86-1d37ffa44291/aaalvik/visast-frontend-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/c86c0bd9-7c85-4559-aad6-4e955787605a/andraus/arbor-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/f334ccb4-3f55-4205-89bc-04a5e27d4fcf/MosquitoAlert/Data-v1.334.2017.zip\r\n",
      "https://zenodo.org/api/files/31efea50-8ba8-4e34-8426-782568ece79e/rundle-research-group/TsunamiSquares-xx-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/3d44bc87-6252-4375-9f1f-c4078594681b/hammerlab/ketrew-ketrew.3.2.1.zip\r\n",
      "https://zenodo.org/api/files/5338a25f-96bd-411b-9a2a-15f48d12342e/MosquitoAlert/Data-v1.184.2019.zip\r\n",
      "https://zenodo.org/api/files/39f56755-5cdf-4144-a9e8-0f3642fe3e5c/the-rocci-project/rOCCI-server-v2.0.2.zip\r\n",
      "https://zenodo.org/api/files/b4413412-6a02-4a4c-9c60-9762df9fc477/MosquitoAlert/Data-v1.008.2017.zip\r\n",
      "https://zenodo.org/api/files/7089dd52-0be9-403f-8efd-d861ab21ccd1/CELavecchia/LMG-v1.1.zip\r\n",
      "https://zenodo.org/api/files/fcb44be4-22f1-4ee2-9a5e-20aaaa45bca9/pepper-grinder-0.1.8-macosx.cocoa.x86_64.zip\r\n",
      "https://zenodo.org/api/files/bda9c2a3-3097-4b3a-98c5-5b80f82d059b/MosquitoAlert/Data-v1.196.2019.zip\r\n",
      "https://zenodo.org/api/files/6b30e50d-fb83-46b8-9fb5-aa7c9324c226/cfe-lab/Kive-v0.14.zip\r\n",
      "https://zenodo.org/api/files/a58dbeb7-2239-43b2-91cb-b0b79f3ad66a/FluVigilanciaBR/fludashboard-v0.4.0.zip\r\n",
      "https://zenodo.org/api/files/503139a4-053f-4f25-9d0a-a80d3563b076/openmalaria-schema-34-niggi-test-1.zip\r\n",
      "https://zenodo.org/api/files/0afeb8af-29d0-4684-9680-a080ad759d0c/kisssg/bbs-v1.0-beta.zip\r\n",
      "https://zenodo.org/api/files/60c0d171-0801-4916-891a-f959dc5d5a76/IRC-SPHERE/HyperStream-v0.2.0-alpha.zip\r\n",
      "https://zenodo.org/api/files/cf100eab-eba3-4f3c-ae98-1c765a2a440c/boutiques/boutiques-0.5.3.zip\r\n",
      "https://zenodo.org/api/files/f74e55ce-9d90-4f80-a9cd-1dff6a0fbb1d/MosquitoAlert/Data-v1.185.2019.zip\r\n",
      "https://zenodo.org/api/files/c003022e-91a4-45c4-a8fc-705fd1f67db1/geneticsMiNIng/cr17-ver0.1.zip\r\n",
      "https://zenodo.org/api/files/e279cf91-5aa2-4719-a318-9e386842d4de/richardjgowers/GCMCworkflow-v0.5.0.zip\r\n",
      "https://zenodo.org/api/files/8a331f45-52da-491f-bb24-67dc7ea26a67/geodynamics/sw4-v2.0.zip\r\n",
      "https://zenodo.org/api/files/d14676b6-cd34-47f3-b92e-260a7c5634a7/BD2KGenomics/dcc-metadata-indexer-2.0.8.zip\r\n",
      "https://zenodo.org/api/files/15b4d749-393b-4066-9d22-1c86eb459f40/onecodex/finch-rs-v0.1.6.zip\r\n",
      "https://zenodo.org/api/files/ca5ab067-9ec8-4a88-ac2b-34b994d5f11b/openmalaria-schema-34-vivax-bloodstage-update-1.zip\r\n",
      "https://zenodo.org/api/files/a4cb3469-58c4-4594-b42f-e5e9e61a0b80/scikit-hep/uproot-3.4.18.zip\r\n",
      "https://zenodo.org/api/files/03944b9f-ed8b-4be1-b89f-4124f75da26a/project-aero/reporting-process-v2.0.zip\r\n",
      "https://zenodo.org/api/files/2f2332cb-7e6f-477d-8790-4d87de322cd5/connorcoley/ASKCOS-v0.2.9.zip\r\n",
      "https://zenodo.org/api/files/7e2c90ac-66ee-49d2-8b11-4cb83dac3e95/astroquery-v0.2.4.zip\r\n",
      "https://zenodo.org/api/files/f1abcaf4-90a4-41da-a393-df7857265ce4/Molmed/checkQC-v1.6.1.zip\r\n",
      "https://zenodo.org/api/files/5f8dbe96-d1b3-40b3-b26f-3f579783f54e/pyoceans/gridgeo-v1.5.0.zip\r\n",
      "https://zenodo.org/api/files/41888698-8654-4933-add7-a7489f243845/easystats/bayestestR-0.0.1.zip\r\n",
      "https://zenodo.org/api/files/1fee57a8-aa27-485c-b3bf-4fe06c9b4c1d/ag-gipp/MathMLTools-2.0.2.zip\r\n",
      "https://zenodo.org/api/files/016763aa-d745-4e34-8083-fba286d036f6/atmtools/konrad-v0.6.6.zip\r\n",
      "https://zenodo.org/api/files/b90768ac-965f-4e88-aa45-fed1ddc3c454/naturalness/sensibility-saner2018-submission.zip\r\n",
      "https://zenodo.org/api/files/91376ece-e39e-4385-b02a-3b5879635e06/weecology/MATSS-v0.1.1.zip\r\n",
      "https://zenodo.org/api/files/62760627-3ffc-4a64-be17-ee1cb46a5948/poldracklab/smriprep-0.2.0a1.zip\r\n",
      "https://zenodo.org/api/files/9c5bdd62-84ca-49ee-94a2-6d46dd4f3763/MosquitoAlert/Data-v1.189.2018.zip\r\n",
      "https://zenodo.org/api/files/f0345841-ed73-408b-8485-78062976ee3f/MosquitoAlert/Data-v1.303.2016.zip\r\n",
      "https://zenodo.org/api/files/de43b96e-2240-49b3-a50e-9bbd9799dd67/MosquitoAlert/Data-v1.046.2019.zip\r\n",
      "https://zenodo.org/api/files/6d2ff49e-80bc-4333-85c9-fc990ab5f7ad/kipoi/kipoi-v0.6.7.zip\r\n",
      "https://zenodo.org/api/files/7730f141-65d5-46bd-b3c4-48ed15542b5f/scriptotek/emnesok-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/cfdab9de-b517-4178-bb0b-35ce47dfc4e7/dib-lab/sourmash-v1.0.zip\r\n",
      "https://zenodo.org/api/files/1fc0ac1e-0e74-4eac-984a-c1b0a8135bdf/fireworks-v1.1.4.zip\r\n",
      "https://zenodo.org/api/files/b96e8633-4222-41f3-872c-c16fad47f0dc/tjespel/barrier-and-look-back-options-v1.0.zip\r\n",
      "https://zenodo.org/api/files/1375f189-8850-4c6d-8efa-849b0b2ba929/MosquitoAlert/Data-v1.341.2018.zip\r\n",
      "https://zenodo.org/api/files/6d3045ce-d5ab-43c3-8904-7bb3c85ccd14/University-of-Potsdam-MM/Reflect.UP-v3.4.1.zip\r\n",
      "https://zenodo.org/api/files/624c6f89-7e00-4d68-933b-891593a4dc7f/JavaSimulation.zip\r\n",
      "https://zenodo.org/api/files/f5705125-d6df-4ef7-8eba-3c9fbe43db8f/biosustain/optlang-1.0.2.zip\r\n",
      "https://zenodo.org/api/files/9199c8f0-30af-457b-a846-3d58e76c523b/molbiodiv/fennec-v0.7.2.zip\r\n",
      "https://zenodo.org/api/files/c438d53f-2a1e-4d95-8c6e-bfad5267fed9/materialsproject/pymatgen-v4.4.2.zip\r\n",
      "https://zenodo.org/api/files/ed87585c-03a9-4387-b6a2-8a4ca0e918e9/PerseusDL/canonical-latinLit-0.0.107.zip\r\n",
      "https://zenodo.org/api/files/bada7590-514e-465a-a376-a8dcd48f7ffc/jasonclark/linked-people-bioscience-v1.0.zip\r\n",
      "https://zenodo.org/api/files/6f368be5-0439-4937-81a0-c1028fd43e76/guaix-ucm/pyemir-v0.12.zip\r\n",
      "https://zenodo.org/api/files/5017eaaf-4fa5-4b9a-8181-d9ef13fa9775/cytoscape/cytoscape.js-v2.7.25.zip\r\n",
      "https://zenodo.org/api/files/aedf1d19-4e37-419b-8ae3-fdb26dc74185/OpenCLSim-0.3.0.zip\r\n",
      "https://zenodo.org/api/files/b30785bd-3d76-49f1-a4a8-d482211ecc87/samaaron/sonic-pi-v2.11.1.zip\r\n",
      "https://zenodo.org/api/files/0eb4424e-23f3-4efd-8b9f-084578b60811/proycon/clam-v2.4.9.zip\r\n",
      "https://zenodo.org/api/files/9745c7ca-328b-4447-b5f9-09afb0d8284c/ncsa/clowder-v1.4.0.zip\r\n",
      "https://zenodo.org/api/files/27b66688-d317-4f31-a9a6-20ef787c7c72/PerseusDL/canonical-greekLit-0.0.1372.zip\r\n",
      "https://zenodo.org/api/files/97928393-fbed-4e3d-bf7e-4263a043b2b8/bocinsky/Bocinsky_Varien_2017-v1.0.zip\r\n",
      "https://zenodo.org/api/files/cd3fcd83-fee8-40e3-a0d5-6f215b5d395c/html.zip\r\n",
      "https://zenodo.org/api/files/44153bb6-75d8-4c2f-80a1-bb8e0c22014e/Guindillator/BioCertainty-v1.0.4.zip\r\n",
      "https://zenodo.org/api/files/7f042e52-1e1f-4640-9bc5-ce2f52bfe07d/ericmandel/js9-v2.3.zip\r\n",
      "https://zenodo.org/api/files/fc1f2458-82cd-4577-a101-27ccab8dc9db/ttankayev/Yemen-cholera-simulation-1.0.zip\r\n",
      "https://zenodo.org/api/files/c5087f12-c4fd-45c1-b64a-7b3d3cf2bb4a/bjodah/chempy-v0.7.7.zip\r\n",
      "https://zenodo.org/api/files/634b258d-8235-4a7f-a401-d1bebe8c4e16/ropensci/beastier-v2.0.7.zip\r\n",
      "https://zenodo.org/api/files/d902138a-a9bd-4bdf-8a62-6fc2d0044aba/higlass/higlass-v1.3.0-rc.2.zip\r\n",
      "https://zenodo.org/api/files/a80fd997-cc3b-46dd-a1a3-251c6bdf8e1a/flamyngo-v0.7.3.zip\r\n",
      "https://zenodo.org/api/files/fcedd3b4-f656-471e-9dae-ce1f28e5774b/ga4gh/tool-registry-service-schemas-2.0.0-beta.1.zip\r\n",
      "https://zenodo.org/api/files/988b51c0-ac5b-4305-ad40-5d99f054959d/cern-it-sdc-id/puppet-dmlite-v0.6.0.zip\r\n",
      "https://zenodo.org/api/files/aa13c654-d7d1-43b8-8403-7b9c9715ae2b/GerardBalaoro/jQuery-Tourer-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/bb9446fc-4e3d-4131-a18c-e3763be0d38c/meyda-v3.0.0.zip\r\n",
      "https://zenodo.org/api/files/c8e59574-8d30-4aed-95d0-288da6f5f3b6/osirrc/jig-v0.1.1.zip\r\n",
      "https://zenodo.org/api/files/51d91175-7af4-433f-9ebd-df2688a50132/SMTG-UCL/galore-0.5.1.zip\r\n",
      "https://zenodo.org/api/files/b6574190-b42d-4870-be96-0eeb09399ea4/materialsvirtuallab/flamyngo-v0.8.1.zip\r\n",
      "https://zenodo.org/api/files/3ac50aa1-be58-4e9b-95c7-746be7faaf5e/adamhsparks/GSODRdata-0.2.12.zip\r\n",
      "https://zenodo.org/api/files/f2ba87e1-1660-4a0a-b022-c0a2af3ba9d4/MosquitoAlert/Data-v1.070.2017.zip\r\n",
      "https://zenodo.org/api/files/9e0cc582-9b56-47fa-ba7e-1db8485db3b6/toniher/seqservice-0.1.1.zip\r\n",
      "https://zenodo.org/api/files/dbef5ff9-da9a-40ee-a458-a5ac86ec5361/tigefa4u.github.io-v3.4.2.zip\r\n",
      "https://zenodo.org/api/files/f109f917-1662-4c6a-ad0e-3592cbf8164e/eweitz/ideogram-v1.4.0.zip\r\n",
      "https://zenodo.org/api/files/6a35cd20-8827-43c3-a762-8895ff2a9c47/mikldk/mitolina-v0.0.1.zip\r\n",
      "https://zenodo.org/api/files/806a6c75-d72f-4e29-93ce-b5e9af4ab6fc/NLeSC/ahn-pointcloud-viewer-ws-v1.1.1.zip\r\n",
      "https://zenodo.org/api/files/5ca9d2ce-58a2-4977-85e0-56ac159b3f2d/common-workflow-language/cwltool-1.0.20170828135420.zip\r\n",
      "https://zenodo.org/api/files/7b6cea1b-3c11-4a2b-ae75-913d36303964/ThomasK81/ToPan-0.3.zip\r\n",
      "https://zenodo.org/api/files/dce7d3d0-137c-465a-bf93-22769ca96569/pcm-ca/vegas-lattice-rs-0.2.0.zip\r\n",
      "https://zenodo.org/api/files/c6192ebc-ad55-456f-ae08-5eb676bfbe99/sobiero/iati-ke-backend-v0.1.0.zip\r\n",
      "https://zenodo.org/api/files/cbed6a77-c968-40b9-9871-924c620c4e89/PerseusDL/canonical-greekLit-0.0.1282.zip\r\n",
      "https://zenodo.org/api/files/cd9221d5-4747-42da-a86d-2c9b8ef85845/sertansenturk/makammusicbrainz-v1.4.1.zip\r\n",
      "https://zenodo.org/api/files/969cfcf5-5146-4742-b83b-cf91e4b14ccf/crazyhottommy/pyflow-RNAseq-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/2f117eab-f2c1-4536-bb89-55599215347f/cldf/csvw-v1.0.zip\r\n",
      "https://zenodo.org/api/files/c6cbfe05-01ed-4683-a99e-8d4f14e3dfb0/esa/pagmo2-v2.11.zip\r\n",
      "https://zenodo.org/api/files/d1520d10-ced6-484d-9aed-2deff66660b1/samapriya/Planet-GEE-Pipeline-GUI-0.1.1.zip\r\n",
      "https://zenodo.org/api/files/fb6f6fd2-5bb8-4773-97b7-815e596e1938/sennierer/apis-core-v0.8.1.zip\r\n",
      "https://zenodo.org/api/files/516b1037-3c1b-4147-9389-e8a495303291/MDSLab/wstun-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/fe0406ea-cd85-4e21-ae38-23d902115445/librairy/study-semantic-similarity-1.0.zip\r\n",
      "https://zenodo.org/api/files/a4a6c36b-d049-4879-bb56-2b0635280fb0/ropensci/drake-v7.0.0.zip\r\n",
      "https://zenodo.org/api/files/e8cce935-e374-478d-8098-4d4b07e5bfbf/exa-analytics/exa-v0.4.1.zip\r\n",
      "https://zenodo.org/api/files/b93b6962-f4a4-4af8-ab95-653838d0b980/Biskit1943/tensorflow-v1.12.0-rc2.zip\r\n",
      "https://zenodo.org/api/files/fc777b1a-2f93-4745-b38c-40cd9e3c5557/stop-it-h2020-project/PublicWarningNotificationSystem-v1.0.zip\r\n",
      "https://zenodo.org/api/files/6f65605f-ef18-4f77-b104-ebdb476d12a7/clld/clld-v4.0.0.zip\r\n",
      "https://zenodo.org/api/files/2a3a5d84-2f8a-40e5-b7a8-cda820c8139d/brianjimenez/lightdock-0.5.2.zip\r\n",
      "https://zenodo.org/api/files/68e4e9c0-c2f6-4d50-831c-8a4d737839ee/seignovert/python-opus-seti-v0.4.zip\r\n",
      "https://zenodo.org/api/files/28d737b4-a245-4919-9744-9ed815174e4c/scivision/pymap3d-v1.7.2.zip\r\n",
      "https://zenodo.org/api/files/e9e9b322-0cf6-43e9-b272-65c4f810dab5/jhpoelen/eol-globi-data-v0.9.11.zip\r\n",
      "https://zenodo.org/api/files/d23ca8d8-37e3-42bf-95da-97f02461a027/makam_recognition_experiments-dlfm2016.zip\r\n",
      "https://zenodo.org/api/files/8c059116-3cb6-402d-bd31-7a10f1c795cf/proycon/clam-v2.4.7.zip\r\n",
      "https://zenodo.org/api/files/cc7af081-a0a3-4361-b482-45927dc41c80/Ewpratten/barebonesFRC-5.0.zip\r\n",
      "https://zenodo.org/api/files/0a4a679e-9457-4278-ab16-1c934aac793a/unit-ontology-review-results-v1.2.0.zip\r\n",
      "https://zenodo.org/api/files/0dbca4eb-3026-4beb-be9c-7f71076351a0/MosquitoAlert/Data-v1.303.2017.zip\r\n",
      "https://zenodo.org/api/files/488734e2-5e97-4309-9a01-882e6774b3b5/dib-lab/sourmash-v2.0.0a10.zip\r\n",
      "https://zenodo.org/api/files/6eb9460f-1655-4253-95ca-fe531913f67c/arm61/pylj-1.3.4.zip\r\n",
      "https://zenodo.org/api/files/69e273e8-d952-42d2-a101-bd1053d32133/kipoi/models-v2019-02-03.zip\r\n",
      "https://zenodo.org/api/files/58ee1226-e099-4119-add6-d640f02b04a7/rorynolan/strex-v1.0.3.zip\r\n",
      "https://zenodo.org/api/files/b11250ac-2777-43ec-8e3a-4110f17917db/PerseusDL/canonical-latinLit-0.0.30.zip\r\n",
      "https://zenodo.org/api/files/235e29c5-8525-4d0a-8663-17d06f729f60/StackTag.zip\r\n",
      "https://zenodo.org/api/files/73d37e32-e64b-401e-86cc-b942305fcdb4/toniher/seqservice-0.1.2.zip\r\n",
      "https://zenodo.org/api/files/a9944826-cc5e-486f-b32e-7da7bba99be6/HBClab/NiBetaSeries-test_release.zip\r\n",
      "https://zenodo.org/api/files/232472d7-a5b9-4d2a-8ff2-ea146b52e703/jhpoelen/eol-globi-data-v0.8.12.zip\r\n",
      "https://zenodo.org/api/files/6b950165-e4eb-441a-8cd5-da59d0c24f0f/matplotlib/matplotlib-v3.1.0.zip\r\n",
      "https://zenodo.org/api/files/9ff5d930-1d8b-4a47-b39e-4af41b9bc84d/bd2kccd/ccd-annotations-cytoscape-v0.3.1-alpha.zip\r\n",
      "https://zenodo.org/api/files/0622966e-e69a-437e-a903-d6864e693969/MosquitoAlert/Data-v1.136.2017.zip\r\n",
      "https://zenodo.org/api/files/99f6a786-ea0f-4732-b754-a271b9d056dc/CESNET/proxystatistics-simplesamlphp-module-v1.2.0.zip\r\n",
      "https://zenodo.org/api/files/3b5a0dc1-0e69-4a41-b71d-27ea0303506d/adibender/pammtools-v0.1.7.zip\r\n",
      "https://zenodo.org/api/files/09059a0b-7c02-434a-8d8d-7769d5440d93/biom-conversion-server-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/565ff11a-88aa-4c0b-bc0a-d4f226cf47a0/CiTR/djland-2.9.0.zip\r\n",
      "https://zenodo.org/api/files/29017782-955a-4678-b2e5-7ca8dacbef00/CLARIAH/grlc-v1.2.1.zip\r\n",
      "https://zenodo.org/api/files/c93f8b47-4ad3-403f-9696-9760839bba0e/ComPlat/ketcher-rails-v0.1.6.zip\r\n",
      "https://zenodo.org/api/files/1d3af06a-dd52-499b-b178-e84cca9f5fa2/pacman-v.0.2.0.zip\r\n",
      "https://zenodo.org/api/files/c6f394f0-d4e1-40d0-b6a3-aa5f9cf40940/BenLangmead/bowtie2-v2.3.5.zip\r\n",
      "https://zenodo.org/api/files/de35e680-2278-4f31-8310-2f806a3d05c3/BenLangmead/bowtie2-v2.3.3.1.zip\r\n",
      "https://zenodo.org/api/files/d6816195-3b43-4b29-8fa6-f1c1566c954b/explosion/spaCy-v2.0.17.zip\r\n",
      "https://zenodo.org/api/files/99fe870c-6d60-4df7-8ea5-953cacad3a56/quanteda/quanteda-v1.3.10.zip\r\n",
      "https://zenodo.org/api/files/1df88604-ddc6-40bc-952b-1235350d53ee/xenon-middleware/xenon-cli-v3.0.3.zip\r\n",
      "https://zenodo.org/api/files/bff716f1-17c5-4003-88aa-25b682583552/pyGSTio/pyGSTi-v0.9.7.1.zip\r\n",
      "https://zenodo.org/api/files/42fc7ea1-630a-4100-bd69-cd8bd11fef2b/cern-it-sdc-id/puppet-dpm-v0.6.0.zip\r\n",
      "https://zenodo.org/api/files/f9d14c7b-d22e-4281-8d6e-9892b23c1031/Qiskit/qiskit-0.9.0.zip\r\n",
      "https://zenodo.org/api/files/e8eaf17e-2807-41da-979b-a02a14bcfff6/myvariant.info-master-fb2a871.zip\r\n",
      "https://zenodo.org/api/files/cd662627-34c9-4fe0-82a9-2fe09fad135a/DailyPythonScripts-TOP-12-042_V4.zip\r\n",
      "https://zenodo.org/api/files/e9b0b7b9-af9a-4e7a-809b-f83d249f3402/RubenVerborgh/N3.js-v0.9.0.zip\r\n",
      "https://zenodo.org/api/files/288a15df-3776-4fd8-92bd-366f2d319d1a/pyviz/holoviews-v1.11.3.zip\r\n",
      "https://zenodo.org/api/files/88259f3c-34f6-465c-a648-6235cabb5f6a/CeON/CERMINE-cermine-parent-1.13.zip\r\n",
      "https://zenodo.org/api/files/ee54a6d6-886c-4c0f-a3ee-9d620a340726/sdss/marvin-2.1.1.zip\r\n",
      "https://zenodo.org/api/files/3b667de8-da94-47e2-b301-9a06ccbb076a/underworldcode/underworld2-v2.6.0b.zip\r\n",
      "https://zenodo.org/api/files/d135da3d-898f-43a6-81c6-8b197761113e/garethcmurphy/PyMetadataCreator-v2.1.1.zip\r\n",
      "https://zenodo.org/api/files/e675f35b-a992-4b9f-b5d9-3c7e9b0c00d4/formr.org-0.2.0.zip\r\n",
      "https://zenodo.org/api/files/68c47204-0163-40f5-970b-8c0af6658d12/DLR-RM/RAFCON-0.12.20.zip\r\n",
      "https://zenodo.org/api/files/c2071219-6009-4381-9dd1-aa4aefa04a25/ConservationInternational/trends.earth-0.64.zip\r\n",
      "https://zenodo.org/api/files/5fcbafa6-96ac-4a01-a2d7-25970ed3bade/collection-v1.3.zip\r\n",
      "https://zenodo.org/api/files/8ed6b101-4d4c-4aa9-accb-b4da53afdc2d/midas-isg/digital-commons-1.0-Dryad.zip\r\n",
      "https://zenodo.org/api/files/6f1691b5-8680-45b7-9ace-d1b307aff36a/Qiskit/qiskit-0.11.2.zip\r\n",
      "https://zenodo.org/api/files/0e2017c2-c3fc-42f4-94b5-72035bd33b17/PerseusDL/canonical-latinLit-0.0.137.zip\r\n",
      "https://zenodo.org/api/files/321c8cb2-f840-4aa8-98ca-78a5f9f7b45a/Becksteinlab/scipy2019-pmda-data-1.0.0.zip\r\n",
      "https://zenodo.org/api/files/3b30205f-c46e-447e-b2c6-8f5e07f1a19f/PGijsbers/gama-19.01.0.zip\r\n",
      "https://zenodo.org/api/files/fc763c55-e02b-45c6-aea0-1976eaaf49c8/joule-kmeans.zip\r\n",
      "https://zenodo.org/api/files/1372807f-5f32-48de-81dc-e6d6aef1f25a/mme-apis-v1.0a.zip\r\n",
      "https://zenodo.org/api/files/bd960341-a284-4034-814d-6ea59021c22a/JuliaSmoothOptimizers/AmplNLReader.jl-v0.5.0.zip\r\n",
      "https://zenodo.org/api/files/6ca52f05-5ce0-4597-ae29-543340037d71/klavinslab/aquarium-v2.5.1.zip\r\n",
      "https://zenodo.org/api/files/53307082-baa7-42ba-ad75-229e2592d9d8/SAOImageDS9/SAOImageDS9-v8.0rc6.zip\r\n",
      "https://zenodo.org/api/files/8cb211e1-b064-4e29-b7e5-227ed2dcd452/stenskjaer/samewords-v0.5.6.zip\r\n",
      "https://zenodo.org/api/files/f2f68a87-1f25-4ce8-93e9-fb9b9991c1fb/FelixHenninger/lab.js-v2017.1.0.zip\r\n",
      "https://zenodo.org/api/files/a870e3c0-d3cd-41f2-a1b3-970a9bf1071d/MosquitoAlert/Data-v1.146.2017.zip\r\n",
      "https://zenodo.org/api/files/977ce3cf-3d3e-43dc-b6df-a7bc260f36f1/exa-analytics/exa-v0.3.0.zip\r\n",
      "https://zenodo.org/api/files/0b7dccbe-eaef-4214-bbf6-be6d89ba98ec/cmssw-CMSSW_6_2_0_SLHC26_patch3.zip\r\n",
      "https://zenodo.org/api/files/2a775ef2-b1d0-4e33-8bc5-c6c99874f9e4/sfsheath/dhap-heath-0.0.3.zip\r\n",
      "https://zenodo.org/api/files/8b72f182-da88-458f-a043-5e934ec15021/failys/cairis-1.5.2.zip\r\n",
      "https://zenodo.org/api/files/20a89a67-10d6-4e05-9a10-d7d62bd6a363/phenomecentre/nPYc-Toolbox-v.1.2.0.zip\r\n",
      "https://zenodo.org/api/files/8a6f05e9-e51c-48cd-89f6-c273e2c6ff85/svkucheryavski/mdatools-0.9.5.zip\r\n",
      "https://zenodo.org/api/files/2cba192f-00dd-4721-99b5-feab80f64e82/cytoscape/cytoscape.js-edgehandles-v3.6.0.zip\r\n",
      "https://zenodo.org/api/files/66df01e6-2b5c-40dc-8f4f-01c4bb23b665/SLACKHA/pyJac-v1.0.3.zip\r\n",
      "https://zenodo.org/api/files/f8319abe-77e8-407a-bc85-192876d745ca/gwu-libraries/sfm-flickr-harvester-1.5.0.zip\r\n",
      "https://zenodo.org/api/files/ff17545b-0b94-49f0-831b-6b242d6c283a/clld-v0.33.zip\r\n",
      "https://zenodo.org/api/files/775922e9-0f23-408e-947a-582f82184fa9/scivision/pymap3d-v1.7.10.zip\r\n",
      "https://zenodo.org/api/files/4dfd668d-9584-44f9-b311-12f3deddbf8c/TTG-1.zip\r\n",
      "https://zenodo.org/api/files/23681253-f08e-4fc7-b8dc-359f559e389f/cgpCaVEManWrapper-v1.6.0.zip\r\n",
      "https://zenodo.org/api/files/cea8997b-5d0a-4ca0-accb-8b65dd413b87/thackl/cr-genomes-v1.8.zip\r\n",
      "https://zenodo.org/api/files/20cff293-2d82-4276-b77c-5c6e6ad0ff02/datascienceinc/Skater-v1.1.1-b3.zip\r\n",
      "https://zenodo.org/api/files/ed650b9b-f902-44f3-a0e2-55f5852f4de0/fact-project/aict-tools-v0.17.0.zip\r\n",
      "https://zenodo.org/api/files/c2493cf9-0cf5-430c-a289-4071d7e8ffe3/MRN-Code/coinstac-v3.1.8.zip\r\n",
      "https://zenodo.org/api/files/903bab90-24e2-4f3b-b6dd-9c4b3d6cfd92/plasticityai/magnitude-0.1.101.zip\r\n",
      "https://zenodo.org/api/files/73b9b780-65fe-415b-8294-ffede6ce7227/samapriya/Clip-Ship-Planet-CLI-0.1.9.zip\r\n",
      "https://zenodo.org/api/files/687c9b33-c208-4939-af7e-12cc4b9b5416/pylayers-v0.3-beta.zip\r\n",
      "https://zenodo.org/api/files/dd3d2962-2941-468a-b85c-81c53beaabe3/MosquitoAlert/Data-v1.155.2018.zip\r\n",
      "https://zenodo.org/api/files/8dba0af4-daa4-4e19-b705-945df9fb78ef/MosquitoAlert/Data-v1.251.2019.zip\r\n",
      "https://zenodo.org/api/files/f2ea7abf-26b7-4c7b-9c3c-c9d4e65ab4f1/moritzploss/tribology-v0.5.11.zip\r\n",
      "https://zenodo.org/api/files/708eac1a-746e-49a9-aeb1-52582298cede/chris-prener/testDriveR-v0.5.1.zip\r\n",
      "https://zenodo.org/api/files/766b3747-aa76-487f-a70a-51550882904a/PCMDI/cmor-3.1.3.zip\r\n",
      "https://zenodo.org/api/files/e066f036-351f-4e5e-8319-9d4082a423e1/cmla/s2p-v1.0b18.zip\r\n",
      "https://zenodo.org/api/files/4a10d70b-749b-4964-b8a3-a3a7b596699d/bio-phys/MDBenchmark-version-1.3.2.zip\r\n",
      "https://zenodo.org/api/files/8e167f88-bb53-42e0-a492-c14c2810959e/DLR-SC/island-viz-v1.0.zip\r\n",
      "https://zenodo.org/api/files/7e414f28-c0b0-4c51-8639-4d5e62180add/esa/pagmo2-v2.11.3.zip\r\n",
      "https://zenodo.org/api/files/c3bb7627-660e-4df7-b743-0b604ae6210d/poldracklab/fmriprep-1.4.1rc1.zip\r\n",
      "https://zenodo.org/api/files/4a8435d6-8cad-404a-98ba-b44569832c4e/dhlab-epfl/LinkedBooksMonographs-v1.1.zip\r\n",
      "https://zenodo.org/api/files/8207dffb-d657-4a17-ba99-e3da21340d17/labra/shaclex-v0.1.00.zip\r\n",
      "https://zenodo.org/api/files/904ae7b5-d384-4d8f-a920-4ce1fea7fc38/MosquitoAlert/Data-v1.019.2017.zip\r\n",
      "https://zenodo.org/api/files/deed6681-fcd4-40d2-b73f-e9bf3842f037/ropensci/drake-v6.1.0.zip\r\n",
      "https://zenodo.org/api/files/8e15a508-8e2c-4a6a-9c9a-4ba59630a7dd/miledrousset/opentheso-4.4.9.zip\r\n",
      "https://zenodo.org/api/files/efce36b4-725a-4771-9f88-09403c9db808/MosquitoAlert/Data-v1.193.2018.zip\r\n",
      "https://zenodo.org/api/files/8d05d05a-af4d-4ebc-be3d-f7df8db7be0b/PerseusDL/canonical-greekLit-0.0.1434.zip\r\n",
      "https://zenodo.org/api/files/dee5052a-794b-4c1d-8f52-92942f7afb03/highlando/dolfin_navier_scipy-v2.0.zip\r\n",
      "https://zenodo.org/api/files/f454321c-c052-43a7-9db6-4c8d25f76229/PerseusDL/canonical-greekLit-0.0.137.zip\r\n",
      "https://zenodo.org/api/files/d90f931f-a973-4fea-a4ad-3bda99e0bf41/urbanriskmap/cognicity-schema-v3.0.0.zip\r\n",
      "https://zenodo.org/api/files/2e206d9d-247a-47be-9cc2-1496169c9ccb/monarch-initiative/mondo-v2018-10-14.zip\r\n",
      "https://zenodo.org/api/files/b40bb443-be08-4f28-b750-10b715e86dfa/esa/pagmo2-v2.8.zip\r\n",
      "https://zenodo.org/api/files/9e99268d-63f5-40ae-9512-3dd62420eddd/gnarayan/WDdata-v1.02.zip\r\n",
      "https://zenodo.org/api/files/5b5f239c-fb1c-4fdd-9e93-dcc493841445/plasticityai/magnitude-0.1.98.zip\r\n",
      "https://zenodo.org/api/files/99168fc5-ddc7-4a9a-9062-20e979a83ae1/3191110276/RAMLFlask-main.zip\r\n",
      "https://zenodo.org/api/files/26a21fa7-f1a5-4ed4-a220-c7164f8354f3/CUAHSI/nwm_subsetting-0.2.zip\r\n",
      "https://zenodo.org/api/files/ebca206e-175f-45be-b2c5-3e585ee7a5d6/marl/jams-v0.3.1.zip\r\n",
      "https://zenodo.org/api/files/e19e7654-c71f-46ca-9244-d2e6d2804392/ElektraInitiative/libelektra-0.8.22.zip\r\n",
      "https://zenodo.org/api/files/6cdda13f-8d62-49ff-ac25-dc847847ab1c/materialsvirtuallab/flamyngo-v0.7.8.zip\r\n",
      "https://zenodo.org/api/files/5d00c870-e01f-4c41-933e-b18470312a79/luciekaffee/NumTab-v0.1.zip\r\n",
      "https://zenodo.org/api/files/f70715a7-c222-4228-a7ce-adb795bdfa55/cognicity-reports-powertrack-v2.0.0.zip\r\n",
      "https://zenodo.org/api/files/e93db670-828a-4cec-87ae-0447e963779b/chronovise.zip\r\n",
      "https://zenodo.org/api/files/1a73b1f9-99f3-4398-945a-c306d41aa1f9/mdtraj-1.4.2.zip\r\n",
      "https://zenodo.org/api/files/3b03dc8c-c1c7-4a7e-903a-8aba982e4a75/blobtools-v0.9.19.3.zip\r\n",
      "https://zenodo.org/api/files/82af1da0-5477-4485-8179-6fd87e1fa36f/mikldk/disclapmix-1.7.zip\r\n",
      "https://zenodo.org/api/files/d233b5ca-6670-46f2-81e8-44e5f2f59dc1/Irys-scaffolding-v1.0.0.zip\r\n",
      "https://zenodo.org/api/files/426ceea4-f397-4479-bfa2-5be6c0255cfd/cligs/projects-v.0.2.1.zip\r\n",
      "https://zenodo.org/api/files/8e7a0425-d38d-4cb1-b94e-3d1af5d9272b/SDG-InterfaceOntology/sdgio-v2018-05-11.zip\r\n",
      "https://zenodo.org/api/files/c90af79d-aa4d-4f3d-b280-9d97e1fe8b16/mathjax/mathjax-v3-3.0.0-beta.3.zip\r\n",
      "https://zenodo.org/api/files/a08205c4-0450-47f9-8eb6-0e76c374685d/pybel/pybel-v0.9.2.zip\r\n",
      "https://zenodo.org/api/files/7bc900c7-e570-43de-9bdf-bc52735c616b/didillysquat/SymPortal_framework-v0.3.3.zip\r\n",
      "https://zenodo.org/api/files/216ef8b5-c97a-425f-a905-da5f0bae60c5/matthiaskoenig/cy3sbml-v0.2.5.zip\r\n",
      "https://zenodo.org/api/files/427a6f57-730c-4891-ad1d-f1bf296686cb/bety-betydb_3.2.1.zip\r\n",
      "https://zenodo.org/api/files/b277f4c0-7579-4b9f-a2f1-456766863a61/globalbioticinteractions/globalbioticinteractions.github.io-0.1.zip\r\n",
      "https://zenodo.org/api/files/b68e1b2e-fe8d-40e5-abac-42ded78e6922/architecture-building-systems/Wireless-Sensor-Network-v0.5.zip\r\n"
     ]
    }
   ],
   "source": [
    "! cat data/metadata-files/json/listing.txt | cut -d \":\" -f 1 | sort | uniq \\\n",
    "  | sed s/listing/sample/ | sed s/txt$/tsv/ | \\\n",
    "  xargs awk '{print $5}' > data/metadata-files/json/urls.txt\n",
    "!cat data/metadata-files/json/urls.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "(Re)download all the JSON files to unzip them."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2020-02-07 11:29:34--  https://zenodo.org/api/files/602ad012-6cad-4e1c-9e13-ffaf65b282b6/raptr-manuscript.zip\n",
      "Resolving zenodo.org (zenodo.org)... 188.184.95.95\n",
      "Connecting to zenodo.org (zenodo.org)|188.184.95.95|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 721714644 (688M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/602ad012-6cad-4e1c-9e13-ffaf65b282b6/raptr-manuscript.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 688.28M  14.0MB/s    in 45s     \n",
      "\n",
      "2020-02-07 11:30:20 (15.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/602ad012-6cad-4e1c-9e13-ffaf65b282b6/raptr-manuscript.zip' saved [721714644/721714644]\n",
      "\n",
      "--2020-02-07 11:30:20--  https://zenodo.org/api/files/792b2464-eedf-427f-8166-883a85de8b5d/Fe-Example_input_files.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 6098 (6.0K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/792b2464-eedf-427f-8166-883a85de8b5d/Fe-Example_input_files.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   5.96K  --.-KB/s    in 0s      \n",
      "\n",
      "2020-02-07 11:30:20 (806 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/792b2464-eedf-427f-8166-883a85de8b5d/Fe-Example_input_files.zip' saved [6098/6098]\n",
      "\n",
      "--2020-02-07 11:30:20--  https://zenodo.org/api/files/ec40beec-f99c-4f95-914e-dcc329df3a84/weecology/PortalData-1.104.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 26841703 (26M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ec40beec-f99c-4f95-914e-dcc329df3a84/weecology/PortalData-1.104.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  25.60M  14.3MB/s    in 1.8s    \n",
      "\n",
      "2020-02-07 11:30:22 (14.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ec40beec-f99c-4f95-914e-dcc329df3a84/weecology/PortalData-1.104.0.zip' saved [26841703/26841703]\n",
      "\n",
      "--2020-02-07 11:30:22--  https://zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/rome.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 204900122 (195M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/rome.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 195.41M  12.3MB/s    in 13s     \n",
      "\n",
      "2020-02-07 11:30:36 (14.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/rome.zip' saved [204900122/204900122]\n",
      "\n",
      "--2020-02-07 11:30:36--  https://zenodo.org/api/files/579d4580-d2b4-4441-ab11-3a9bdbf8aa89/CN-TU/nta-meta-analysis-2018.11.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 198859 (194K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/579d4580-d2b4-4441-ab11-3a9bdbf8aa89/CN-TU/nta-meta-analysis-2018.11.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 194.20K  --.-KB/s    in 0.02s   \n",
      "\n",
      "2020-02-07 11:30:37 (8.20 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/579d4580-d2b4-4441-ab11-3a9bdbf8aa89/CN-TU/nta-meta-analysis-2018.11.zip' saved [198859/198859]\n",
      "\n",
      "--2020-02-07 11:30:37--  https://zenodo.org/api/files/5d6ce4de-f954-4e1d-9285-4fce0ef0a146/Collocated%20Datesets%20for%20Terra%20and%20Aqua.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2080538 (2.0M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/5d6ce4de-f954-4e1d-9285-4fce0ef0a146/Collocated Datesets for Terra and Aqua.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.98M  11.1MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 11:30:38 (11.1 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/5d6ce4de-f954-4e1d-9285-4fce0ef0a146/Collocated Datesets for Terra and Aqua.zip' saved [2080538/2080538]\n",
      "\n",
      "--2020-02-07 11:30:38--  https://zenodo.org/api/files/5935dbec-ac23-4b32-8421-4b3bf07e3f75/LANL_Mustang_parquet.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 18199843 (17M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/5935dbec-ac23-4b32-8421-4b3bf07e3f75/LANL_Mustang_parquet.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  17.36M  13.3MB/s    in 1.3s    \n",
      "\n",
      "2020-02-07 11:30:39 (13.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/5935dbec-ac23-4b32-8421-4b3bf07e3f75/LANL_Mustang_parquet.zip' saved [18199843/18199843]\n",
      "\n",
      "--2020-02-07 11:30:39--  https://zenodo.org/api/files/217beb43-3a3a-429b-86cf-a154ded2f6e5/graphs_json.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 578402315 (552M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/217beb43-3a3a-429b-86cf-a154ded2f6e5/graphs_json.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 551.61M  14.2MB/s    in 41s     \n",
      "\n",
      "2020-02-07 11:31:21 (13.6 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/217beb43-3a3a-429b-86cf-a154ded2f6e5/graphs_json.zip' saved [578402315/578402315]\n",
      "\n",
      "--2020-02-07 11:31:21--  https://zenodo.org/api/files/a2fec3ee-69f7-467e-aeba-f9d8bab7853d/persons2.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 582533754 (556M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a2fec3ee-69f7-467e-aeba-f9d8bab7853d/persons2.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 555.55M  15.2MB/s    in 34s     \n",
      "\n",
      "2020-02-07 11:31:55 (16.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a2fec3ee-69f7-467e-aeba-f9d8bab7853d/persons2.zip' saved [582533754/582533754]\n",
      "\n",
      "--2020-02-07 11:31:55--  https://zenodo.org/api/files/706a040e-d9be-41c7-8677-cd94d52a1fb8/data_metrics.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 27812 (27K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/706a040e-d9be-41c7-8677-cd94d52a1fb8/data_metrics.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  27.16K  --.-KB/s    in 0s      \n",
      "\n",
      "2020-02-07 11:31:56 (150 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/706a040e-d9be-41c7-8677-cd94d52a1fb8/data_metrics.zip' saved [27812/27812]\n",
      "\n",
      "--2020-02-07 11:31:56--  https://zenodo.org/api/files/8b54bee9-9932-4f9d-8cbb-61f83c5ccf56/collection-v1.11.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 18359249 (18M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/8b54bee9-9932-4f9d-8cbb-61f83c5ccf56/collection-v1.11.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  17.51M  11.9MB/s    in 1.5s    \n",
      "\n",
      "2020-02-07 11:31:58 (11.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/8b54bee9-9932-4f9d-8cbb-61f83c5ccf56/collection-v1.11.zip' saved [18359249/18359249]\n",
      "\n",
      "--2020-02-07 11:31:58--  https://zenodo.org/api/files/335b11e6-9ee3-4fc6-b892-4b650bcfe02b/collection-v1.10.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 18296734 (17M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/335b11e6-9ee3-4fc6-b892-4b650bcfe02b/collection-v1.10.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  17.45M  13.2MB/s    in 1.3s    \n",
      "\n",
      "2020-02-07 11:32:00 (13.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/335b11e6-9ee3-4fc6-b892-4b650bcfe02b/collection-v1.10.zip' saved [18296734/18296734]\n",
      "\n",
      "--2020-02-07 11:32:00--  https://zenodo.org/api/files/7d1192dc-a006-4ee0-a718-2b894fec7a6e/weecology/forecasts-2018-02-16.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 10937101 (10M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/7d1192dc-a006-4ee0-a718-2b894fec7a6e/weecology/forecasts-2018-02-16.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  10.43M  13.9MB/s    in 0.7s    \n",
      "\n",
      "2020-02-07 11:32:01 (13.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/7d1192dc-a006-4ee0-a718-2b894fec7a6e/weecology/forecasts-2018-02-16.zip' saved [10937101/10937101]\n",
      "\n",
      "--2020-02-07 11:32:01--  https://zenodo.org/api/files/ed4de091-d86d-465a-bc3b-9d70e12a77b4/antiSMASH_Actinobacterial_BGCs.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 27769190264 (26G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ed4de091-d86d-465a-bc3b-9d70e12a77b4/antiSMASH_Actinobacterial_BGCs.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  25.86G  18.1MB/s    in 30m 48s \n",
      "\n",
      "2020-02-07 12:02:49 (14.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ed4de091-d86d-465a-bc3b-9d70e12a77b4/antiSMASH_Actinobacterial_BGCs.zip' saved [27769190264/27769190264]\n",
      "\n",
      "--2020-02-07 12:02:49--  https://zenodo.org/api/files/e0425375-1288-4105-b970-08ebd8c654a8/dataset_for_validation_experiments_mitmprobe.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 15741512476 (15G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/e0425375-1288-4105-b970-08ebd8c654a8/dataset_for_validation_experiments_mitmprobe.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  14.66G  15.1MB/s    in 17m 25s \n",
      "\n",
      "2020-02-07 12:20:14 (14.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/e0425375-1288-4105-b970-08ebd8c654a8/dataset_for_validation_experiments_mitmprobe.zip' saved [15741512476/15741512476]\n",
      "\n",
      "--2020-02-07 12:20:14--  https://zenodo.org/api/files/9712835c-b4d6-4fcc-b0ec-dba4f346dd40/clinicalcodes.article5-v0.1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 965721 (943K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/9712835c-b4d6-4fcc-b0ec-dba4f346dd40/clinicalcodes.article5-v0.1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 943.09K  4.60MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:20:15 (4.60 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/9712835c-b4d6-4fcc-b0ec-dba4f346dd40/clinicalcodes.article5-v0.1.0.zip' saved [965721/965721]\n",
      "\n",
      "--2020-02-07 12:20:15--  https://zenodo.org/api/files/e6b5c812-92ab-4552-8a03-1782bc8245bc/wfschneidergroup/SI-mehta-plasma-N2-vibrations-0.5.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 392632 (383K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/e6b5c812-92ab-4552-8a03-1782bc8245bc/wfschneidergroup/SI-mehta-plasma-N2-vibrations-0.5.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 383.43K  --.-KB/s    in 0.07s   \n",
      "\n",
      "2020-02-07 12:20:16 (4.99 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/e6b5c812-92ab-4552-8a03-1782bc8245bc/wfschneidergroup/SI-mehta-plasma-N2-vibrations-0.5.zip' saved [392632/392632]\n",
      "\n",
      "--2020-02-07 12:20:16--  https://zenodo.org/api/files/77d2fd12-585f-4e94-a5ea-56ba5f9e942d/zenodo_open_metadata_06_03_2017.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 82876682 (79M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/77d2fd12-585f-4e94-a5ea-56ba5f9e942d/zenodo_open_metadata_06_03_2017.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  79.04M  14.1MB/s    in 5.5s    \n",
      "\n",
      "2020-02-07 12:20:22 (14.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/77d2fd12-585f-4e94-a5ea-56ba5f9e942d/zenodo_open_metadata_06_03_2017.zip' saved [82876682/82876682]\n",
      "\n",
      "--2020-02-07 12:20:22--  https://zenodo.org/api/files/c101d543-3be4-4734-91bc-385cbc8123f3/gbv/kos-registry-20190603.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 7899 (7.7K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/c101d543-3be4-4734-91bc-385cbc8123f3/gbv/kos-registry-20190603.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   7.71K  --.-KB/s    in 0s      \n",
      "\n",
      "2020-02-07 12:20:23 (937 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/c101d543-3be4-4734-91bc-385cbc8123f3/gbv/kos-registry-20190603.zip' saved [7899/7899]\n",
      "\n",
      "--2020-02-07 12:20:23--  https://zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/kuopio.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4745040 (4.5M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/kuopio.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.52M  18.7MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:20:24 (18.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/cf2590f4-e026-4400-89ba-bc83fe3c665c/kuopio.zip' saved [4745040/4745040]\n",
      "\n",
      "--2020-02-07 12:20:24--  https://zenodo.org/api/files/3500a1bc-360a-45f4-8d31-4b6c61d942bb/Pegasus_P7_parquet.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 51845 (51K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3500a1bc-360a-45f4-8d31-4b6c61d942bb/Pegasus_P7_parquet.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  50.63K  --.-KB/s    in 0.002s  \n",
      "\n",
      "2020-02-07 12:20:24 (22.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3500a1bc-360a-45f4-8d31-4b6c61d942bb/Pegasus_P7_parquet.zip' saved [51845/51845]\n",
      "\n",
      "--2020-02-07 12:20:24--  https://zenodo.org/api/files/64b75e05-2673-4a59-8425-97be3d5b3ca0/emmanuel-senft/data-foodchain-v1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 790763 (772K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/64b75e05-2673-4a59-8425-97be3d5b3ca0/emmanuel-senft/data-foodchain-v1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 772.23K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2020-02-07 12:20:25 (5.13 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/64b75e05-2673-4a59-8425-97be3d5b3ca0/emmanuel-senft/data-foodchain-v1.0.zip' saved [790763/790763]\n",
      "\n",
      "--2020-02-07 12:20:25--  https://zenodo.org/api/files/59a30a09-aa06-4799-ad5c-ecc010d9724d/collection-v1.25.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 19237249 (18M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/59a30a09-aa06-4799-ad5c-ecc010d9724d/collection-v1.25.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  18.35M  15.8MB/s    in 1.2s    \n",
      "\n",
      "2020-02-07 12:20:27 (15.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/59a30a09-aa06-4799-ad5c-ecc010d9724d/collection-v1.25.zip' saved [19237249/19237249]\n",
      "\n",
      "--2020-02-07 12:20:27--  https://zenodo.org/api/files/a3b36c12-4a7b-4fa8-9172-d2be937f8cf1/WEBSITE_basic.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 84080135 (80M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a3b36c12-4a7b-4fa8-9172-d2be937f8cf1/WEBSITE_basic.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  80.18M  14.9MB/s    in 5.5s    \n",
      "\n",
      "2020-02-07 12:20:33 (14.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a3b36c12-4a7b-4fa8-9172-d2be937f8cf1/WEBSITE_basic.zip' saved [84080135/84080135]\n",
      "\n",
      "--2020-02-07 12:20:33--  https://zenodo.org/api/files/a1fa0f27-cbc8-4eb5-92ce-30c8076e3865/Supplementary%20files.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1725069920 (1.6G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a1fa0f27-cbc8-4eb5-92ce-30c8076e3865/Supplementary files.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.61G  13.9MB/s    in 1m 48s  \n",
      "\n",
      "2020-02-07 12:22:22 (15.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a1fa0f27-cbc8-4eb5-92ce-30c8076e3865/Supplementary files.zip' saved [1725069920/1725069920]\n",
      "\n",
      "--2020-02-07 12:22:22--  https://zenodo.org/api/files/742ff057-9769-4469-8c66-15b96e672336/WL04.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3514610204 (3.3G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/742ff057-9769-4469-8c66-15b96e672336/WL04.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.27G  29.8MB/s    in 3m 19s  \n",
      "\n",
      "2020-02-07 12:25:42 (16.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/742ff057-9769-4469-8c66-15b96e672336/WL04.zip' saved [3514610204/3514610204]\n",
      "\n",
      "--2020-02-07 12:25:42--  https://zenodo.org/api/files/6cce1121-0f26-4516-88b7-17bb1663a0c8/survey.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 316605 (309K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/6cce1121-0f26-4516-88b7-17bb1663a0c8/survey.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 309.18K  --.-KB/s    in 0.02s   \n",
      "\n",
      "2020-02-07 12:25:42 (14.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/6cce1121-0f26-4516-88b7-17bb1663a0c8/survey.zip' saved [316605/316605]\n",
      "\n",
      "--2020-02-07 12:25:42--  https://zenodo.org/api/files/14466297-96dc-4445-83d0-4c3299eb42c4/weecology/PortalData-1.98.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 26784196 (26M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/14466297-96dc-4445-83d0-4c3299eb42c4/weecology/PortalData-1.98.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  25.54M  18.4MB/s    in 1.4s    \n",
      "\n",
      "2020-02-07 12:25:44 (18.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/14466297-96dc-4445-83d0-4c3299eb42c4/weecology/PortalData-1.98.0.zip' saved [26784196/26784196]\n",
      "\n",
      "--2020-02-07 12:25:44--  https://zenodo.org/api/files/a191b63a-9d66-4c5d-81ac-18987486d7b2/weecology/portalPredictions-2018-04-27.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 67776550 (65M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a191b63a-9d66-4c5d-81ac-18987486d7b2/weecology/portalPredictions-2018-04-27.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  64.64M  15.9MB/s    in 4.1s    \n",
      "\n",
      "2020-02-07 12:25:49 (15.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a191b63a-9d66-4c5d-81ac-18987486d7b2/weecology/portalPredictions-2018-04-27.zip' saved [67776550/67776550]\n",
      "\n",
      "--2020-02-07 12:25:49--  https://zenodo.org/api/files/0c32f27d-01c5-412a-bf6c-a8fc42f55cc4/SeminalSurveyDBLP.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 90097849 (86M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/0c32f27d-01c5-412a-bf6c-a8fc42f55cc4/SeminalSurveyDBLP.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  85.92M  19.5MB/s    in 5.0s    \n",
      "\n",
      "2020-02-07 12:25:54 (17.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/0c32f27d-01c5-412a-bf6c-a8fc42f55cc4/SeminalSurveyDBLP.zip' saved [90097849/90097849]\n",
      "\n",
      "--2020-02-07 12:25:54--  https://zenodo.org/api/files/4222b3e2-3777-454f-b1c2-95e264431b46/dataset-nobagfiles.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 5702460052 (5.3G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/4222b3e2-3777-454f-b1c2-95e264431b46/dataset-nobagfiles.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   5.31G  15.7MB/s    in 5m 22s  \n",
      "\n",
      "2020-02-07 12:31:17 (16.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/4222b3e2-3777-454f-b1c2-95e264431b46/dataset-nobagfiles.zip' saved [5702460052/5702460052]\n",
      "\n",
      "--2020-02-07 12:31:17--  https://zenodo.org/api/files/17542c5f-9242-4403-a6cb-8ec034c04c9f/Data_and_Analyses.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1887100496 (1.8G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/17542c5f-9242-4403-a6cb-8ec034c04c9f/Data_and_Analyses.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.76G  28.9MB/s    in 93s     \n",
      "\n",
      "2020-02-07 12:32:51 (19.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/17542c5f-9242-4403-a6cb-8ec034c04c9f/Data_and_Analyses.zip' saved [1887100496/1887100496]\n",
      "\n",
      "--2020-02-07 12:32:51--  https://zenodo.org/api/files/05df56e1-f3e3-4728-a09e-b6b4f0d0a340/weecology/PortalData-1.72.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 26612810 (25M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/05df56e1-f3e3-4728-a09e-b6b4f0d0a340/weecology/PortalData-1.72.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  25.38M  23.1MB/s    in 1.1s    \n",
      "\n",
      "2020-02-07 12:32:52 (23.1 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/05df56e1-f3e3-4728-a09e-b6b4f0d0a340/weecology/PortalData-1.72.0.zip' saved [26612810/26612810]\n",
      "\n",
      "--2020-02-07 12:32:52--  https://zenodo.org/api/files/b2572be2-9276-4433-99e5-f7bc030ff081/ricklupton/uk-steel-model-v1.0.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1742417 (1.7M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/b2572be2-9276-4433-99e5-f7bc030ff081/ricklupton/uk-steel-model-v1.0.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.66M  --.-KB/s    in 0.07s   \n",
      "\n",
      "2020-02-07 12:32:53 (23.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/b2572be2-9276-4433-99e5-f7bc030ff081/ricklupton/uk-steel-model-v1.0.0.zip' saved [1742417/1742417]\n",
      "\n",
      "--2020-02-07 12:32:53--  https://zenodo.org/api/files/10cfa833-96ce-43c8-a01d-a07675e842d4/domain_split.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 57283 (56K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/10cfa833-96ce-43c8-a01d-a07675e842d4/domain_split.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  55.94K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2020-02-07 12:32:53 (84.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/10cfa833-96ce-43c8-a01d-a07675e842d4/domain_split.zip' saved [57283/57283]\n",
      "\n",
      "--2020-02-07 12:32:53--  https://zenodo.org/api/files/b60d6c09-61d5-47c3-b4d4-cfc06c6a3b80/weecology/forecasts-2018-10-19.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 10656700 (10M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/b60d6c09-61d5-47c3-b4d4-cfc06c6a3b80/weecology/forecasts-2018-10-19.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  10.16M  25.9MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:32:54 (25.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/b60d6c09-61d5-47c3-b4d4-cfc06c6a3b80/weecology/forecasts-2018-10-19.zip' saved [10656700/10656700]\n",
      "\n",
      "--2020-02-07 12:32:54--  https://zenodo.org/api/files/72dd54ca-efe0-4be7-a275-928fa43912cb/weecology/portalPredictions-2018-09-07.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 69131553 (66M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/72dd54ca-efe0-4be7-a275-928fa43912cb/weecology/portalPredictions-2018-09-07.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  65.93M  19.8MB/s    in 3.3s    \n",
      "\n",
      "2020-02-07 12:32:58 (20.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/72dd54ca-efe0-4be7-a275-928fa43912cb/weecology/portalPredictions-2018-09-07.zip' saved [69131553/69131553]\n",
      "\n",
      "--2020-02-07 12:32:58--  https://zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_21.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 698871068 (666M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_21.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 666.50M  21.4MB/s    in 33s     \n",
      "\n",
      "2020-02-07 12:33:31 (20.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_21.zip' saved [698871068/698871068]\n",
      "\n",
      "--2020-02-07 12:33:31--  https://zenodo.org/api/files/4fcac888-9cf5-44b9-b0d5-94ceba1f23dc/lingpy/language-island-paper-v1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3123452 (3.0M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/4fcac888-9cf5-44b9-b0d5-94ceba1f23dc/lingpy/language-island-paper-v1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.98M  18.5MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:33:32 (18.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/4fcac888-9cf5-44b9-b0d5-94ceba1f23dc/lingpy/language-island-paper-v1.0.zip' saved [3123452/3123452]\n",
      "\n",
      "--2020-02-07 12:33:32--  https://zenodo.org/api/files/b37db781-e7a5-4c00-9750-6bd8c2ffbcd2/sub-04.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3182849385 (3.0G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/b37db781-e7a5-4c00-9750-6bd8c2ffbcd2/sub-04.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.96G  11.2MB/s    in 2m 59s  \n",
      "\n",
      "2020-02-07 12:36:31 (17.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/b37db781-e7a5-4c00-9750-6bd8c2ffbcd2/sub-04.zip' saved [3182849385/3182849385]\n",
      "\n",
      "--2020-02-07 12:36:31--  https://zenodo.org/api/files/875d5f34-367a-4cab-b243-a20df1ead590/weecology/portalPredictions-2018-03-16.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 67549249 (64M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/875d5f34-367a-4cab-b243-a20df1ead590/weecology/portalPredictions-2018-03-16.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  64.42M  11.2MB/s    in 6.2s    \n",
      "\n",
      "2020-02-07 12:36:38 (10.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/875d5f34-367a-4cab-b243-a20df1ead590/weecology/portalPredictions-2018-03-16.zip' saved [67549249/67549249]\n",
      "\n",
      "--2020-02-07 12:36:38--  https://zenodo.org/api/files/bbc7f075-eac0-4968-851f-0a475a09819b/ComplexNetTSP/MultilayerParis-v1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 5032738 (4.8M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/bbc7f075-eac0-4968-851f-0a475a09819b/ComplexNetTSP/MultilayerParis-v1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.80M  9.76MB/s    in 0.5s    \n",
      "\n",
      "2020-02-07 12:36:39 (9.76 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/bbc7f075-eac0-4968-851f-0a475a09819b/ComplexNetTSP/MultilayerParis-v1.0.zip' saved [5032738/5032738]\n",
      "\n",
      "--2020-02-07 12:36:39--  https://zenodo.org/api/files/abb3d7fa-d2de-43bd-9423-269746cf8b67/weecology/portalPredictions-2018-03-09.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 67508723 (64M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/abb3d7fa-d2de-43bd-9423-269746cf8b67/weecology/portalPredictions-2018-03-09.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  64.38M  11.7MB/s    in 5.7s    \n",
      "\n",
      "2020-02-07 12:36:45 (11.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/abb3d7fa-d2de-43bd-9423-269746cf8b67/weecology/portalPredictions-2018-03-09.zip' saved [67508723/67508723]\n",
      "\n",
      "--2020-02-07 12:36:45--  https://zenodo.org/api/files/021841c1-717b-456c-8f31-f7968f4122b5/CORALCorpus.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1119453 (1.1M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/021841c1-717b-456c-8f31-f7968f4122b5/CORALCorpus.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.07M  4.88MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:36:46 (4.88 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/021841c1-717b-456c-8f31-f7968f4122b5/CORALCorpus.zip' saved [1119453/1119453]\n",
      "\n",
      "--2020-02-07 12:36:46--  https://zenodo.org/api/files/7ce871ea-5f97-43db-853a-ba1b4ddf07fd/askalon-new_ee26_parquet.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 59118 (58K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/7ce871ea-5f97-43db-853a-ba1b4ddf07fd/askalon-new_ee26_parquet.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  57.73K  --.-KB/s    in 0.004s  \n",
      "\n",
      "2020-02-07 12:36:47 (12.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/7ce871ea-5f97-43db-853a-ba1b4ddf07fd/askalon-new_ee26_parquet.zip' saved [59118/59118]\n",
      "\n",
      "--2020-02-07 12:36:47--  https://zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_15.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2335412699 (2.2G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_15.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.17G  14.7MB/s    in 3m 1s   \n",
      "\n",
      "2020-02-07 12:39:48 (12.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3fd98033-05f9-45e5-b8c4-0a536337920f/Patient_Reports_15.zip' saved [2335412699/2335412699]\n",
      "\n",
      "--2020-02-07 12:39:48--  https://zenodo.org/api/files/c0e7a65b-97e2-4d61-9b6b-150d6933327f/MathStackExchangeAPI_Part_2_TimeStamps_1484447260_1512760268.json.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 171955533 (164M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/c0e7a65b-97e2-4d61-9b6b-150d6933327f/MathStackExchangeAPI_Part_2_TimeStamps_1484447260_1512760268.json.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 163.99M  8.65MB/s    in 14s     \n",
      "\n",
      "2020-02-07 12:40:02 (11.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/c0e7a65b-97e2-4d61-9b6b-150d6933327f/MathStackExchangeAPI_Part_2_TimeStamps_1484447260_1512760268.json.zip' saved [171955533/171955533]\n",
      "\n",
      "--2020-02-07 12:40:02--  https://zenodo.org/api/files/d28a40ef-f657-4c4b-945b-c4f2b805b81c/workflowhub_epigenomics_dataset-hep_grid5000_schema-0-2_epigenomics-hep-g5k-run001_parquet.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 39815 (39K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/d28a40ef-f657-4c4b-945b-c4f2b805b81c/workflowhub_epigenomics_dataset-hep_grid5000_schema-0-2_epigenomics-hep-g5k-run001_parquet.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  38.88K  --.-KB/s    in 0.02s   \n",
      "\n",
      "2020-02-07 12:40:03 (2.37 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/d28a40ef-f657-4c4b-945b-c4f2b805b81c/workflowhub_epigenomics_dataset-hep_grid5000_schema-0-2_epigenomics-hep-g5k-run001_parquet.zip' saved [39815/39815]\n",
      "\n",
      "--2020-02-07 12:40:03--  https://zenodo.org/api/files/4b96de2a-2a4f-4078-83ec-098c28098863/globalbioticinteractions/Catalogue-of-Afrotropical-Bees-v0.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2513226 (2.4M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/4b96de2a-2a4f-4078-83ec-098c28098863/globalbioticinteractions/Catalogue-of-Afrotropical-Bees-v0.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.40M  3.48MB/s    in 0.7s    \n",
      "\n",
      "2020-02-07 12:40:04 (3.48 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/4b96de2a-2a4f-4078-83ec-098c28098863/globalbioticinteractions/Catalogue-of-Afrotropical-Bees-v0.1.zip' saved [2513226/2513226]\n",
      "\n",
      "--2020-02-07 12:40:04--  https://zenodo.org/api/files/ff29524f-d899-4699-974f-584f5247b19e/artifact.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 493183501 (470M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ff29524f-d899-4699-974f-584f5247b19e/artifact.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 470.34M  13.9MB/s    in 38s     \n",
      "\n",
      "2020-02-07 12:40:43 (12.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ff29524f-d899-4699-974f-584f5247b19e/artifact.zip' saved [493183501/493183501]\n",
      "\n",
      "--2020-02-07 12:40:43--  https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Sending%20Christmas%20Greetings%20IaaS.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 983574 (961K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Sending Christmas Greetings IaaS.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 960.52K  --.-KB/s    in 0.09s   \n",
      "\n",
      "2020-02-07 12:40:44 (10.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Sending Christmas Greetings IaaS.zip' saved [983574/983574]\n",
      "\n",
      "--2020-02-07 12:40:44--  https://zenodo.org/api/files/621c107b-e0e8-416d-be69-d6d0ce82bd9c/datacarpentry/R-ecology-lesson-v2017.04.3.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4711348 (4.5M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/621c107b-e0e8-416d-be69-d6d0ce82bd9c/datacarpentry/R-ecology-lesson-v2017.04.3.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.49M  13.5MB/s    in 0.3s    \n",
      "\n",
      "2020-02-07 12:40:45 (13.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/621c107b-e0e8-416d-be69-d6d0ce82bd9c/datacarpentry/R-ecology-lesson-v2017.04.3.zip' saved [4711348/4711348]\n",
      "\n",
      "--2020-02-07 12:40:45--  https://zenodo.org/api/files/e5a55eb3-b97e-4591-99f9-063400e0d852/glo-d-i-um/glo-d-i-um.github.io-1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2371693 (2.3M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/e5a55eb3-b97e-4591-99f9-063400e0d852/glo-d-i-um/glo-d-i-um.github.io-1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.26M  12.7MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:40:45 (12.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/e5a55eb3-b97e-4591-99f9-063400e0d852/glo-d-i-um/glo-d-i-um.github.io-1.0.zip' saved [2371693/2371693]\n",
      "\n",
      "--2020-02-07 12:40:45--  https://zenodo.org/api/files/5d1dc30d-0d3e-48a2-a335-15491bd55b0e/terraref/documentation-v0.01.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2193572 (2.1M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/5d1dc30d-0d3e-48a2-a335-15491bd55b0e/terraref/documentation-v0.01.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.09M  12.1MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:40:46 (12.1 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/5d1dc30d-0d3e-48a2-a335-15491bd55b0e/terraref/documentation-v0.01.zip' saved [2193572/2193572]\n",
      "\n",
      "--2020-02-07 12:40:46--  https://zenodo.org/api/files/a30fb1b2-f042-4cf8-a1ed-6ff79c076deb/swcarpentry/python-novice-gapminder-v2019.06.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2624094 (2.5M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a30fb1b2-f042-4cf8-a1ed-6ff79c076deb/swcarpentry/python-novice-gapminder-v2019.06.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.50M  12.2MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:40:47 (12.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a30fb1b2-f042-4cf8-a1ed-6ff79c076deb/swcarpentry/python-novice-gapminder-v2019.06.1.zip' saved [2624094/2624094]\n",
      "\n",
      "--2020-02-07 12:40:47--  https://zenodo.org/api/files/25c965d3-18fb-4737-b007-c215a37319be/analyses.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 898142869 (857M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/25c965d3-18fb-4737-b007-c215a37319be/analyses.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 856.54M  11.6MB/s    in 73s     \n",
      "\n",
      "2020-02-07 12:42:00 (11.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/25c965d3-18fb-4737-b007-c215a37319be/analyses.zip' saved [898142869/898142869]\n",
      "\n",
      "--2020-02-07 12:42:00--  https://zenodo.org/api/files/6739c06c-f1a7-4a98-8e4f-f012fd023b03/datacarpentry/r-raster-vector-geospatial-v2018.08.2.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4729721 (4.5M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/6739c06c-f1a7-4a98-8e4f-f012fd023b03/datacarpentry/r-raster-vector-geospatial-v2018.08.2.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.51M  12.4MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:42:01 (12.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/6739c06c-f1a7-4a98-8e4f-f012fd023b03/datacarpentry/r-raster-vector-geospatial-v2018.08.2.zip' saved [4729721/4729721]\n",
      "\n",
      "--2020-02-07 12:42:01--  https://zenodo.org/api/files/f3ce341c-70ab-4acb-95e3-dcaeaf0f930d/via-src-2.0.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2527344 (2.4M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/f3ce341c-70ab-4acb-95e3-dcaeaf0f930d/via-src-2.0.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.41M  11.2MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:42:02 (11.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/f3ce341c-70ab-4acb-95e3-dcaeaf0f930d/via-src-2.0.0.zip' saved [2527344/2527344]\n",
      "\n",
      "--2020-02-07 12:42:02--  https://zenodo.org/api/files/34f5cc18-c5e8-4e08-af74-1f7f717bff13/o2r_project_website_and_blog_git-repository.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 59052443 (56M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/34f5cc18-c5e8-4e08-af74-1f7f717bff13/o2r_project_website_and_blog_git-repository.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  56.32M  13.4MB/s    in 4.3s    \n",
      "\n",
      "2020-02-07 12:42:07 (13.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/34f5cc18-c5e8-4e08-af74-1f7f717bff13/o2r_project_website_and_blog_git-repository.zip' saved [59052443/59052443]\n",
      "\n",
      "--2020-02-07 12:42:07--  https://zenodo.org/api/files/ba8f9c1c-a22b-4254-91f1-65d215a80656/Diaspora_proof_of_concept.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 168025842 (160M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ba8f9c1c-a22b-4254-91f1-65d215a80656/Diaspora_proof_of_concept.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 160.24M  13.5MB/s    in 13s     \n",
      "\n",
      "2020-02-07 12:42:20 (12.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ba8f9c1c-a22b-4254-91f1-65d215a80656/Diaspora_proof_of_concept.zip' saved [168025842/168025842]\n",
      "\n",
      "--2020-02-07 12:42:20--  https://zenodo.org/api/files/3b2af3a0-ec88-4eaf-8738-1a7d27609e43/CIDER-Seq%20Results.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4767050 (4.5M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3b2af3a0-ec88-4eaf-8738-1a7d27609e43/CIDER-Seq Results.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.55M  12.4MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:42:21 (12.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3b2af3a0-ec88-4eaf-8738-1a7d27609e43/CIDER-Seq Results.zip' saved [4767050/4767050]\n",
      "\n",
      "--2020-02-07 12:42:21--  https://zenodo.org/api/files/b856436c-80a0-449d-a44f-447812c06fbd/datacarpentry/spreadsheets-socialsci-v2018.05-2.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2347690 (2.2M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/b856436c-80a0-449d-a44f-447812c06fbd/datacarpentry/spreadsheets-socialsci-v2018.05-2.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.24M  12.1MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:42:22 (12.1 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/b856436c-80a0-449d-a44f-447812c06fbd/datacarpentry/spreadsheets-socialsci-v2018.05-2.zip' saved [2347690/2347690]\n",
      "\n",
      "--2020-02-07 12:42:22--  https://zenodo.org/api/files/2ac05097-56ee-4a31-bbf6-b28f25c70ed8/Docker_Analysis.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 243879178 (233M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/2ac05097-56ee-4a31-bbf6-b28f25c70ed8/Docker_Analysis.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 232.58M  14.2MB/s    in 16s     \n",
      "\n",
      "2020-02-07 12:42:38 (14.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/2ac05097-56ee-4a31-bbf6-b28f25c70ed8/Docker_Analysis.zip' saved [243879178/243879178]\n",
      "\n",
      "--2020-02-07 12:42:38--  https://zenodo.org/api/files/9d2e22e0-b077-4844-96ee-09f2556e1310/2019-05-23_HandsOn_GrainBoundary1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 93656378 (89M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/9d2e22e0-b077-4844-96ee-09f2556e1310/2019-05-23_HandsOn_GrainBoundary1.zip'\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "zenodo.org/api/file 100%[===================>]  89.32M  12.9MB/s    in 7.0s    \n",
      "\n",
      "2020-02-07 12:42:46 (12.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/9d2e22e0-b077-4844-96ee-09f2556e1310/2019-05-23_HandsOn_GrainBoundary1.zip' saved [93656378/93656378]\n",
      "\n",
      "--2020-02-07 12:42:46--  https://zenodo.org/api/files/aceb9a66-c991-4d19-a299-0c64061e1f19/larmarange/analyse-R-2018-12-08.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 36770908 (35M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/aceb9a66-c991-4d19-a299-0c64061e1f19/larmarange/analyse-R-2018-12-08.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  35.07M  13.2MB/s    in 2.7s    \n",
      "\n",
      "2020-02-07 12:42:49 (13.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/aceb9a66-c991-4d19-a299-0c64061e1f19/larmarange/analyse-R-2018-12-08.zip' saved [36770908/36770908]\n",
      "\n",
      "--2020-02-07 12:42:49--  https://zenodo.org/api/files/a991f7bc-f870-42ff-8ed1-9613fe3f9c46/o2r_project_website_and_blog_git-repository.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 75138265 (72M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/a991f7bc-f870-42ff-8ed1-9613fe3f9c46/o2r_project_website_and_blog_git-repository.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  71.66M  10.1MB/s    in 6.5s    \n",
      "\n",
      "2020-02-07 12:42:56 (11.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/a991f7bc-f870-42ff-8ed1-9613fe3f9c46/o2r_project_website_and_blog_git-repository.zip' saved [75138265/75138265]\n",
      "\n",
      "--2020-02-07 12:42:56--  https://zenodo.org/api/files/ded35234-409b-4dac-be09-979db1e92ab8/ro-1.0.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 16166633 (15M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ded35234-409b-4dac-be09-979db1e92ab8/ro-1.0.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  15.42M  15.0MB/s    in 1.0s    \n",
      "\n",
      "2020-02-07 12:42:57 (15.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ded35234-409b-4dac-be09-979db1e92ab8/ro-1.0.0.zip' saved [16166633/16166633]\n",
      "\n",
      "--2020-02-07 12:42:57--  https://zenodo.org/api/files/239861a2-8ee1-4129-879f-26119cb0c2bc/1_orochi-0.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 87804 (86K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/239861a2-8ee1-4129-879f-26119cb0c2bc/1_orochi-0.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  85.75K  --.-KB/s    in 0.02s   \n",
      "\n",
      "2020-02-07 12:42:58 (3.81 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/239861a2-8ee1-4129-879f-26119cb0c2bc/1_orochi-0.1.zip' saved [87804/87804]\n",
      "\n",
      "--2020-02-07 12:42:58--  https://zenodo.org/api/files/6fd24302-64d9-4464-83dc-87ad48d54744/models.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 697338056 (665M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/6fd24302-64d9-4464-83dc-87ad48d54744/models.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 665.03M  13.3MB/s    in 48s     \n",
      "\n",
      "2020-02-07 12:43:46 (14.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/6fd24302-64d9-4464-83dc-87ad48d54744/models.zip' saved [697338056/697338056]\n",
      "\n",
      "--2020-02-07 12:43:46--  https://zenodo.org/api/files/1bac5a90-319a-4d22-a462-15ca96c5f080/hydrosoc/rhydro_EGU19-v1.0.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 71723758 (68M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/1bac5a90-319a-4d22-a462-15ca96c5f080/hydrosoc/rhydro_EGU19-v1.0.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  68.40M  12.1MB/s    in 5.4s    \n",
      "\n",
      "2020-02-07 12:43:52 (12.6 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/1bac5a90-319a-4d22-a462-15ca96c5f080/hydrosoc/rhydro_EGU19-v1.0.0.zip' saved [71723758/71723758]\n",
      "\n",
      "--2020-02-07 12:43:52--  https://zenodo.org/api/files/3ae0de4d-2e4e-44f0-bace-84cd4dff924b/datacarpentry/r-intro-geospatial-v2018.08.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 6022735 (5.7M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3ae0de4d-2e4e-44f0-bace-84cd4dff924b/datacarpentry/r-intro-geospatial-v2018.08.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   5.74M  12.3MB/s    in 0.5s    \n",
      "\n",
      "2020-02-07 12:43:53 (12.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3ae0de4d-2e4e-44f0-bace-84cd4dff924b/datacarpentry/r-intro-geospatial-v2018.08.1.zip' saved [6022735/6022735]\n",
      "\n",
      "--2020-02-07 12:43:53--  https://zenodo.org/api/files/06562a9c-22e1-4c16-af79-efc21cec2b89/ro-crate-0.2.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4314752 (4.1M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/06562a9c-22e1-4c16-af79-efc21cec2b89/ro-crate-0.2.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.11M  12.7MB/s    in 0.3s    \n",
      "\n",
      "2020-02-07 12:43:54 (12.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/06562a9c-22e1-4c16-af79-efc21cec2b89/ro-crate-0.2.1.zip' saved [4314752/4314752]\n",
      "\n",
      "--2020-02-07 12:43:54--  https://zenodo.org/api/files/5a46c61f-c44c-4530-a9db-4a40fcb4f071/lazappi/phd-thesis-2019-05-08.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 86162898 (82M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/5a46c61f-c44c-4530-a9db-4a40fcb4f071/lazappi/phd-thesis-2019-05-08.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  82.17M  12.5MB/s    in 6.3s    \n",
      "\n",
      "2020-02-07 12:44:01 (13.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/5a46c61f-c44c-4530-a9db-4a40fcb4f071/lazappi/phd-thesis-2019-05-08.zip' saved [86162898/86162898]\n",
      "\n",
      "--2020-02-07 12:44:01--  https://zenodo.org/api/files/8c8516ea-3412-4bb9-849d-5a4ef91a6f59/datacarpentry/socialsci-workshop-v2019.06.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1784366 (1.7M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/8c8516ea-3412-4bb9-849d-5a4ef91a6f59/datacarpentry/socialsci-workshop-v2019.06.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.70M  4.06MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:44:02 (4.06 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/8c8516ea-3412-4bb9-849d-5a4ef91a6f59/datacarpentry/socialsci-workshop-v2019.06.1.zip' saved [1784366/1784366]\n",
      "\n",
      "--2020-02-07 12:44:02--  https://zenodo.org/api/files/9fef9cdb-5901-48e0-ad55-63328bfaf7b9/adamisntdead/higher-shapes-1.0.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1799379 (1.7M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/9fef9cdb-5901-48e0-ad55-63328bfaf7b9/adamisntdead/higher-shapes-1.0.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.72M  8.19MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:44:03 (8.19 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/9fef9cdb-5901-48e0-ad55-63328bfaf7b9/adamisntdead/higher-shapes-1.0.1.zip' saved [1799379/1799379]\n",
      "\n",
      "--2020-02-07 12:44:03--  https://zenodo.org/api/files/9ddb5281-3ddd-4bd4-9746-fa74089c8b3a/swcarpentry/git-novice-v2019.06.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3872151 (3.7M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/9ddb5281-3ddd-4bd4-9746-fa74089c8b3a/swcarpentry/git-novice-v2019.06.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.69M  8.98MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:44:04 (8.98 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/9ddb5281-3ddd-4bd4-9746-fa74089c8b3a/swcarpentry/git-novice-v2019.06.1.zip' saved [3872151/3872151]\n",
      "\n",
      "--2020-02-07 12:44:04--  https://zenodo.org/api/files/92d8577a-bf28-452c-847f-3a542087daeb/Open-Scholarship-Strategy/indexed-2.04.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 36254504 (35M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/92d8577a-bf28-452c-847f-3a542087daeb/Open-Scholarship-Strategy/indexed-2.04.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  34.57M  12.2MB/s    in 2.8s    \n",
      "\n",
      "2020-02-07 12:44:07 (12.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/92d8577a-bf28-452c-847f-3a542087daeb/Open-Scholarship-Strategy/indexed-2.04.zip' saved [36254504/36254504]\n",
      "\n",
      "--2020-02-07 12:44:07--  https://zenodo.org/api/files/7968d3ec-1501-4bd2-8bd3-3d934dd492a2/datacarpentry/OpenRefine-ecology-lesson-v2017.04.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 529473 (517K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/7968d3ec-1501-4bd2-8bd3-3d934dd492a2/datacarpentry/OpenRefine-ecology-lesson-v2017.04.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 517.06K  --.-KB/s    in 0.04s   \n",
      "\n",
      "2020-02-07 12:44:08 (11.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/7968d3ec-1501-4bd2-8bd3-3d934dd492a2/datacarpentry/OpenRefine-ecology-lesson-v2017.04.0.zip' saved [529473/529473]\n",
      "\n",
      "--2020-02-07 12:44:08--  https://zenodo.org/api/files/4f58ed6e-f2b8-449a-8d38-2aa8de8f0436/sensebox-binder.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1434295240 (1.3G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/4f58ed6e-f2b8-449a-8d38-2aa8de8f0436/sensebox-binder.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.33G  17.5MB/s    in 77s     \n",
      "\n",
      "2020-02-07 12:45:25 (17.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/4f58ed6e-f2b8-449a-8d38-2aa8de8f0436/sensebox-binder.zip' saved [1434295240/1434295240]\n",
      "\n",
      "--2020-02-07 12:45:25--  https://zenodo.org/api/files/0a61420f-dcd0-42a9-8cb1-b80a27533210/result.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 10038361 (9.6M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/0a61420f-dcd0-42a9-8cb1-b80a27533210/result.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   9.57M  15.5MB/s    in 0.6s    \n",
      "\n",
      "2020-02-07 12:45:26 (15.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/0a61420f-dcd0-42a9-8cb1-b80a27533210/result.zip' saved [10038361/10038361]\n",
      "\n",
      "--2020-02-07 12:45:26--  https://zenodo.org/api/files/dc18a22f-3b8a-460d-acdf-667f4295f09c/alt-pub-alt-pub_v1.0.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 28536 (28K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/dc18a22f-3b8a-460d-acdf-667f4295f09c/alt-pub-alt-pub_v1.0.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  27.87K  --.-KB/s    in 0s      \n",
      "\n",
      "2020-02-07 12:45:27 (123 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/dc18a22f-3b8a-460d-acdf-667f4295f09c/alt-pub-alt-pub_v1.0.0.zip' saved [28536/28536]\n",
      "\n",
      "--2020-02-07 12:45:27--  https://zenodo.org/api/files/d8f24c9f-d583-4fdd-bbb9-008342056fa0/onecodex/bacillus-anthracis-panel-jan2019.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 376584 (368K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/d8f24c9f-d583-4fdd-bbb9-008342056fa0/onecodex/bacillus-anthracis-panel-jan2019.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 367.76K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2020-02-07 12:45:27 (3.48 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/d8f24c9f-d583-4fdd-bbb9-008342056fa0/onecodex/bacillus-anthracis-panel-jan2019.zip' saved [376584/376584]\n",
      "\n",
      "--2020-02-07 12:45:27--  https://zenodo.org/api/files/687b0e16-5d65-4def-8a7e-e301845a27d4/datacarpentry/python-ecology-lesson-es-v2019.01.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 11322458 (11M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/687b0e16-5d65-4def-8a7e-e301845a27d4/datacarpentry/python-ecology-lesson-es-v2019.01.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  10.80M  15.0MB/s    in 0.7s    \n",
      "\n",
      "2020-02-07 12:45:29 (15.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/687b0e16-5d65-4def-8a7e-e301845a27d4/datacarpentry/python-ecology-lesson-es-v2019.01.0.zip' saved [11322458/11322458]\n",
      "\n",
      "--2020-02-07 12:45:29--  https://zenodo.org/api/files/75358699-957e-49c0-8d1f-d885b9a9acad/programminghistorian/jekyll-2017-11.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 244976781 (234M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/75358699-957e-49c0-8d1f-d885b9a9acad/programminghistorian/jekyll-2017-11.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 233.63M  16.9MB/s    in 15s     \n",
      "\n",
      "2020-02-07 12:45:45 (15.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/75358699-957e-49c0-8d1f-d885b9a9acad/programminghistorian/jekyll-2017-11.zip' saved [244976781/244976781]\n",
      "\n",
      "--2020-02-07 12:45:45--  https://zenodo.org/api/files/0598fa8e-78b7-4826-a412-9cd5dd37c7db/UofTCoders/rcourse-v2.2.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 12433837 (12M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/0598fa8e-78b7-4826-a412-9cd5dd37c7db/UofTCoders/rcourse-v2.2.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  11.86M  16.2MB/s    in 0.7s    \n",
      "\n",
      "2020-02-07 12:45:46 (16.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/0598fa8e-78b7-4826-a412-9cd5dd37c7db/UofTCoders/rcourse-v2.2.0.zip' saved [12433837/12433837]\n",
      "\n",
      "--2020-02-07 12:45:46--  https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/RunTimeResearchObject-f0b553d37e4255a3291393948f3e308bd88ed301.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 141133 (138K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/RunTimeResearchObject-f0b553d37e4255a3291393948f3e308bd88ed301.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 137.83K  --.-KB/s    in 0.008s  \n",
      "\n",
      "2020-02-07 12:45:46 (16.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/RunTimeResearchObject-f0b553d37e4255a3291393948f3e308bd88ed301.zip' saved [141133/141133]\n",
      "\n",
      "--2020-02-07 12:45:46--  https://zenodo.org/api/files/5a570ce1-9182-4931-929b-cd24fc663c13/common-workflow-language/user_guide-v0.3.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 7126114 (6.8M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/5a570ce1-9182-4931-929b-cd24fc663c13/common-workflow-language/user_guide-v0.3.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   6.80M  15.3MB/s    in 0.4s    \n",
      "\n",
      "2020-02-07 12:45:47 (15.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/5a570ce1-9182-4931-929b-cd24fc663c13/common-workflow-language/user_guide-v0.3.zip' saved [7126114/7126114]\n",
      "\n",
      "--2020-02-07 12:45:47--  https://zenodo.org/api/files/966442f8-3955-4ab1-8706-fd1e0a84b300/Bachelorarbeit.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 34212350 (33M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/966442f8-3955-4ab1-8706-fd1e0a84b300/Bachelorarbeit.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  32.63M  14.2MB/s    in 2.3s    \n",
      "\n",
      "2020-02-07 12:45:50 (14.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/966442f8-3955-4ab1-8706-fd1e0a84b300/Bachelorarbeit.zip' saved [34212350/34212350]\n",
      "\n",
      "--2020-02-07 12:45:50--  https://zenodo.org/api/files/6707b75d-848d-46f5-8cbf-36f7047a6458/data.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 148931121 (142M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/6707b75d-848d-46f5-8cbf-36f7047a6458/data.zip'\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "zenodo.org/api/file 100%[===================>] 142.03M  15.8MB/s    in 9.3s    \n",
      "\n",
      "2020-02-07 12:46:00 (15.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/6707b75d-848d-46f5-8cbf-36f7047a6458/data.zip' saved [148931121/148931121]\n",
      "\n",
      "--2020-02-07 12:46:00--  https://zenodo.org/api/files/94257e8d-bb76-45df-8448-68b04dfeb2bc/v.2018-02-28.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 666510 (651K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/94257e8d-bb76-45df-8448-68b04dfeb2bc/v.2018-02-28.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 650.89K  --.-KB/s    in 0.07s   \n",
      "\n",
      "2020-02-07 12:46:00 (9.50 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/94257e8d-bb76-45df-8448-68b04dfeb2bc/v.2018-02-28.zip' saved [666510/666510]\n",
      "\n",
      "--2020-02-07 12:46:00--  https://zenodo.org/api/files/cb9df964-0c4e-4829-a6a4-b9739ccf3c19/DMPs.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 170620 (167K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/cb9df964-0c4e-4829-a6a4-b9739ccf3c19/DMPs.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 166.62K  --.-KB/s    in 0.03s   \n",
      "\n",
      "2020-02-07 12:46:01 (4.90 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/cb9df964-0c4e-4829-a6a4-b9739ccf3c19/DMPs.zip' saved [170620/170620]\n",
      "\n",
      "--2020-02-07 12:46:01--  https://zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Send%20Invoice%20PaaS.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1095001 (1.0M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Send Invoice PaaS.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.04M  --.-KB/s    in 0.1s    \n",
      "\n",
      "2020-02-07 12:46:01 (9.25 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/289958e3-1a45-4b8b-bf13-f7b20bf6be10/Send Invoice PaaS.zip' saved [1095001/1095001]\n",
      "\n",
      "--2020-02-07 12:46:01--  https://zenodo.org/api/files/b607155e-c427-4380-86fd-4667ee10fa4f/tsamsonov/r-geo-course-v0.1-alpha.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 6665918 (6.4M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/b607155e-c427-4380-86fd-4667ee10fa4f/tsamsonov/r-geo-course-v0.1-alpha.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   6.36M  12.2MB/s    in 0.5s    \n",
      "\n",
      "2020-02-07 12:46:02 (12.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/b607155e-c427-4380-86fd-4667ee10fa4f/tsamsonov/r-geo-course-v0.1-alpha.zip' saved [6665918/6665918]\n",
      "\n",
      "--2020-02-07 12:46:02--  https://zenodo.org/api/files/1a03eb70-03a2-454c-8f8c-a03c4775ea44/ICSE-data.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 56739072 (54M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/1a03eb70-03a2-454c-8f8c-a03c4775ea44/ICSE-data.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  54.11M  16.3MB/s    in 3.4s    \n",
      "\n",
      "2020-02-07 12:46:06 (16.1 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/1a03eb70-03a2-454c-8f8c-a03c4775ea44/ICSE-data.zip' saved [56739072/56739072]\n",
      "\n",
      "--2020-02-07 12:46:06--  https://zenodo.org/api/files/0dbaadcb-0029-45e9-b25f-8f426e54d1d7/AnnexesNumeriques.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 401118831 (383M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/0dbaadcb-0029-45e9-b25f-8f426e54d1d7/AnnexesNumeriques.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 382.54M  16.7MB/s    in 20s     \n",
      "\n",
      "2020-02-07 12:46:27 (18.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/0dbaadcb-0029-45e9-b25f-8f426e54d1d7/AnnexesNumeriques.zip' saved [401118831/401118831]\n",
      "\n",
      "--2020-02-07 12:46:27--  https://zenodo.org/api/files/e934c4b3-70ae-4214-b045-10e166125907/swcarpentry/r-novice-gapminder-es-v2018.05-1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 5124022 (4.9M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/e934c4b3-70ae-4214-b045-10e166125907/swcarpentry/r-novice-gapminder-es-v2018.05-1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.89M  15.8MB/s    in 0.3s    \n",
      "\n",
      "2020-02-07 12:46:28 (15.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/e934c4b3-70ae-4214-b045-10e166125907/swcarpentry/r-novice-gapminder-es-v2018.05-1.zip' saved [5124022/5124022]\n",
      "\n",
      "--2020-02-07 12:46:28--  https://zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/cwltool-61797e04db7db33486d6074ec4ecc7dadd20d5e6.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3552427 (3.4M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/cwltool-61797e04db7db33486d6074ec4ecc7dadd20d5e6.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.39M  14.5MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:46:28 (14.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/39641b52-1c14-40d6-9763-b4eb2130609d/cwltool-61797e04db7db33486d6074ec4ecc7dadd20d5e6.zip' saved [3552427/3552427]\n",
      "\n",
      "--2020-02-07 12:46:28--  https://zenodo.org/api/files/bf6bb00d-dd26-4a6f-8924-95a16d8ce24d/Duerfeld_Stein_2018_ID%2B_An_Interdisciplinarity_Model_Website_Code.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4267439 (4.1M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/bf6bb00d-dd26-4a6f-8924-95a16d8ce24d/Duerfeld_Stein_2018_ID+_An_Interdisciplinarity_Model_Website_Code.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   4.07M  16.4MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:46:29 (16.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/bf6bb00d-dd26-4a6f-8924-95a16d8ce24d/Duerfeld_Stein_2018_ID+_An_Interdisciplinarity_Model_Website_Code.zip' saved [4267439/4267439]\n",
      "\n",
      "--2020-02-07 12:46:29--  https://zenodo.org/api/files/c8094de0-dd94-4284-b860-4f375c76f10b/datacarpentry/openrefine-socialsci-v2018.05-3.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2075422 (2.0M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/c8094de0-dd94-4284-b860-4f375c76f10b/datacarpentry/openrefine-socialsci-v2018.05-3.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.98M  12.4MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:46:30 (12.4 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/c8094de0-dd94-4284-b860-4f375c76f10b/datacarpentry/openrefine-socialsci-v2018.05-3.zip' saved [2075422/2075422]\n",
      "\n",
      "--2020-02-07 12:46:30--  https://zenodo.org/api/files/ae20c12b-5380-496d-9569-d57c5f5bfcca/ensembleTopic-1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 44120858 (42M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ae20c12b-5380-496d-9569-d57c5f5bfcca/ensembleTopic-1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  42.08M  11.7MB/s    in 3.6s    \n",
      "\n",
      "2020-02-07 12:46:34 (11.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ae20c12b-5380-496d-9569-d57c5f5bfcca/ensembleTopic-1.0.zip' saved [44120858/44120858]\n",
      "\n",
      "--2020-02-07 12:46:34--  https://zenodo.org/api/files/8d80e007-d1a7-4634-9652-d810839251fc/Results.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3159542 (3.0M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/8d80e007-d1a7-4634-9652-d810839251fc/Results.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.01M  11.5MB/s    in 0.3s    \n",
      "\n",
      "2020-02-07 12:46:35 (11.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/8d80e007-d1a7-4634-9652-d810839251fc/Results.zip' saved [3159542/3159542]\n",
      "\n",
      "--2020-02-07 12:46:35--  https://zenodo.org/api/files/2d2652c6-c555-43e4-aa87-223d3da93b54/swcarpentry/make-novice-v2019.06.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3602821 (3.4M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/2d2652c6-c555-43e4-aa87-223d3da93b54/swcarpentry/make-novice-v2019.06.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.44M  11.5MB/s    in 0.3s    \n",
      "\n",
      "2020-02-07 12:46:36 (11.5 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/2d2652c6-c555-43e4-aa87-223d3da93b54/swcarpentry/make-novice-v2019.06.1.zip' saved [3602821/3602821]\n",
      "\n",
      "--2020-02-07 12:46:36--  https://zenodo.org/api/files/75e63414-b867-4a08-af77-69ae4c058c27/Coupette_Juristische_Netzwerkforschung_Online-Appendix.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 129183992 (123M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/75e63414-b867-4a08-af77-69ae4c058c27/Coupette_Juristische_Netzwerkforschung_Online-Appendix.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 123.20M  17.0MB/s    in 7.9s    \n",
      "\n",
      "2020-02-07 12:46:44 (15.7 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/75e63414-b867-4a08-af77-69ae4c058c27/Coupette_Juristische_Netzwerkforschung_Online-Appendix.zip' saved [129183992/129183992]\n",
      "\n",
      "--2020-02-07 12:46:44--  https://zenodo.org/api/files/8e5381ec-98e4-4232-a816-71ffca086162/UofTCoders/rcourse-v2.1.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 12433922 (12M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/8e5381ec-98e4-4232-a816-71ffca086162/UofTCoders/rcourse-v2.1.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  11.86M  15.8MB/s    in 0.8s    \n",
      "\n",
      "2020-02-07 12:46:45 (15.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/8e5381ec-98e4-4232-a816-71ffca086162/UofTCoders/rcourse-v2.1.0.zip' saved [12433922/12433922]\n",
      "\n",
      "--2020-02-07 12:46:45--  https://zenodo.org/api/files/8c9a9852-1516-4dde-b7fe-61ebc7325ba7/trallard/ML_retreat_presentation-1.1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 13786605 (13M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/8c9a9852-1516-4dde-b7fe-61ebc7325ba7/trallard/ML_retreat_presentation-1.1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  13.15M  17.2MB/s    in 0.8s    \n",
      "\n",
      "2020-02-07 12:46:46 (17.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/8c9a9852-1516-4dde-b7fe-61ebc7325ba7/trallard/ML_retreat_presentation-1.1.zip' saved [13786605/13786605]\n",
      "\n",
      "--2020-02-07 12:46:46--  https://zenodo.org/api/files/c037e866-1fcd-4c7a-ab00-fc18061527bc/ore-0.9.0.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 18562 (18K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/c037e866-1fcd-4c7a-ab00-fc18061527bc/ore-0.9.0.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  18.13K  --.-KB/s    in 0s      \n",
      "\n",
      "2020-02-07 12:46:47 (193 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/c037e866-1fcd-4c7a-ab00-fc18061527bc/ore-0.9.0.zip' saved [18562/18562]\n",
      "\n",
      "--2020-02-07 12:46:47--  https://zenodo.org/api/files/3a13c696-b652-4883-8323-00bb3c8f76a0/supplementary_material.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 131464101 (125M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3a13c696-b652-4883-8323-00bb3c8f76a0/supplementary_material.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 125.37M  16.3MB/s    in 8.2s    \n",
      "\n",
      "2020-02-07 12:46:56 (15.3 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3a13c696-b652-4883-8323-00bb3c8f76a0/supplementary_material.zip' saved [131464101/131464101]\n",
      "\n",
      "--2020-02-07 12:46:56--  https://zenodo.org/api/files/ff0c635a-6166-4159-826a-5bdcfa68bfd2/gds15-v0.9.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 134599835 (128M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/ff0c635a-6166-4159-826a-5bdcfa68bfd2/gds15-v0.9.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 128.36M  17.5MB/s    in 8.1s    \n",
      "\n",
      "2020-02-07 12:47:04 (15.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/ff0c635a-6166-4159-826a-5bdcfa68bfd2/gds15-v0.9.zip' saved [134599835/134599835]\n",
      "\n",
      "--2020-02-07 12:47:04--  https://zenodo.org/api/files/677eec6c-cbd3-4e93-823b-bde72f7e2319/labordynamicsinstitute/replicability-presentation2019-v20190328.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2422338 (2.3M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/677eec6c-cbd3-4e93-823b-bde72f7e2319/labordynamicsinstitute/replicability-presentation2019-v20190328.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   2.31M  13.9MB/s    in 0.2s    \n",
      "\n",
      "2020-02-07 12:47:05 (13.9 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/677eec6c-cbd3-4e93-823b-bde72f7e2319/labordynamicsinstitute/replicability-presentation2019-v20190328.zip' saved [2422338/2422338]\n",
      "\n",
      "--2020-02-07 12:47:05--  https://zenodo.org/api/files/dbedfef0-2d4c-4183-8373-85887afb76ab/savmap_dataset_v2.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 3509832128 (3.3G) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/dbedfef0-2d4c-4183-8373-85887afb76ab/savmap_dataset_v2.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   3.27G  14.3MB/s    in 3m 19s  \n",
      "\n",
      "2020-02-07 12:50:24 (16.8 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/dbedfef0-2d4c-4183-8373-85887afb76ab/savmap_dataset_v2.zip' saved [3509832128/3509832128]\n",
      "\n",
      "--2020-02-07 12:50:24--  https://zenodo.org/api/files/3db8c7f1-5d79-4ec0-ab3f-eff34d09e58f/2018-10-19_MD-Dislocation-Motion-in-Silicon.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 92300632 (88M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/3db8c7f1-5d79-4ec0-ab3f-eff34d09e58f/2018-10-19_MD-Dislocation-Motion-in-Silicon.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]  88.02M  16.2MB/s    in 5.2s    \n",
      "\n",
      "2020-02-07 12:50:30 (17.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/3db8c7f1-5d79-4ec0-ab3f-eff34d09e58f/2018-10-19_MD-Dislocation-Motion-in-Silicon.zip' saved [92300632/92300632]\n",
      "\n",
      "--2020-02-07 12:50:30--  https://zenodo.org/api/files/f1dfc12c-d9ba-4eed-987d-9eeab82b48fb/pddi-cds-level1-implementation.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1271677 (1.2M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/f1dfc12c-d9ba-4eed-987d-9eeab82b48fb/pddi-cds-level1-implementation.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>]   1.21M  --.-KB/s    in 0.09s   \n",
      "\n",
      "2020-02-07 12:50:31 (13.2 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/f1dfc12c-d9ba-4eed-987d-9eeab82b48fb/pddi-cds-level1-implementation.zip' saved [1271677/1271677]\n",
      "\n",
      "--2020-02-07 12:50:31--  https://zenodo.org/api/files/aed46454-e7fd-4615-bd88-d038c0a56e59/WG3-MetadataSpecifications-v1.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 873413 (853K) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/aed46454-e7fd-4615-bd88-d038c0a56e59/WG3-MetadataSpecifications-v1.zip'\n",
      "\n",
      "zenodo.org/api/file 100%[===================>] 852.94K  --.-KB/s    in 0.08s   \n",
      "\n",
      "2020-02-07 12:50:31 (11.0 MB/s) - 'data/metadata-files/json/zenodo.org/api/files/aed46454-e7fd-4615-bd88-d038c0a56e59/WG3-MetadataSpecifications-v1.zip' saved [873413/873413]\n",
      "\n",
      "--2020-02-07 12:50:31--  https://zenodo.org/api/files/2d2c8700-631b-4d19-9c7b-477be6931bda/o2r_project_website_and_blog_git-repository.zip\n",
      "Reusing existing connection to zenodo.org:443.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 74741781 (71M) [application/octet-stream]\n",
      "Saving to: 'data/metadata-files/json/zenodo.org/api/files/2d2c8700-631b-4d19-9c7b-477be6931bda/o2r_project_website_and_blog_git-repository.zip'\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pi/files/2d2c8700-6  75%[==============>     ]  53.49M  10.3MB/s    eta 2s     "
     ]
    }
   ],
   "source": [
    "!cat data/metadata-files/json/urls.txt | xargs wget --mirror --directory-prefix=data/metadata-files/json/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cd data/metadata-files/json; for f in `find . -name '*zip'` ; do DIR=`echo $f | sed s/.zip$//` ; unzip -d \"$DIR\" $f '*.json' ;  done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Commonly JSON files start with `{ key: value}` at the top level, so it might be a good question to see if there are any common top-level keys such as `creator` or `metadata`. Note that the counts here are per JSON file, and many archives will have many JSON files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! find data/metadata-files/json -type f -name '*json' | xargs jq -r 'keys[]'  | sort | uniq -c > data/metadata-files/json/top-keys.txt\n",
    "! sort data/metadata-files/json/top-keys.txt | grep -v \" 1 \""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## XML files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "! egrep -ri '(xml)$' data/*/listing > data/metadata-files/xml/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "data/dataset/listing/zbgj.txt:Math_89/findbugs-exclude-filter.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/build.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/pom.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/test-jar.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/project.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/checkstyle.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/maven.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/assembly/src.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/assembly/bin.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/developers.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/index.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/tasks.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/optimization.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/index.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/transform.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/complex.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/overview.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/random.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/special.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/utilities.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/analysis.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/geometry.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/linear.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/distribution.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/ode.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/stat.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/userguide/fraction.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/proposal.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/download_math.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/releases.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/issue-tracking.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/changes.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/xdoc/navigation.xml\r\n",
      "data/dataset/listing/zbgj.txt:Math_89/src/site/site.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NA52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NB52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NC52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/ND51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NE51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NF51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NG51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NH52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NI54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NJ54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NK55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NL54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NM57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN03.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN04.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN29 -- needs to be incorporated into final dataset - not yet clipped, projected, etc/NN29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN29.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN31.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NN59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO03.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO04.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO05.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO07.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO08.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO30.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NO58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP03.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP04.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP05.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP06.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP07.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP08.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP27.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NP60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ01.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ02.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ03.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ04.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ05.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ06.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ07.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ08.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ27.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ28.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NQ60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR01.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR03.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR04.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR05.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR06.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR07.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR08.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR09.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR26.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR27.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NR60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS10.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS11.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS13.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS14.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS26.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS27.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS43.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS44.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS45.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS46.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NS56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT12.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT27.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT42.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NT54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU15.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU16.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/NU41.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA47.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SA56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB25.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB48.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SB57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC17.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC25.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC49.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC57.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SC58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SD55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE32.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE37.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE52.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SE60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF24.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF39.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF40.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF51.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF53.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SF58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG23.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG38.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SG56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH33.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH36.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SH56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI22.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI34.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI35.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI50.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI56.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SI60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ21.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ54.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SJ60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK20.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK55.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SK60.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SL18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SL19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SL58.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SL59.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SM18.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SM19.shp.xml\r\n",
      "data/dataset/listing/fk.txt:GRWL_centerlineData/SN19.shp.xml\r\n",
      "data/dataset/listing/zaem.txt:Closure_98/build.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/data/raw/BioClim_variables/pca.tif.aux.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/data/raw/GRUMP_V1_Population_Density/GlobalRuralUrbanMappingProjectVersion1GRUMPv1PopulationDensityGrid.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/Hmisc/tests/test.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/CurlSSL/curlSSL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/doc/cookies.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/doc/fileUploads.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/doc/philosophy.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/examples/concurrent.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/examples/nestedHTML.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/RCurl/examples/xmlParse.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/geometry/doc/html/qhull-cpp.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestBoundary.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestCentroid.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestConvexHull-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestConvexHull.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionAAPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionLAPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionLLPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionPA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionPL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionPLPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestFunctionPP.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestInteriorPoint.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRectanglePredicate.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelateAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelateAC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelateLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelateLC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelateLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelatePA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelatePL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestRelatePP.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestSimple.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestValid.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestValid2-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestValid2.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/general/TestWithinDistance.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/robust/ExternalRobustness.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/robust/TestRobustOverlayFixed.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/robust/TestRobustOverlayFloat.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/robust/TestRobustRelate.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateAA-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateAC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateLC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelateLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelatePA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelatePL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.2/rgeos/tests/testthat/testxml/validate/TestRelatePP.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/Hmisc/tests/test.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/CurlSSL/curlSSL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/doc/cookies.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/doc/fileUploads.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/doc/philosophy.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/examples/concurrent.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/examples/nestedHTML.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/RCurl/examples/xmlParse.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/geometry/doc/html/qhull-cpp.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestBoundary.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestCentroid.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestConvexHull-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestConvexHull.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionAAPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionLAPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionLLPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionPA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionPL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionPLPrec.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestFunctionPP.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestInteriorPoint.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRectanglePredicate.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelateAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelateAC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelateLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelateLC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelateLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelatePA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelatePL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestRelatePP.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestSimple.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestValid.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestValid2-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestValid2.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/general/TestWithinDistance.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/robust/ExternalRobustness.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/robust/TestRobustOverlayFixed.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/robust/TestRobustOverlayFloat.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/robust/TestRobustRelate.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateAA-big.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateAA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateAC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateLA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateLC.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelateLL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelatePA.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelatePL.xml\r\n",
      "data/dataset/listing/ar.txt:raptr-manuscript/packrat/lib/x86_64-pc-linux-gnu/3.3.3/rgeos/tests/testthat/testxml/validate/TestRelatePP.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/toolsrc/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/toolsrc/org/mozilla/javascript/tools/debugger/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/deprecatedsrc/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/xmlimplsrc/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/testsrc/build.xml\r\n",
      "data/dataset/listing/zakh.txt:Closure_8/lib/rhino/src/build.xml\r\n",
      "data/dataset/listing/zazp.txt:game-121-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-122-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-123-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-124-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-125-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-126-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-127-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-128-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-129-boot.xml\r\n",
      "data/dataset/listing/zazp.txt:game-130-boot.xml\r\n",
      "data/dataset/listing/zbqp.txt:Closure_81/build.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h0_EQ.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h0_SV.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h0_TP.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h10_EQ.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h10_SV.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h10_TP.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h11_EQ.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h11_SV.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h11_TP.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h12_EQ.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h12_SV.xml\r\n",
      "data/dataset/listing/zbtg.txt:N44_20150914/CIMSnapshots/N44_h12_TP.xml\r\n"
     ]
    }
   ],
   "source": [
    "!head -n 1000 data/metadata-files/xml/listing.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cat data/metadata-files/xml/urls.txt | xargs wget --mirror --directory-prefix=data/metadata-files/xml/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cd data/metadata-files/xml; for f in `find . -name '*zip'` ; do DIR=`echo $f | sed s/.zip$//` ; unzip -d \"$DIR\" $f '*.xml' ;  done"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
